<!DOCTYPE html>
<!-- saved from url=(0079)https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485 -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" class=" js no-touch cssanimations csstransitions" id="mainTemplate" data-darkreader-mode="dynamic" data-darkreader-scheme="dark" style=""><!--<![endif]--><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><style class="darkreader darkreader--fallback" media="screen"></style><style class="darkreader darkreader--text" media="screen"></style><style class="darkreader darkreader--invert" media="screen">.jfk-bubble.gtx-bubble, .captcheck_answer_label > input + img, span#closed_text > img[src^="https://www.gstatic.com/images/branding/googlelogo"], span[data-href^="https://www.hcaptcha.com/"] > #icon, #bit-notification-bar-iframe, ::-webkit-calendar-picker-indicator, img.Wirisformula {
    filter: invert(100%) hue-rotate(180deg) contrast(70%) !important;
}</style><style class="darkreader darkreader--inline" media="screen">[data-darkreader-inline-bgcolor] {
  background-color: var(--darkreader-inline-bgcolor) !important;
}
[data-darkreader-inline-bgimage] {
  background-image: var(--darkreader-inline-bgimage) !important;
}
[data-darkreader-inline-border] {
  border-color: var(--darkreader-inline-border) !important;
}
[data-darkreader-inline-border-bottom] {
  border-bottom-color: var(--darkreader-inline-border-bottom) !important;
}
[data-darkreader-inline-border-left] {
  border-left-color: var(--darkreader-inline-border-left) !important;
}
[data-darkreader-inline-border-right] {
  border-right-color: var(--darkreader-inline-border-right) !important;
}
[data-darkreader-inline-border-top] {
  border-top-color: var(--darkreader-inline-border-top) !important;
}
[data-darkreader-inline-boxshadow] {
  box-shadow: var(--darkreader-inline-boxshadow) !important;
}
[data-darkreader-inline-color] {
  color: var(--darkreader-inline-color) !important;
}
[data-darkreader-inline-fill] {
  fill: var(--darkreader-inline-fill) !important;
}
[data-darkreader-inline-stroke] {
  stroke: var(--darkreader-inline-stroke) !important;
}
[data-darkreader-inline-outline] {
  outline-color: var(--darkreader-inline-outline) !important;
}
[data-darkreader-inline-stopcolor] {
  stop-color: var(--darkreader-inline-stopcolor) !important;
}
[data-darkreader-inline-bg] {
  background: var(--darkreader-inline-bg) !important;
}</style><style class="darkreader darkreader--variables" media="screen">:root {
   --darkreader-neutral-background: #292a2b;
   --darkreader-neutral-text: #c6c3bf;
   --darkreader-selection-background: #1957a2;
   --darkreader-selection-text: #d3d2cf;
}</style><style class="darkreader darkreader--root-vars" media="screen"></style><style class="darkreader darkreader--user-agent" media="screen">html {
    background-color: #2d2e2f !important;
}
html {
    color-scheme: dark !important;
}
html, body, input, textarea, select, button, dialog {
    background-color: #2d2e2f;
}
html, body, input, textarea, select, button {
    border-color: #766f65;
    color: #d3d2cf;
}
a {
    color: #428ee6;
}
table {
    border-color: #5d6265;
}
::placeholder {
    color: #a8a29a;
}
input:-webkit-autofill,
textarea:-webkit-autofill,
select:-webkit-autofill {
    background-color: #4d5019 !important;
    color: #d3d2cf !important;
}
::selection {
    background-color: #1957a2 !important;
    color: #d3d2cf !important;
}
::-moz-selection {
    background-color: #1957a2 !important;
    color: #d3d2cf !important;
}</style>

<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- BEGIN SHAREAHOLIC CODE -->
<link rel="preload" href="https://cdn.shareaholic.net/assets/pub/shareaholic.js" as="script">
<meta name="shareaholic:site_id" content="857a9d44ab03c0132db4fc34e9bc8d44">

<!-- END SHAREAHOLIC CODE -->
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>Mutational analysis of the MS2 lysis protein L | Microbiology Society</title>
<script type="text/javascript" async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/js"></script><script async="" referrerpolicy="unsafe-url" type="text/javascript" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/v2"></script><script type="text/javascript" async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/analytics.js"></script><script type="text/javascript" async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/js(1)"></script><script type="text/javascript" async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/js(2)"></script><script async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/gtm.js"></script><script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/f.txt" async=""></script>
<link rel="schema.CRAWLER" href="https://labs.ingenta.com/2006/06/16/crawler">
<meta name="dc.title" content="Mutational analysis of the MS2 lysis protein L">
<meta name="dc.publisher" content="Microbiology Society">
<meta name="dc.type" scheme="DCMIType" content="Text">
<meta name="author" content="Karthik R. Chamakura|Garrett B. Edwards|Ry Young">
<meta name="dc.creator" content="Karthik R. Chamakura">
<meta name="dc.creator" content="Garrett B. Edwards">
<meta name="dc.creator" content="Ry Young">
<meta name="dc.identifier" content="doi:10.1099/mic.0.000485">
<meta name="dc.date" content="2017/07/01">
<meta property="og:title" content="Mutational analysis of the MS2 lysis protein L">
<meta property="og:type" content="article">
<meta property="og:image" content="https://www.microbiologyresearch.org/images/hybrid/logo/MBS_logo_full_size_desktop_lo-res.png">
<meta property="og:url" content="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485">
<meta property="og:site_name" content="microbiologyresearch.org">
<meta name="citation_journal_title" content="Microbiology">
<meta name="citation_issn" content="1350-0872">
<meta name="citation_issn" content="1465-2080">
<meta name="citation_doi" content="10.1099/mic.0.000485">
<meta name="citation_publication_date" content="2017/07/01">
<meta name="citation_date" content="2017/07/01">
<meta name="citation_year" content="2017">
<meta name="citation_online_date" content="2017/07/01">
<meta name="citation_title" content="Mutational analysis of the MS2 lysis protein L">
<meta name="citation_author" content="Karthik R. Chamakura">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Texas A&amp;M University, College Station, TX, USA">
<meta name="citation_author_institution" content="Center for Phage Technology, Texas A&amp;M AgriLife, Texas A&amp;M University, College Station, TX, USA">
<meta name="citation_author" content="Garrett B. Edwards">
<meta name="citation_author_institution" content="Present address: Department of Chemistry and Biochemistry, University of Colorado Boulder, Boulder, CO, USA.">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Texas A&amp;M University, College Station, TX, USA">
<meta name="citation_author_institution" content="Center for Phage Technology, Texas A&amp;M AgriLife, Texas A&amp;M University, College Station, TX, USA">
<meta name="citation_author" content="Ry Young">
<meta name="citation_author_institution" content="Center for Phage Technology, Texas A&amp;M AgriLife, Texas A&amp;M University, College Station, TX, USA">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Texas A&amp;M University, College Station, TX, USA">
<meta name="citation_volume" content="163">
<meta name="citation_issue" content="7">
<meta name="citation_firstpage" content="961">
<meta name="citation_lastpage" content="969">
<meta name="citation_publisher" content="Microbiology Society">
<meta name="citation_abstract" content="Small single-stranded nucleic acid phages effect lysis by expressing a single protein, the amurin, lacking muralytic enzymatic activity. Three amurins have been shown to act like ‘protein antibiotics’ by inhibiting cell-wall biosynthesis. However, the L lysis protein of the canonical ssRNA phage MS2, a 75 aa polypeptide, causes lysis by an unknown mechanism without affecting net peptidoglycan synthesis. To identify residues important for lytic function, randomly mutagenized alleles of L were generated, cloned into an inducible plasmid and the transformants were selected on agar containing the inducer. From a total of 396 clones, 67 were unique single base-pair changes that rendered L non-functional, of which 44 were missense mutants and 23 were nonsense mutants. Most of the non-functional missense alleles that accumulated in levels comparable to the wild-type allele are localized in the C-terminal half of L, clustered in and around an LS dipeptide sequence. The LS motif was used to align L genes from ssRNA phages lacking any sequence similarity to MS2 or to each other. This alignment revealed a conserved domain structure, in terms of charge, hydrophobic character and predicted helical content. None of the missense mutants affected membrane-association of L. Several of the L mutations in the central domains were highly conservative and recessive, suggesting a defect in a heterotypic protein–protein interaction, rather than in direct disruption of the bilayer structure, as had been previously proposed for L.">
<meta name="citation_abstract_html_url" content="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485">
<meta name="description" property="og:description" content="Small single-stranded nucleic acid phages effect lysis by expressing a single protein, the amurin, lacking muralytic enzymatic activity. Three amurins have been shown to act like ‘protein antibiotics’ by inhibiting cell-wall biosynthesis. However, the L lysis protein of the canonical ssRNA phage MS2, a 75 aa polypeptide, causes lysis by an unknown mechanism without affecting net peptidoglycan synthesis. To identify residues important for lytic function, randomly mutagenized alleles of L were generated, cloned into an inducible plasmid and the transformants were selected on agar containing the inducer. From a total of 396 clones, 67 were unique single base-pair changes that rendered L non-functional, of which 44 were missense mutants and 23 were nonsense mutants. Most of the non-functional missense alleles that accumulated in levels comparable to the wild-type allele are localized in the C-terminal half of L, clustered in and around an LS dipeptide sequence. The LS motif was used to align L genes from ssRNA phages lacking any sequence similarity to MS2 or to each other. This alignment revealed a conserved domain structure, in terms of charge, hydrophobic character and predicted helical content. None of the missense mutants affected membrane-association of L. Several of the L mutations in the central domains were highly conservative and recessive, suggesting a defect in a heterotypic protein–protein interaction, rather than in direct disruption of the bilayer structure, as had been previously proposed for L.">
<meta name="robots" content="NOODP,noarchive">
<meta name="stats-meta" content="stats" data-logstatisticsurl="/logstatistics.action" data-itemid="/content/journal/micro/10.1099/mic.0.000485" data-baseuri="http://instance.metastore.ingenta.com">
<link rel="shortcut icon" href="https://www.microbiologyresearch.org/upload/favicon/favicon.ico">
<link rel="stylesheet" href="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/google-material-design-font.css" type="text/css"><style class="darkreader darkreader--sync" media="screen"></style>
<link rel="stylesheet" media="screen and (max-width: 844px)" href="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/component.css" type="text/css"><style class="darkreader darkreader--sync" media="screen"></style>
<link rel="stylesheet" href="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/site.css" type="text/css"><style class="darkreader darkreader--sync" media="screen"></style>
<!--[if lte IE 10]>
<link rel="stylesheet" href="/css/hybrid/ie.css" type="text/css" />
<![endif]-->
<link rel="stylesheet" href="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/preview.css" type="text/css" media="screen, print"><style class="darkreader darkreader--sync" media="screen"></style>
<link rel="stylesheet" href="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/fulltext-html-tab.css" type="text/css" media="screen, print"><style class="darkreader darkreader--sync" media="screen"></style>
<link rel="stylesheet" href="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/jquery.fancybox.css" type="text/css" media="screen, print"><style class="darkreader darkreader--sync" media="screen"></style>
<link href="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/froala_style.min.css" rel="stylesheet" type="text/css"><style class="darkreader darkreader--cors" media="screen">.clearfix::after{clear:both;display:block;content:"";height:0}.hide-by-clipping{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);border:0}img.fr-rounded,.fr-img-caption.fr-rounded img{border-radius:10px;-moz-border-radius:10px;-webkit-border-radius:10px;-moz-background-clip:padding;-webkit-background-clip:padding-box;background-clip:padding-box}img.fr-bordered,.fr-img-caption.fr-bordered img{border:solid 5px #CCC}img.fr-bordered{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}.fr-img-caption.fr-bordered img{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}img.fr-shadow,.fr-img-caption.fr-shadow img{-webkit-box-shadow:0 1px 3px rgba(0,0,0,.12),0 1px 1px 1px rgba(0,0,0,.16);-moz-box-shadow:0 1px 3px rgba(0,0,0,.12),0 1px 1px 1px rgba(0,0,0,.16);box-shadow:0 1px 3px rgba(0,0,0,.12),0 1px 1px 1px rgba(0,0,0,.16)}.fr-view span[style~="color:"] a{color:inherit}.fr-view strong{font-weight:700}.fr-view table{border:0;border-collapse:collapse;empty-cells:show;max-width:100%;table-layout:fixed}.fr-view table td{min-width:5px}.fr-view table.fr-dashed-borders td,.fr-view table.fr-dashed-borders th{border-style:dashed}.fr-view table.fr-alternate-rows tbody tr:nth-child(2n){background:#f5f5f5}.fr-view table td,.fr-view table th{border:1px solid #ddd}.fr-view table td:empty,.fr-view table th:empty{height:20px}.fr-view table td.fr-highlighted,.fr-view table th.fr-highlighted{border:1px double red}.fr-view table td.fr-thick,.fr-view table th.fr-thick{border-width:2px}.fr-view table th{background:#e6e6e6}.fr-view hr{clear:both;user-select:none;-o-user-select:none;-moz-user-select:none;-khtml-user-select:none;-webkit-user-select:none;-ms-user-select:none;page-break-after:always}.fr-view .fr-file{position:relative}.fr-view .fr-file::after{position:relative;content:"\1F4CE";font-weight:400}.fr-view pre{white-space:pre-wrap;word-wrap:break-word;overflow:visible}.fr-view[dir=rtl] blockquote{border-left:0;border-right:solid 2px #5e35b1;margin-right:0;padding-right:5px;padding-left:0}.fr-view[dir=rtl] blockquote blockquote{border-color:#00bcd4}.fr-view[dir=rtl] blockquote blockquote blockquote{border-color:#43a047}.fr-view blockquote{border-left:solid 2px #5e35b1;margin-left:0;padding-left:5px;color:#5e35b1}.fr-view blockquote blockquote{border-color:#00bcd4;color:#00bcd4}.fr-view blockquote blockquote blockquote{border-color:#43a047;color:#43a047}.fr-view span.fr-emoticon{font-weight:400;font-family:"Apple Color Emoji","Segoe UI Emoji",NotoColorEmoji,"Segoe UI Symbol","Android Emoji",EmojiSymbols;display:inline;line-height:0}.fr-view span.fr-emoticon.fr-emoticon-img{background-repeat:no-repeat!important;font-size:inherit;height:1em;width:1em;min-height:20px;min-width:20px;display:inline-block;margin:-.1em .1em .1em;line-height:1;vertical-align:middle}.fr-view .fr-text-gray{color:#AAA!important}.fr-view .fr-text-bordered{border-top:solid 1px #222;border-bottom:solid 1px #222;padding:10px 0}.fr-view .fr-text-spaced{letter-spacing:1px}.fr-view .fr-text-uppercase{text-transform:uppercase}.fr-view img{position:relative;max-width:100%}.fr-view img.fr-dib{margin:5px auto;display:block;float:none;vertical-align:top}.fr-view img.fr-dib.fr-fil{margin-left:0;text-align:left}.fr-view img.fr-dib.fr-fir{margin-right:0;text-align:right}.fr-view img.fr-dii{display:inline-block;float:none;vertical-align:bottom;margin-left:5px;margin-right:5px;max-width:calc(100% - (2 * 5px))}.fr-view img.fr-dii.fr-fil{float:left;margin:5px 5px 5px 0;max-width:calc(100% - 5px)}.fr-view img.fr-dii.fr-fir{float:right;margin:5px 0 5px 5px;max-width:calc(100% - 5px)}.fr-view span.fr-img-caption{position:relative;max-width:100%}.fr-view span.fr-img-caption.fr-dib{margin:5px auto;display:block;float:none;vertical-align:top}.fr-view span.fr-img-caption.fr-dib.fr-fil{margin-left:0;text-align:left}.fr-view span.fr-img-caption.fr-dib.fr-fir{margin-right:0;text-align:right}.fr-view span.fr-img-caption.fr-dii{display:inline-block;float:none;vertical-align:bottom;margin-left:5px;margin-right:5px;max-width:calc(100% - (2 * 5px))}.fr-view span.fr-img-caption.fr-dii.fr-fil{float:left;margin:5px 5px 5px 0;max-width:calc(100% - 5px)}.fr-view span.fr-img-caption.fr-dii.fr-fir{float:right;margin:5px 0 5px 5px;max-width:calc(100% - 5px)}.fr-view .fr-video{text-align:center;position:relative}.fr-view .fr-video>*{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;max-width:100%;border:0}.fr-view .fr-video.fr-dvb{display:block;clear:both}.fr-view .fr-video.fr-dvb.fr-fvl{text-align:left}.fr-view .fr-video.fr-dvb.fr-fvr{text-align:right}.fr-view .fr-video.fr-dvi{display:inline-block}.fr-view .fr-video.fr-dvi.fr-fvl{float:left}.fr-view .fr-video.fr-dvi.fr-fvr{float:right}.fr-view a.fr-strong{font-weight:700}.fr-view a.fr-green{color:green}.fr-view .fr-img-caption{text-align:center}.fr-view .fr-img-caption .fr-img-wrap{padding:0;display:inline-block;margin:auto;text-align:center;width:100%}.fr-view .fr-img-caption .fr-img-wrap img{display:block;margin:auto;width:100%}.fr-view .fr-img-caption .fr-img-wrap>span{margin:auto;display:block;padding:5px 5px 10px;font-size:14px;font-weight:initial;-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;-webkit-opacity:.9;-moz-opacity:.9;opacity:.9;-ms-filter:"alpha(Opacity=0)";width:100%;text-align:center}.fr-view button.fr-rounded,.fr-view input.fr-rounded,.fr-view textarea.fr-rounded{border-radius:10px;-moz-border-radius:10px;-webkit-border-radius:10px;-moz-background-clip:padding;-webkit-background-clip:padding-box;background-clip:padding-box}.fr-view button.fr-large,.fr-view input.fr-large,.fr-view textarea.fr-large{font-size:24px}a.fr-view.fr-strong{font-weight:700}a.fr-view.fr-green{color:green}img.fr-view{position:relative;max-width:100%}img.fr-view.fr-dib{margin:5px auto;display:block;float:none;vertical-align:top}img.fr-view.fr-dib.fr-fil{margin-left:0;text-align:left}img.fr-view.fr-dib.fr-fir{margin-right:0;text-align:right}img.fr-view.fr-dii{display:inline-block;float:none;vertical-align:bottom;margin-left:5px;margin-right:5px;max-width:calc(100% - (2 * 5px))}img.fr-view.fr-dii.fr-fil{float:left;margin:5px 5px 5px 0;max-width:calc(100% - 5px)}img.fr-view.fr-dii.fr-fir{float:right;margin:5px 0 5px 5px;max-width:calc(100% - 5px)}span.fr-img-caption.fr-view{position:relative;max-width:100%}span.fr-img-caption.fr-view.fr-dib{margin:5px auto;display:block;float:none;vertical-align:top}span.fr-img-caption.fr-view.fr-dib.fr-fil{margin-left:0;text-align:left}span.fr-img-caption.fr-view.fr-dib.fr-fir{margin-right:0;text-align:right}span.fr-img-caption.fr-view.fr-dii{display:inline-block;float:none;vertical-align:bottom;margin-left:5px;margin-right:5px;max-width:calc(100% - (2 * 5px))}span.fr-img-caption.fr-view.fr-dii.fr-fil{float:left;margin:5px 5px 5px 0;max-width:calc(100% - 5px)}span.fr-img-caption.fr-view.fr-dii.fr-fir{float:right;margin:5px 0 5px 5px;max-width:calc(100% - 5px)}</style><style class="darkreader darkreader--sync" media="screen"></style>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/modernizer.custom.min.js"></script><meta name="darkreader" content="027bcaa447244060ad7f516eb8e5da24"><style class="darkreader darkreader--override" media="screen">.vimvixen-hint {
    background-color: #7c5c08 !important;
    border-color: #c6a629 !important;
    color: #dcd3b9 !important;
}
#vimvixen-console-frame {
    color-scheme: light !important;
}
::placeholder {
    opacity: 0.5 !important;
}
#edge-translate-panel-body,
.MuiTypography-body1,
.nfe-quote-text {
    color: var(--darkreader-neutral-text) !important;
}
gr-main-header {
    background-color: #264853 !important;
}
.tou-z65h9k,
.tou-mignzq,
.tou-1b6i2ox,
.tou-lnqlqk {
    background-color: var(--darkreader-neutral-background) !important;
}
.tou-75mvi {
    background-color: #1c333a !important;
}
.tou-ta9e87,
.tou-1w3fhi0,
.tou-1b8t2us,
.tou-py7lfi,
.tou-1lpmd9d,
.tou-1frrtv8,
.tou-17ezmgn {
    background-color: #212121 !important;
}
.tou-uknfeu {
    background-color: #352b1c !important;
}
.tou-6i3zyv {
    background-color: #2e5f70 !important;
}
div.mermaid-viewer-control-panel .btn {
    background-color: var(--darkreader-neutral-background);
    fill: var(--darkreader-neutral-text);
}
svg g rect.er {
    fill: var(--darkreader-neutral-background) !important;
}
svg g rect.er.entityBox {
    fill: var(--darkreader-neutral-background) !important;
}
svg g rect.er.attributeBoxOdd {
    fill: var(--darkreader-neutral-background) !important;
}
svg g rect.er.attributeBoxEven {
    fill: var(--darkreader-selection-background);
    fill-opacity: 0.8 !important;
}
svg rect.er.relationshipLabelBox {
    fill: var(--darkreader-neutral-background) !important;
}
svg g g.nodes rect,
svg g g.nodes polygon {
    fill: var(--darkreader-neutral-background) !important;
}
svg g rect.task {
    fill: var(--darkreader-selection-background) !important;
}
svg line.messageLine0,
svg line.messageLine1 {
    stroke: var(--darkreader-neutral-text) !important;
}
div.mermaid .actor {
    fill: var(--darkreader-neutral-background) !important;
}
.google-material-icons {
    font-family: 'Google Material Icons' !important;
}
.google-symbols {
    font-family: 'Google Symbols Subset', 'Google Symbols' !important;
}
.material-icons-extended {
    font-family: 'Material Icons Extended' !important;
}
mitid-authenticators-code-app > .code-app-container {
    background-color: white !important;
    padding-top: 1rem;
}
iframe#unpaywall[src$="unpaywall.html"] {
    color-scheme: light !important;
}
.oui-icon {
    font-family: 'Oui Icons' !important;
}
embed[type="application/pdf"][src="about:blank"] { filter: invert(100%) contrast(90%); }</style>
<!-- HTML5 Respond.js IE8 support of media queries -->
<!--[if lt IE 9]>
<script src="//oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
<![endif]-->
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/jquery-1.11.1.min.js"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/jquery.dlmenu.js"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/f.txt" async=""></script>
<script id="altmetric-embed-js" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/altmetric_badges-e7ba41a192efc3e63f5d48f8a1916dec4b1b49c01075bc7b39481408957b3b28.js"></script><script type="text/javascript" async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/MathJax.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style class="darkreader darkreader--sync" media="screen"></style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style class="darkreader darkreader--sync" media="screen"></style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style class="darkreader darkreader--sync" media="screen"></style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style class="darkreader darkreader--sync" media="screen"></style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style class="darkreader darkreader--sync" media="screen"></style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style class="darkreader darkreader--sync" media="screen"></style><style id="shr-common">.shr-hide{display:none !important}@media print{.shr-no-print{display:none !important}}body.shr-ios-modal{position:fixed}@-moz-keyframes shareaholic-status-spin-once{0%{transform:rotate(0deg)}100%{transform:rotate(360deg)}}@-webkit-keyframes shareaholic-status-spin-once{0%{transform:rotate(0deg)}100%{transform:rotate(360deg)}}@-o-keyframes shareaholic-status-spin-once{0%{transform:rotate(0deg)}100%{transform:rotate(360deg)}}@keyframes shareaholic-status-spin-once{0%{transform:rotate(0deg)}100%{transform:rotate(360deg)}}</style><style class="darkreader darkreader--sync" media="screen"></style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><style class="darkreader darkreader--sync" media="screen"></style><style id="shr-outline-style" type="text/css">.shr-outline{outline:none}</style><style class="darkreader darkreader--sync" media="screen"></style><link rel="stylesheet" href="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/badge.css"><style class="darkreader darkreader--cors" media="screen">.__dimensions_badge_embed__{display:block}.__dimensions_badge_embed__ html,.__dimensions_badge_embed__ body,.__dimensions_badge_embed__ div,.__dimensions_badge_embed__ span,.__dimensions_badge_embed__ applet,.__dimensions_badge_embed__ object,.__dimensions_badge_embed__ iframe,.__dimensions_badge_embed__ h1,.__dimensions_badge_embed__ h2,.__dimensions_badge_embed__ h3,.__dimensions_badge_embed__ h4,.__dimensions_badge_embed__ h5,.__dimensions_badge_embed__ h6,.__dimensions_badge_embed__ p,.__dimensions_badge_embed__ blockquote,.__dimensions_badge_embed__ pre,.__dimensions_badge_embed__ a,.__dimensions_badge_embed__ abbr,.__dimensions_badge_embed__ acronym,.__dimensions_badge_embed__ address,.__dimensions_badge_embed__ big,.__dimensions_badge_embed__ cite,.__dimensions_badge_embed__ code,.__dimensions_badge_embed__ del,.__dimensions_badge_embed__ dfn,.__dimensions_badge_embed__ em,.__dimensions_badge_embed__ img,.__dimensions_badge_embed__ ins,.__dimensions_badge_embed__ kbd,.__dimensions_badge_embed__ q,.__dimensions_badge_embed__ s,.__dimensions_badge_embed__ samp,.__dimensions_badge_embed__ small,.__dimensions_badge_embed__ strike,.__dimensions_badge_embed__ strong,.__dimensions_badge_embed__ sub,.__dimensions_badge_embed__ sup,.__dimensions_badge_embed__ tt,.__dimensions_badge_embed__ var,.__dimensions_badge_embed__ b,.__dimensions_badge_embed__ u,.__dimensions_badge_embed__ i,.__dimensions_badge_embed__ center,.__dimensions_badge_embed__ dl,.__dimensions_badge_embed__ dt,.__dimensions_badge_embed__ dd,.__dimensions_badge_embed__ ol,.__dimensions_badge_embed__ ul,.__dimensions_badge_embed__ li,.__dimensions_badge_embed__ fieldset,.__dimensions_badge_embed__ form,.__dimensions_badge_embed__ label,.__dimensions_badge_embed__ legend,.__dimensions_badge_embed__ table,.__dimensions_badge_embed__ caption,.__dimensions_badge_embed__ tbody,.__dimensions_badge_embed__ tfoot,.__dimensions_badge_embed__ thead,.__dimensions_badge_embed__ tr,.__dimensions_badge_embed__ th,.__dimensions_badge_embed__ td,.__dimensions_badge_embed__ article,.__dimensions_badge_embed__ aside,.__dimensions_badge_embed__ canvas,.__dimensions_badge_embed__ details,.__dimensions_badge_embed__ embed,.__dimensions_badge_embed__ figure,.__dimensions_badge_embed__ figcaption,.__dimensions_badge_embed__ footer,.__dimensions_badge_embed__ header,.__dimensions_badge_embed__ hgroup,.__dimensions_badge_embed__ menu,.__dimensions_badge_embed__ nav,.__dimensions_badge_embed__ output,.__dimensions_badge_embed__ ruby,.__dimensions_badge_embed__ section,.__dimensions_badge_embed__ summary,.__dimensions_badge_embed__ time,.__dimensions_badge_embed__ mark,.__dimensions_badge_embed__ audio,.__dimensions_badge_embed__ video{margin:0;padding:0;border:0;font-size:100%;font:inherit;vertical-align:baseline}.__dimensions_badge_embed__ article,.__dimensions_badge_embed__ aside,.__dimensions_badge_embed__ details,.__dimensions_badge_embed__ figcaption,.__dimensions_badge_embed__ figure,.__dimensions_badge_embed__ footer,.__dimensions_badge_embed__ header,.__dimensions_badge_embed__ hgroup,.__dimensions_badge_embed__ menu,.__dimensions_badge_embed__ nav,.__dimensions_badge_embed__ section{display:block}.__dimensions_badge_embed__ body{line-height:1}.__dimensions_badge_embed__ ol,.__dimensions_badge_embed__ ul{list-style:none}.__dimensions_badge_embed__ blockquote,.__dimensions_badge_embed__ q{quotes:none}.__dimensions_badge_embed__ blockquote:before,.__dimensions_badge_embed__ blockquote:after,.__dimensions_badge_embed__ q:before,.__dimensions_badge_embed__ q:after{content:'';content:none}.__dimensions_badge_embed__ table{border-collapse:collapse;border-spacing:0}.__dimensions_badge_embed__,.__dimensions_badge_embed__ a,.__dimensions_badge_embed__ a:hover,.__dimensions_badge_embed__ div{background:transparent;border:none;box-shadow:none;color:#222;font-family:'dimensions-badge', 'Nelvetica Neue', Helvetica, Arial, sans-serif;font-kerning:normal;font-size:13px;font-style:normal;font-weight:normal;height:auto;line-height:18px;margin:0;min-height:auto;min-width:auto;opacity:1;outline:none;overflow:visible;padding:0;text-align:left;text-decoration:none;width:auto;vertical-align:middle}.__dimensions_badge_embed__ .__dimensions_png{position:absolute;top:0;left:0;bottom:0;right:0;opacity:0;max-width:100%}.__dimensions_badge_embed__::after{display:table;clear:both;content:''}.__dimensions_badge_embed__ .__dimensions_Link{display:inline-block !important;position:relative}.__dimensions_badge_embed__ .__dimensions_Link.__dimensions_with_legend{padding:10px}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_padding{display:none;padding:15px}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_padding.__dimensions_Badge_Legend_always{display:block;margin-left:10px;padding:0 10px;float:left;background:white}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_hover{display:block;position:absolute;z-index:1000000}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_hover .__dimensions_Badge_Legend{position:relative;width:250px;padding:0 10px 5px 10px;border:4px #444 solid;border-radius:5px;background:#f4f4f4;box-shadow:1px 1px 20px rgba(0,0,0,0.25)}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_hover.__dimensions_Badge_Legend_hover-top{bottom:100%;left:50%;-ms-transform:translateX(-50%);transform:translateX(-50%)}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_hover.__dimensions_Badge_Legend_hover-top .__dimensions_Badge_Legend::before{position:absolute;width:0;height:0;border-style:solid;content:'';bottom:-15px;left:50%;margin-left:-11.25px;border-width:15px 10px 0 10px;border-color:#444 transparent transparent transparent}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_hover.__dimensions_Badge_Legend_hover-right{top:50%;left:100%;-ms-transform:translateY(-50%);transform:translateY(-50%)}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_hover.__dimensions_Badge_Legend_hover-right .__dimensions_Badge_Legend::before{position:absolute;width:0;height:0;border-style:solid;content:'';top:50%;left:-15px;margin-top:-11.25px;border-width:10px 15px 10px 0;border-color:transparent #444 transparent transparent}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_hover.__dimensions_Badge_Legend_hover-bottom{top:100%;left:50%;-ms-transform:translateX(-50%);transform:translateX(-50%)}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_hover.__dimensions_Badge_Legend_hover-bottom .__dimensions_Badge_Legend::before{position:absolute;width:0;height:0;border-style:solid;content:'';top:-15px;left:50%;margin-left:-11.25px;border-width:0 10px 15px 10px;border-color:transparent transparent #444 transparent}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_hover.__dimensions_Badge_Legend_hover-left{top:50%;right:100%;-ms-transform:translateY(-50%);transform:translateY(-50%)}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_hover.__dimensions_Badge_Legend_hover-left .__dimensions_Badge_Legend::before{position:absolute;width:0;height:0;border-style:solid;content:'';top:50%;right:-15px;margin-top:-11.25px;border-width:10px 0 10px 15px;border-color:transparent transparent transparent #444}.__dimensions_badge_embed__ .__dimensions_Badge_stat_group_citations{padding:2.5px 0 5px;border-bottom:1px #ccc dotted}.__dimensions_badge_embed__ .__dimensions_Badge_stat_group_cr{padding:5px 0 2.5px}.__dimensions_badge_embed__ .__dimensions_Badge_stat_icon{display:block;position:relative;top:1px;width:20px;height:20px;float:left;clear:left;background-repeat:no-repeat;background-position:center center;background-size:16px}.__dimensions_badge_embed__ .__dimensions_Badge_stat_total_citations .__dimensions_Badge_stat_icon{background-image:url('data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSItMjUgLTE1IDEyNSAxMTEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CiAgPHBhdGggZD0iTTk5LjQ5OCA5LjI0M2MwLTQuNjY4LTMuNzg4LTguNDUtOC40NjUtOC40NUg2My41MmMtNC42NzMgMC04LjQ2NyAzLjc4Mi04LjQ2NyA4LjQ1djMzLjc5NWMwIDQuNjY4IDMuNzk0IDguNDU1IDguNDY3IDguNDU1aDEzLjU5YTQuMjE5IDQuMjE5IDAgMCAxIDQuMjMgNC4yMThjMCAuNzY2LS4xODcgMS4yODgtLjE4NyAxLjI4OC0zLjA5IDEwLjIyOC0xMC45IDE4LjQxOC0yMC45MSAyMi4wNTNsLS4wNDUuMDQ1Yy0zLjAyOSAxLjI5LTUuMTQ1IDQuMjg1LTUuMTQ1IDcuNzY5IDAgNC42NyAzLjc5NCA4LjQ1NSA4LjQ2NyA4LjQ1NSAxLjE0NyAwIDIuMjMtLjIzNCAzLjIxNS0uNjM4bC4zOS0uMTQ1YzE4LjkzOS03LjM3NCAzMi4zNzQtMjUuNzQ4IDMyLjM3NC00Ny4yNzNWOS4yNDN6bS01NS4wMjMgMGMwLTQuNjY4LTMuNzg4LTguNDUtOC40NjctOC40NUg4LjQ5NkMzLjgyLjc5My4wMjYgNC41NzUuMDI2IDkuMjQzdjMzLjc5NWMwIDQuNjY4IDMuNzkzIDguNDU1IDguNDcgOC40NTVoMTMuNTg3YTQuMjIgNC4yMiAwIDAgMSA0LjIzIDQuMjE4YzAgLjc4My0uMTg5IDEuMjg4LS4xODkgMS4yODgtMy4wOSAxMC4yMjgtMTAuODk1IDE4LjQxOC0yMC45MSAyMi4wNTNsLS4wMzguMDQ1Yy0zLjAzIDEuMjktNS4xNSA0LjI4NS01LjE1IDcuNzY5IDAgNC42NyAzLjc5MyA4LjQ1NSA4LjQ3IDguNDU1YTguNDggOC40OCAwIDAgMCAzLjIxLS42MzhsLjM4Ni0uMTQ1YzE4Ljk0My03LjM3NCAzMi4zOC0yNS43NDggMzIuMzgtNDcuMjczVjkuMjQzaC4wMDN6IiBmaWxsPSIjMzMzIi8+Cjwvc3ZnPgo=')}.__dimensions_badge_embed__ .__dimensions_Badge_stat_recent_citations .__dimensions_Badge_stat_icon{background-image:url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAzNSAzMCI+CiAgPHBhdGggZmlsbD0nIzMzMycgZD0iTTIwIDJjNy43MzIgMCAxNCA2LjI2OCAxNCAxNHMtNi4yNjggMTQtMTQgMTR2LTNjMi45MzggMCA1LjcwMS0xLjE0NCA3Ljc3OC0zLjIyMnMzLjIyMi00Ljg0IDMuMjIyLTcuNzc4YzAtMi45MzgtMS4xNDQtNS43MDEtMy4yMjItNy43NzhzLTQuODQtMy4yMjItNy43NzgtMy4yMjJjLTIuOTM4IDAtNS43MDEgMS4xNDQtNy43NzggMy4yMjItMS41OTggMS41OTgtMi42NDMgMy42MDEtMy4wNDEgNS43NzhoNS44MTlsLTcgOC03LThoNS4xNDNjMC45NzEtNi43ODQgNi44MDQtMTIgMTMuODU3LTEyek0yNiAxNHY0aC04di0xMGg0djZ6Ii8+Cjwvc3ZnPgo=')}.__dimensions_badge_embed__ .__dimensions_Badge_stat_rcr .__dimensions_Badge_stat_icon{background-image:url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyOSAzMiI+PHBhdGggZmlsbD0iIzMzMyIgZD0iTTI4LjQxNiAxOC4yNzJhMy4yMjMgMy4yMjMgMCAwIDEtMy4yMzIgMy4yMzJjLTEuNzYgMC0zLjItMS40NC0zLjItMy4yMzJ2LS4wNjRsLTIuNC0uNDhhNS4zNjcgNS4zNjcgMCAwIDEtMi4yNCAyLjkxMmwxLjUzNiAzLjE2OGMuMzItLjA5Ni42NC0uMTYuOTkyLS4xNmEzLjIyMyAzLjIyMyAwIDAgMSAzLjIzMiAzLjIzMmMwIDEuNzYtMS40NCAzLjItMy4yMzIgMy4yYTMuMjEgMy4yMSAwIDAgMS0zLjItMy4yYzAtMS4wODguNTEyLTIuMDE2IDEuMzQ0LTIuNjI0bC0xLjU2OC0zLjE2OGMtLjYwOC4yMjQtMS4yOC4zODQtMS45NTIuMzg0LTEuNDcyIDAtMi43ODQtLjYwOC0zLjc3Ni0xLjU2OGwtNC4wNjQgMy4yYy4yNTYuNDguNDQ4IDEuMDI0LjQ0OCAxLjYzMiAwIDEuNzYtMS40NCAzLjItMy4yMzIgMy4yYTMuMjEgMy4yMSAwIDAgMS0zLjItMy4yYzAtMS43OTIgMS40NC0zLjIzMiAzLjItMy4yMzJhMy4yIDMuMiAwIDAgMSAyLjE0NC44MzJsNC4wNjQtMy4yYTUuMzE1IDUuMzE1IDAgMCAxLS45MjgtMy4wMDhjMC0xLjI0OC40NDgtMi40MzIgMS4xODQtMy4zMjhMNS43MjggOC4xOTJhMy4yNDggMy4yNDggMCAwIDEtMS44MjQuNTc2Yy0xLjc2IDAtMy4yLTEuNDQtMy4yLTMuMjMyczEuNDQtMy4yIDMuMi0zLjJjMS43NiAwIDMuMiAxLjQ0IDMuMiAzLjIgMCAuNzM2LS4yMjQgMS40MDgtLjY0IDEuOTUybDQuNTQ0IDQuNjA4Yy45MjgtLjgzMiAyLjE0NC0xLjMxMiAzLjQ4OC0xLjMxMi41MTIgMCAuOTkyLjA5NiAxLjQ3Mi4yMjRsMS4xMi0yLjY1NmEzLjIgMy4yIDAgMCAxLTEuNi0yLjc4NGMwLTEuNzYgMS40NC0zLjIgMy4yMzItMy4yIDEuNzYgMCAzLjIgMS40NCAzLjIgMy4yIDAgMS43OTItMS40NCAzLjIzMi0zLjIgMy4yMzItLjI1NiAwLS40OC0uMDMyLS43MDQtLjA5NmwtMS4xMiAyLjY1NmE1LjM2NSA1LjM2NSAwIDAgMSAyLjkxMiA0Ljc2OGMwIC4yMjQgMCAuNDE2LS4wMzIuNjA4bDIuNC41MTJhMy4xNDIgMy4xNDIgMCAwIDEgMy4wMDgtMi4xNzZjMS43OTIgMCAzLjIzMiAxLjQ0IDMuMjMyIDMuMnoiLz48L3N2Zz4K')}.__dimensions_badge_embed__ .__dimensions_Badge_stat_fcr .__dimensions_Badge_stat_icon{background-image:url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyOSAzMiI+PGcgZmlsbD0iIzMzMyI+PGcgb3BhY2l0eT0iLjUiPjxwYXRoIGQ9Ik0xOCAxOS45bDQuNSA0LjZjLS40LjUtLjYgMS4yLS42IDIgMCAxLjggMS40IDMuMiAzLjIgMy4yczMuMi0xLjQgMy4yLTMuMi0xLjQtMy4yLTMuMi0zLjJjLS43IDAtMS4zLjItMS44LjZsLTQuNi00LjZjLS4yLjItLjUuNC0uNy42ek0xMi4xIDIwLjZMMTEgMjMuM2MtLjItLjEtLjQtLjEtLjctLjEtMS44IDAtMy4yIDEuNC0zLjIgMy4yIDAgMS44IDEuNCAzLjIgMy4yIDMuMiAxLjggMCAzLjItMS40IDMuMi0zLjIgMC0xLjEtLjYtMi4yLTEuNi0yLjhMMTMgMjFjLS40LS4xLS43LS4yLTEtLjRoLjF6TTMuOCAxNi45YzEuNCAwIDIuNi0uOSAzLTIuMmwyLjQuNWMwLS4zLjEtLjcuMi0xTDcgMTMuN3YtLjFjMC0xLjgtMS40LTMuMi0zLjItMy4yQzIgMTAuNC42IDExLjkuNiAxMy42YzAgMS45IDEuNCAzLjMgMy4yIDMuM3pNMTEgNy43Yy44LS42IDEuMy0xLjUgMS4zLTIuNiAwLTEuOC0xLjQtMy4yLTMuMi0zLjItMS44IDAtMy4yIDEuNC0zLjIgMy4yIDAgMS44IDEuNSAzLjIgMy4yIDMuMi40IDAgLjctLjEgMS0uMmwxLjUgMy4yYy4zLS4yLjYtLjMuOS0uNUwxMSA3Ljd6TTIzIDkuN2MuNi41IDEuNC44IDIuMS44IDEuOCAwIDMuMi0xLjQgMy4yLTMuMiAwLTEuOC0xLjQtMy4yLTMuMi0zLjItMS44IDAtMy4yIDEuNC0zLjIgMy4yIDAgLjYuMiAxLjIuNCAxLjZsLTQgMy4yYy4yLjIuNC41LjYuOEwyMyA5Ljd6Ii8+PC9nPjxwYXRoIGQ9Ik05LjIgMTUuM3YuNmMwIDIgMS4xIDMuOCAyLjggNC43LjMuMi43LjMgMSAuNC4yLjEuNS4xLjcuMi4yIDAgLjUuMS43LjEuNiAwIDEuMi0uMSAxLjctLjMuMSAwIC4yIDAgLjMtLjEuMi0uMS41LS4yLjctLjMuMSAwIC4xLS4xLjItLjEuMy0uMi41LS4zLjctLjUuMi0uMi41LS40LjctLjcuNy0uOSAxLjItMi4xIDEuMi0zLjMgMC0uMyAwLS41LS4xLS44IDAtLjItLjEtLjUtLjItLjd2LS4xYy0uMi0uNS0uNC0xLS43LTEuNC0uMi0uMy0uNC0uNS0uNi0uOC0uOC0uOC0xLjktMS40LTMuMS0xLjVoLTEuMS0uMWMtLjUuMS0xIC4yLTEuNC40LS4zLjEtLjYuMy0uOS41LTEuMS43LTEuOSAxLjctMi4yIDIuOS0uMi4xLS4zLjQtLjMuOHoiLz48L2c+PC9zdmc+')}.__dimensions_badge_embed__ .__dimensions_Badge_stat_count{display:inline-block;min-width:50px;padding:0 5px;font-weight:bold;text-align:center}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_style_small_circle{padding:18.75px}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_style_medium_circle{padding:22.5px}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_style_large_circle{padding:30px}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_style_small_rectangle.__dimensions_Badge_Legend_always{margin-top:-7px}.__dimensions_badge_embed__ .__dimensions_Badge_Legend_style_large_rectangle.__dimensions_Badge_Legend_always{margin-top:-7px}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle{float:left}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__dimensions_Badge_Image,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle .__dimensions_Badge_Image,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle .__dimensions_Badge_Image{position:relative}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__db_background,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle .__db_background,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle .__db_background{position:absolute;top:0;right:0;bottom:0;left:0;transition:box-shadow 500ms cubic-bezier(0.77, 0, 0.175, 1);border-radius:100%;background:#fff;box-shadow:0 0 10px rgba(0,0,0,0.2);will-change:transform, opacity}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__db_shape,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle .__db_shape,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle .__db_shape{position:absolute;top:-50%;right:-50%;bottom:-50%;left:-50%;-ms-transform:rotate(0) scale(0.55);transform:rotate(0) scale(0.55);transform-origin:center center;transition:opacity 500ms cubic-bezier(0.77, 0, 0.175, 1),transform 500ms cubic-bezier(0.77, 0, 0.175, 1);opacity:1;will-change:transform, opacity}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__db_shape svg,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle .__db_shape svg,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle .__db_shape svg{width:100%}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__db_score,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle .__db_score,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle .__db_score{position:absolute;top:50%;right:0;left:0;-ms-transform:translateY(-50%) scale(0.5);transform:translate3d(0, -50%, 0) scale(0.5);transition:opacity 500ms cubic-bezier(0.77, 0, 0.175, 1),transform 500ms cubic-bezier(0.77, 0, 0.175, 1);font-family:'dimensions-badge', Helvetica, Arial, sans-serif;font-size:45px;text-align:center;will-change:transform, opacity}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__db_score.__db_score_small,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle .__db_score.__db_score_small,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle .__db_score.__db_score_small{font-size:35px}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__db_label,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle .__db_label,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle .__db_label{position:absolute;top:75%;right:0;bottom:0;left:0;-ms-transform:translateY(-75%);transform:translate3d(0, -75%, 0);transition:opacity 250ms cubic-bezier(0.77, 0, 0.175, 1),transform 250ms cubic-bezier(0.77, 0, 0.175, 1);transition-delay:0;font-size:12px;text-align:center;opacity:0;will-change:transform, opacity}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle.__dimensions_Badge_hover .__db_background,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle.__dimensions_Badge_hover .__db_background,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle.__dimensions_Badge_hover .__db_background{box-shadow:0 0 20px rgba(0,0,0,0.2)}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle.__dimensions_Badge_hover .__db_shape,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle.__dimensions_Badge_hover .__db_shape,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle.__dimensions_Badge_hover .__db_shape{-ms-transform:rotate(360deg) scale(0.7);transform:rotate(360deg) scale(0.7);opacity:1}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle.__dimensions_Badge_hover .__db_score,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle.__dimensions_Badge_hover .__db_score,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle.__dimensions_Badge_hover .__db_score{-ms-transform:translateY(-100%) scale(0.7);transform:translate3d(0, -100%, 0) scale(0.7)}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle.__dimensions_Badge_hover .__db_label,.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle.__dimensions_Badge_hover .__db_label,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle.__dimensions_Badge_hover .__db_label{-ms-transform:translateY(-100%);transform:translate3d(0, -100%, 0);transition-delay:250ms;opacity:1}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__dimensions_Badge_Image{width:64px;height:64px}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__db_score{font-size:30px}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__db_score.__db_score_small{font-size:25px}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__db_label{display:none}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle.__dimensions_Badge_hover .__db_score{-ms-transform:translateY(-50%) scale(0.7);transform:translate3d(0, -50%, 0) scale(0.7)}.__dimensions_badge_embed__ .__dimensions_Badge_style_medium_circle .__dimensions_Badge_Image{width:120px;height:120px}.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle .__dimensions_Badge_Image{width:180px;height:180px}.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle .__db_score{font-size:55px}.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle .__db_score.__db_score_small{font-size:50px}.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle .__db_label{font-size:16px}.__dimensions_badge_embed__ .__dimensions_Badge_style_large_circle.__dimensions_Badge_hover .__db_label{-ms-transform:translateY(-90%);transform:translate3d(0, -90%, 0)}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_rectangle,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_rectangle{float:left}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_rectangle .__dimensions_Badge_Image,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_rectangle .__dimensions_Badge_Image{position:relative}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_rectangle .__db_score,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_rectangle .__db_score{position:absolute;top:0;right:1px;bottom:0;left:66%;font-family:'dimensions-badge', Helvetica, Arial, sans-serif;font-size:12px;text-align:center}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_rectangle .__db_score.__db_score_small,.__dimensions_badge_embed__ .__dimensions_Badge_style_large_rectangle .__db_score.__db_score_small{font-size:11px}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_rectangle .__dimensions_Badge_Image{width:88px;height:17.6px}.__dimensions_badge_embed__ .__dimensions_Badge_style_small_rectangle .__db_score{font-size:12px;line-height:17.6px}.__dimensions_badge_embed__ .__dimensions_Badge_style_large_rectangle .__dimensions_Badge_Image{width:110px;height:22px}.__dimensions_badge_embed__ .__dimensions_Badge_style_large_rectangle .__db_score{font-size:14px;line-height:22px}</style><style class="darkreader darkreader--sync" media="screen"></style><link rel="stylesheet" type="text/css" href="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/trendmd.min.css" media="all"><style class="darkreader darkreader--cors" media="screen">.trendmd-widget{border:0!important;font:13px/18px Arial,Helvetica CY,Nimbus Sans L,sans-serif!important;margin:0!important;outline:0!important;padding:0!important}.trendmd-widget-brand,.trendmd-widget-brand__logo,.trendmd-widget-container,.trendmd-widget-footer,.trendmd-widget-header,.trendmd-widget-header__heading,.trendmd-widget-list,.trendmd-widget-list-item,.trendmd-widget-list-item__link,.trendmd-widget-list-item__mendeley,.trendmd-widget-list-item__mendeley-img,.trendmd-widget-list-item__meta{background:none!important;border:0!important;font-size:100%!important;font:inherit!important;margin:0!important;outline:0!important;padding:0!important;vertical-align:baseline!important}.trendmd-widget-brand:after,.trendmd-widget-brand:before,.trendmd-widget-brand__logo:after,.trendmd-widget-brand__logo:before,.trendmd-widget-footer:after,.trendmd-widget-footer:before,.trendmd-widget-header:after,.trendmd-widget-header:before,.trendmd-widget-header__heading:after,.trendmd-widget-header__heading:before,.trendmd-widget-list-item:after,.trendmd-widget-list-item:before,.trendmd-widget-list-item__link:after,.trendmd-widget-list-item__link:before,.trendmd-widget-list-item__mendeley-img:after,.trendmd-widget-list-item__mendeley-img:before,.trendmd-widget-list-item__mendeley:after,.trendmd-widget-list-item__mendeley:before,.trendmd-widget-list-item__meta:after,.trendmd-widget-list-item__meta:before,.trendmd-widget-list:after,.trendmd-widget-list:before{display:none!important}.trendmd-widget{color:#000!important}.trendmd-widget-container:after,.trendmd-widget-container:before{clear:both!important;content:""!important;display:table!important}.trendmd-widget-inner:after,.trendmd-widget-inner:before{content:""!important;display:table!important}.trendmd-widget-inner:after{clear:both!important}.trendmd-widget-header{font-size:14px!important;margin:0 0 20px!important;padding:8px 0 0 4px!important}.trendmd-widget-header h3{padding:0!important}.trendmd-widget-header__heading{font-size:1em!important;font-weight:700!important;line-height:18px!important;text-align:left!important}.trendmd-widget-section:first-child{border:none!important;margin:0!important;padding-left:0!important}.trendmd-widget-section:nth-child(2) .trendmd-widget-header{display:none}.trendmd-widget-list{list-style:none!important;padding:4px 0 0 4px!important}.trendmd-widget-list-item{list-style-type:none!important;margin:0 0 12px!important;position:relative!important;text-align:left!important}.trendmd-widget-list-item__link{color:#000!important;text-decoration:none!important}.trendmd-widget-list-item__link:hover{text-decoration:underline!important}.trendmd-widget-list-item__link:hover span{opacity:1!important}.trendmd-widget-list-item__link__external{background:url('https://css.trendmd.com/external.png') no-repeat 0 0!important;display:inline-block!important;height:11px!important;margin:0 0 0 5px!important;opacity:.25!important;vertical-align:baseline!important;width:11px!important}.trendmd-widget-list-item__meta{color:#a2b1b6!important;margin:2px 0 0!important}.trendmd-widget-list-item__mendeley-img{height:16px!important;margin-left:5px!important;vertical-align:text-bottom!important;width:16px!important}.trendmd-widget-footer{border-top:2px solid #e9eeef!important;color:#cad5d9!important;font-family:SF Pro Display,Arial,Helvetica CY,sans-serif!important;font-size:14px!important;margin:13px 0 0!important;padding:15px 0 0!important}.trendmd-widget-footer-inner{font-size:1em!important}.trendmd-widget-footer-inner-left{float:left!important;text-align:left!important;width:50%!important}.trendmd-widget-footer-inner-right{float:right!important;text-align:right!important;width:50%!important}.trendmd-widget-footer-inner-clear{clear:both!important}.trendmd-widget-footer__login-text{color:#16325c;font-size:14px!important;padding-right:15px}.trendmd-widget-footer__login-link{color:#16325c;text-decoration:underline!important}.trendmd-widget-brand{color:#cad5d9!important;font-size:inherit!important;font-weight:400!important;line-height:16px!important;text-decoration:none!important}.trendmd-widget-brand:hover{background:none!important}.trendmd-widget-brand__recommended{display:inline-block!important;float:left!important;line-height:22px!important}.trendmd-widget-brand__logo{display:inline-block!important;font-size:0!important;height:22px!important;margin:0 0 0 5px!important;vertical-align:8px!important;width:93px!important}.trendmd-widget-brand__logo__faded #group1,.trendmd-widget-brand__logo__faded #shape0{fill:#a3afc5!important}.trendmd-widget-settings{cursor:pointer!important;float:right!important;position:relative!important}.trendmd-widget-settings__cog{background:url('https://css.trendmd.com/settings.png') no-repeat!important;background-size:cover!important;display:block!important;height:16px!important;margin-top:2px!important;width:16px!important}.trendmd-widget-settings__modal{bottom:100%!important;display:none!important;margin-bottom:-15px!important;padding-bottom:20px!important;position:absolute!important;right:0!important}.trendmd-widget-settings__list{background-color:#fff!important;border:1px solid #d8dde6!important;border-bottom:transparent!important;border-radius:3px!important;box-shadow:0 4px 8px 0 rgba(0,0,0,.15)!important;margin:0!important;padding:6px 0!important}.trendmd-widget-settings__item{color:#16325c!important;line-height:20px!important;list-style-type:none!important;text-align:left!important;white-space:nowrap!important}.trendmd-widget-settings__link{color:#16325c!important;display:block;padding:10px 20px!important;text-decoration:none!important;transition:.2s ease-in-out}.trendmd-widget-settings__link:hover{color:#196eed!important}.trendmd-widget-settings:hover .trendmd-widget-settings__modal{display:block!important}.trendmd-widget-cookie-notification{background-color:#eef2f8!important;box-sizing:border-box!important;color:#16325c!important;display:none!important;float:left!important;font-size:14px!important;font-weight:400!important;margin-top:10px!important;padding:5px 10px 10px!important;width:100%!important}.trendmd-widget-cookie-notification__info{float:left!important;line-height:20px!important;margin-bottom:10px!important}.trendmd-widget-cookie-notification__info a{color:#16325c!important;text-decoration:underline!important}@media (max-width:224px){.trendmd-widget-cookie-notification__info{margin-bottom:30px!important}}.trendmd-widget-cookie-notification__actions{float:right!important;width:100%!important}.trendmd-widget-cookie-notification__actions button{background-color:#fff!important;border:1px solid #b8c4d7!important;border-radius:3px!important;color:#16325c!important;cursor:pointer!important;font-weight:700!important;height:32px!important;line-height:26px!important;margin:0 15px 0 0!important;padding:0!important;width:79px!important}@media (max-width:319px){.trendmd-widget-cookie-notification__actions button{margin:0 5px 0 0!important}}@media (max-width:224px){.trendmd-widget-cookie-notification__actions button{width:60px!important}}.trendmd-widget-cookie-notification__actions span{display:inline-block!important;text-align:left!important;width:50px!important}.trendmd-widget-addto{color:#cad5d9!important;font-size:inherit!important;font-weight:400!important;text-decoration:none!important}.trendmd-widget-addto:hover{background:none!important}.trendmd-widget-addto__plus{background:url('https://css.trendmd.com/plus.png') no-repeat 0 0!important;display:inline-block!important;height:14px!important;margin:0 5px 0 0!important;vertical-align:-2px!important;width:14px!important}.trendmd-widget_horizontal{padding-top:42px!important;position:relative!important}.trendmd-widget_horizontal .trendmd-widget-header{margin-top:-42px!important}.trendmd-widget_horizontal .trendmd-widget-header+.trendmd-widget-list{padding-top:0!important}.trendmd-widget_horizontal .trendmd-widget-section{border:none!important;float:left!important;margin:0 0 0 2%!important;padding:7px 0 0 2%!important;width:45%!important}.trendmd-widget_horizontal .trendmd-widget-section:first-child{margin:0!important;padding-left:0!important}.trendmd-widget_horizontal .trendmd-widget-section:first-child:before{display:none!important}@media (max-width:450px){.trendmd-widget_horizontal .trendmd-widget-section{float:none!important;margin:0!important;padding:0!important;width:100%!important}}.trendmd-widget_vertical .trendmd-widget-footer-inner{position:relative!important}.trendmd-widget_vertical .trendmd-widget-footer-inner-left{width:80%!important}.trendmd-widget_vertical .trendmd-widget-footer-inner-right{margin-bottom:15px!important;text-align:left!important;width:100%!important}.trendmd-widget_vertical .trendmd-widget-settings{position:absolute!important;right:5px;top:5px}.trendmd-widget:hover .trendmd-widget-brand{color:#000!important}.trendmd-widget:hover .trendmd-widget-brand__logo #shape0{fill:#ca2127!important}.trendmd-widget:hover .trendmd-widget-brand__logo #group1{fill:#16325c!important}.trendmd-widget:hover .trendmd-widget-addto{color:#000!important}.trendmd-widget:hover .trendmd-widget-addto__plus{background-position:0 -15px!important}.trendmd-widget:hover .trendmd-widget-settings__cog{background-position:0 -16px!important}.trendmd-widget-disclaimer__wrapper{cursor:pointer!important;display:inline-block!important;position:relative!important}.trendmd-widget-disclaimer__wrapper .trendmd-widget-disclaimer__text{background-color:#000;border-radius:6px;bottom:150%;color:#fff;left:50%;margin-left:-85px;padding:5px 0;position:absolute;text-align:center;visibility:hidden;width:170px;z-index:1}.trendmd-widget-disclaimer__wrapper .trendmd-widget-disclaimer__text:after{border:5px solid transparent;border-top-color:#000;content:"";left:50%;margin-left:-5px;position:absolute;top:100%}.trendmd-widget-disclaimer__wrapper:hover .trendmd-widget-disclaimer__text{visibility:visible}.trendmd-widget-disclaimer__icon{background-image:url('https://css.trendmd.com/info.svg')!important;background-repeat:no-repeat!important;background-size:92%!important;display:inline-block!important;height:11px!important;opacity:.35!important;width:12px!important}.trendmd-widget-disclaimer__icon:hover{opacity:1!important}.trendmd-widget_title-smaller .trendmd-widget-footer-inner,.trendmd-widget_title-smaller .trendmd-widget-header__heading{font-size:.9em!important}.trendmd-widget_title-smaller .trendmd-widget-brand__recommended{line-height:16px!important}.trendmd-widget_title-smaller .trendmd-widget-brand__logo{height:16px!important;vertical-align:5px!important;width:69px!important}.trendmd-widget_title-smaller .trendmd-widget-settings__cog{margin-top:0!important}.trendmd-widget_title-larger .trendmd-widget-footer-inner,.trendmd-widget_title-larger .trendmd-widget-header__heading{font-size:1.1em!important}.trendmd-widget_title-larger .trendmd-widget-brand__recommended{line-height:25px!important}.trendmd-widget_title-larger .trendmd-widget-brand__logo{height:25px!important;vertical-align:10px!important;width:104px!important}.trendmd-widget_title-larger .trendmd-widget-settings__cog{margin-top:0!important}.trendmd-widget_body-larger .trendmd-widget-inner{font-size:1.1em;line-height:20px}.trendmd-widget_body-smaller .trendmd-widget-inner{font-size:.9em}</style><style class="darkreader darkreader--sync" media="screen"></style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/embed-59614f5c46b49b21eeef3bb28c4fb38d1e7069e8d014752fcb66e84942556802.css"><style class="darkreader darkreader--cors" media="screen">div.altmetric-embed[id^=_altmetric_popover_el] div,div.altmetric-embed[id^=_altmetric_popover_el] span,div.altmetric-embed[id^=_altmetric_popover_el] applet,div.altmetric-embed[id^=_altmetric_popover_el] object,div.altmetric-embed[id^=_altmetric_popover_el] iframe,div.altmetric-embed[id^=_altmetric_popover_el] h1,div.altmetric-embed[id^=_altmetric_popover_el] h2,div.altmetric-embed[id^=_altmetric_popover_el] h3,div.altmetric-embed[id^=_altmetric_popover_el] h4,div.altmetric-embed[id^=_altmetric_popover_el] h5,div.altmetric-embed[id^=_altmetric_popover_el] h6,div.altmetric-embed[id^=_altmetric_popover_el] p,div.altmetric-embed[id^=_altmetric_popover_el] blockquote,div.altmetric-embed[id^=_altmetric_popover_el] pre,div.altmetric-embed[id^=_altmetric_popover_el] a,div.altmetric-embed[id^=_altmetric_popover_el] abbr,div.altmetric-embed[id^=_altmetric_popover_el] acronym,div.altmetric-embed[id^=_altmetric_popover_el] address,div.altmetric-embed[id^=_altmetric_popover_el] big,div.altmetric-embed[id^=_altmetric_popover_el] cite,div.altmetric-embed[id^=_altmetric_popover_el] code,div.altmetric-embed[id^=_altmetric_popover_el] del,div.altmetric-embed[id^=_altmetric_popover_el] dfn,div.altmetric-embed[id^=_altmetric_popover_el] em,div.altmetric-embed[id^=_altmetric_popover_el] img,div.altmetric-embed[id^=_altmetric_popover_el] ins,div.altmetric-embed[id^=_altmetric_popover_el] kbd,div.altmetric-embed[id^=_altmetric_popover_el] q,div.altmetric-embed[id^=_altmetric_popover_el] s,div.altmetric-embed[id^=_altmetric_popover_el] samp,div.altmetric-embed[id^=_altmetric_popover_el] small,div.altmetric-embed[id^=_altmetric_popover_el] strike,div.altmetric-embed[id^=_altmetric_popover_el] strong,div.altmetric-embed[id^=_altmetric_popover_el] sub,div.altmetric-embed[id^=_altmetric_popover_el] sup,div.altmetric-embed[id^=_altmetric_popover_el] tt,div.altmetric-embed[id^=_altmetric_popover_el] var,div.altmetric-embed[id^=_altmetric_popover_el] b,div.altmetric-embed[id^=_altmetric_popover_el] u,div.altmetric-embed[id^=_altmetric_popover_el] i,div.altmetric-embed[id^=_altmetric_popover_el] center,div.altmetric-embed[id^=_altmetric_popover_el] dl,div.altmetric-embed[id^=_altmetric_popover_el] dt,div.altmetric-embed[id^=_altmetric_popover_el] dd,div.altmetric-embed[id^=_altmetric_popover_el] ol,div.altmetric-embed[id^=_altmetric_popover_el] ul,div.altmetric-embed[id^=_altmetric_popover_el] li,div.altmetric-embed[id^=_altmetric_popover_el] fieldset,div.altmetric-embed[id^=_altmetric_popover_el] form,div.altmetric-embed[id^=_altmetric_popover_el] label,div.altmetric-embed[id^=_altmetric_popover_el] legend,div.altmetric-embed[id^=_altmetric_popover_el] table,div.altmetric-embed[id^=_altmetric_popover_el] caption,div.altmetric-embed[id^=_altmetric_popover_el] tbody,div.altmetric-embed[id^=_altmetric_popover_el] tfoot,div.altmetric-embed[id^=_altmetric_popover_el] thead,div.altmetric-embed[id^=_altmetric_popover_el] tr,div.altmetric-embed[id^=_altmetric_popover_el] th,div.altmetric-embed[id^=_altmetric_popover_el] td,div.altmetric-embed[id^=_altmetric_popover_el] article,div.altmetric-embed[id^=_altmetric_popover_el] aside,div.altmetric-embed[id^=_altmetric_popover_el] canvas,div.altmetric-embed[id^=_altmetric_popover_el] details,div.altmetric-embed[id^=_altmetric_popover_el] embed,div.altmetric-embed[id^=_altmetric_popover_el] figure,div.altmetric-embed[id^=_altmetric_popover_el] figcaption,div.altmetric-embed[id^=_altmetric_popover_el] footer,div.altmetric-embed[id^=_altmetric_popover_el] header,div.altmetric-embed[id^=_altmetric_popover_el] hgroup,div.altmetric-embed[id^=_altmetric_popover_el] menu,div.altmetric-embed[id^=_altmetric_popover_el] nav,div.altmetric-embed[id^=_altmetric_popover_el] output,div.altmetric-embed[id^=_altmetric_popover_el] ruby,div.altmetric-embed[id^=_altmetric_popover_el] section,div.altmetric-embed[id^=_altmetric_popover_el] summary,div.altmetric-embed[id^=_altmetric_popover_el] time,div.altmetric-embed[id^=_altmetric_popover_el] mark,div.altmetric-embed[id^=_altmetric_popover_el] audio,div.altmetric-embed[id^=_altmetric_popover_el] video{margin:0;padding:0;border:0;font:inherit;font-size:100%;vertical-align:baseline}div.altmetric-embed[id^=_altmetric_popover_el] table{border-collapse:collapse;border-spacing:0}div.altmetric-embed[id^=_altmetric_popover_el] caption,div.altmetric-embed[id^=_altmetric_popover_el] th,div.altmetric-embed[id^=_altmetric_popover_el] td{text-align:left;font-weight:normal;vertical-align:middle}div.altmetric-embed[id^=_altmetric_popover_el] q,div.altmetric-embed[id^=_altmetric_popover_el] blockquote{quotes:none}div.altmetric-embed[id^=_altmetric_popover_el] q:before,div.altmetric-embed[id^=_altmetric_popover_el] q:after,div.altmetric-embed[id^=_altmetric_popover_el] blockquote:before,div.altmetric-embed[id^=_altmetric_popover_el] blockquote:after{content:"";content:none}div.altmetric-embed[id^=_altmetric_popover_el] a img{border:none}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container a.altmetric_details{text-decoration:none}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container a.altmetric_details:hover{text-decoration:underline}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container a.altmetric_details,div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container a.altmetric_details:hover{color:#1565c0}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container a.link-to-altmetric-details-tab{text-decoration:none;color:#000000}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container a.link-to-altmetric-details-tab:hover{text-decoration:underline}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container .modal-open .altmetric-popover{z-index:2060}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container .altmetric-popover.altmetric-top{margin-top:-5px;margin-bottom:0;margin-left:0;margin-right:0}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner.altmetric-right{margin-left:5px;margin-bottom:0;margin-top:0;margin-right:0}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container .altmetric-popover.altmetric-bottom{margin-top:5px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container .altmetric-popover.altmetric-left{margin-left:-5px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric_arrow.altmetric-top{bottom:0;left:50%;margin-left:-5px;border-top:5px solid #000000;border-right:5px solid transparent;border-left:5px solid transparent}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric_arrow.altmetric-right{top:50%;left:0;margin-top:-5px;border-top:5px solid transparent;border-right:5px solid #000000;border-bottom:5px solid transparent}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric_arrow.altmetric-bottom{top:0;left:50%;margin-left:-5px;border-right:5px solid transparent;border-bottom:5px solid #000000;border-left:5px solid transparent}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric_arrow.altmetric-left{top:50%;right:0;margin-top:-5px;border-top:5px solid transparent;border-bottom:5px solid transparent;border-left:5px solid #000000}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric_arrow{position:absolute;width:0;height:0}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner.altmetric-floating{width:280px;padding:3px;overflow:hidden;background:#000000;background:rgba(0,0,0,0.8);-webkit-border-radius:6px;-moz-border-radius:6px;border-radius:6px;-webkit-box-shadow:0 3px 7px rgba(0,0,0,0.3);-moz-box-shadow:0 3px 7px rgba(0,0,0,0.3);box-shadow:0 3px 7px rgba(0,0,0,0.3);margin:0}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .readers{margin-top:10px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_mendeley{border-left:16px solid #A60000;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_connotea{border-left:16px solid #FF4040;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_citeulike{border-left:16px solid #BCD2EF;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_tweets,div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_tweeters{border-left:16px solid #00aacc;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_pinterest{border-left:16px solid #CC6600;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_fbwalls{border-left:16px solid #313186;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_rdt,div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_rdts{border-left:16px solid #BDE0FF;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_feeds{border-left:16px solid #FAA400;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_msm{border-left:16px solid #C90032;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_rh{border-left:16px solid #F4006E;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_f1000{border-left:16px solid #F4006E;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_qna{border-left:16px solid #DEDEDE;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_forums{border-left:16px solid #FF3366;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_gplus{border-left:16px solid #912470;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_videos{border-left:16px solid #94DB5E;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .tip_linkedin{border-left:16px solid #1E90FF;padding-left:10px;line-height:18px}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner b{font-weight:bold}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .altmetric-popover-content{font-size:13px;font-family:"Helvetica Neue", Helvetica, Arial, sans-serif;line-height:13px;padding:8px;background-color:#ffffff;margin:0}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner .altmetric-popover-content.altmetric-floating{-webkit-border-radius:3px 3px 3px 3px;-moz-border-radius:3p 3p 3px 3px;border-radius:3px 3px 3px 3px;-webkit-background-clip:padding-box;-moz-background-clip:padding-box;background-clip:padding-box}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container div.altmetric-popover-inner div{margin:0}div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container .altmetric-popover-content p,div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container .altmetric-popover-content ul,div.altmetric-embed[id^=_altmetric_popover_el] div.altmetric_container .altmetric-popover-content ol{margin-bottom:0}div.altmetric-embed[id^=_altmetric_popover_el] .modal-open .tooltip{z-index:2070}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip{position:absolute;z-index:1020;display:block;padding:5px;font-size:11px;opacity:0;filter:alpha(opacity=0);visibility:visible}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip.in{opacity:0.8;filter:alpha(opacity=80)}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip.top{margin-top:-2px}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip.right{margin-left:2px}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip.bottom{margin-top:2px}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip.left{margin-left:-2px}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-top:5px solid #000000;border-right:5px solid transparent;border-left:5px solid transparent}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-top:5px solid transparent;border-bottom:5px solid transparent;border-left:5px solid #000000}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-right:5px solid transparent;border-bottom:5px solid #000000;border-left:5px solid transparent}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-top:5px solid transparent;border-right:5px solid #000000;border-bottom:5px solid transparent}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip-inner{max-width:200px;padding:3px 8px;color:#ffffff;text-align:center;text-decoration:none;background-color:#000000;-webkit-border-radius:4px;-moz-border-radius:4px;border-radius:4px}div.altmetric-embed[id^=_altmetric_popover_el] .tooltip-arrow{position:absolute;width:0;height:0}div.altmetric-embed[id^=_altmetric_popover_el]{position:absolute;top:0;left:0;display:none;padding:5px}div.altmetric-embed.altmetric-popover[id^=_altmetric_popover_el]{z-index:1010}.altmetric-condensed-legend{float:left;margin:0 10px 0 0}.altmetric-normal-legend{float:left;margin:0 20px 0 0}.altmetric-condensed-legend .altmetric-see-more-details{display:none}</style><style class="darkreader darkreader--sync" media="screen"></style><script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/js(3)"></script></head>
<body id="article" class="body is-sticky-enabled is-altlayout-enabled is-hybrid-skin-template" data-instanceprefix="instance" data-elements-to-offset-scroll-when-fixed=".main-header-container,.article-navigation-bar"><div id="MathJax_Message" style="display: none;"></div>
<!-- Google Tag Manager-->
<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-M8DSFPZR"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-M8DSFPZR');</script>
<!-- End Google Tag Manager-->
<div id="hiddenContext" class="hidden-js-div" data-cookiemessage="We use &lt;a href=&quot;/cookie-info&quot;&gt;cookies&lt;/a&gt; to track usage and preferences." data-cookieaccepttext="I Understand" data-cookiedeclinetext="Disable Cookies" data-cookiepolicytext="Privacy Policy"></div>
<div id="skinPublishingDates" class="hidden-js-div">1887</div>
<div id="wrapper" class="siteWrapper">
<header class="header" id="header">
<div class="navbar navbar-default main-header-container" role="navigation">
<div class="container header-container">
<a href="https://www.microbiologyresearch.org/" class="logo header-container__logo"><picture>
<source srcset="/images/hybrid/logo/MBS_logo_mobile_lo-res.png, /images/hybrid/logo/MBS_logo_full_size_desktop_lo-res.png 2x" media="(max-width: 910px)">
<source srcset="/images/hybrid/logo/MBS_logo_full_size_desktop_lo-res.png, /images/hybrid/logo/MBS_logo_full_size_desktop_hi-res.png 2x"> 
<img alt="Microbiology Society logo" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/MBS_logo_full_size_desktop_lo-res.png" class="img-responsive main-logo-image">
</picture></a>
<ul class="navbar-nav-main-menu header-container__icon-menu" aria-label="User navigation items" role="menu">
<li class="search-nav-container 
navbar-nav-main-menu__search-container" role="none">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="js-toggle-search-window" title="Search" aria-label="Search"><img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/search.svg" alt="search icon"></a>
<div class="search-nav js-global-site-search-container" id="search-nav">
<form id="global-search-form" action="https://www.microbiologyresearch.org/search" class="search-nav__form " data-advancedsearchavailable="false" data-nosearchtermerror="Please enter a search term">
<div class="search-nav__up-triangle "></div> 
<input type="text" id="quickSearchBox" class="js-searchcomplete search-nav__input-box ui-autocomplete-input" name="value1" placeholder="Search Site" aria-label="Search" autocomplete="off"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span>
<input type="hidden" value="fulltext" name="option1" aria-label="Search">
<!-- <input name="pageSize" value="8" type="hidden" /> taken out at present - as interferes with search at present and not scrolling -->
<button type="submit" class="search-nav__submit-btn">
<img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/search.svg" alt="search icon" class="search-nav__submit-btn-image">
</button>
</form>
</div>
</li>
<li class="navbar-nav-main-menu__login-container" role="none">
<a href="https://www.microbiologyresearch.org/registration/signin-or-register.action?signInTarget=%2Fcontent%2Fjournal%2Fmicro%2F10.1099%2Fmic.0.000485" class="js-profile" title="Login or Register" aria-label="Login or Register">
<img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/account_circle.svg" alt="account icon">
</a>
</li>
<li class=" navbar-nav-main-menu__cart-container" role="none">
<a title="Shopping cart" href="https://www.microbiologyresearch.org/cart" aria-label="Cart" role="menuitem" class="cart-button-desktop">
<img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/shopping_cart.svg" alt="shopping cart icon">
</a>
</li>
<li class="main-navigation-menu" role="none">
<button type="button" class="main-navigation-menu__button" data-toggle="collapse" data-target="#collapse-main-nav" role="menuitem" aria-controls="collapse-main-nav" aria-label="Site menu" title="Site menu">
<span class="icon-bar bar-one"></span>
<span class="icon-bar bar-two"></span>
<span class="icon-bar bar-three"></span>
</button>
<div class="collapse main-navigation-menu__dropdown 
main-navigation-dropdown" id="collapse-main-nav">
<ul class="main-navigation-dropdown__menu" role="menu">
<li class="main-navigation-dropdown__item active">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" title="Publications" role="button" aria-label="Publications expand or collapse" aria-controls="collapse-main-sub-nav-1" aria-expanded="false" class="main-navigation-dropdown__link js-link-toggle-main-sub-menu">Publications</a>
<span class="main-navigation-dropdown__toggle-expand js-link-toggle-main-sub-menu" role="button" aria-label="Publications expand or collapse" title="expand or collapse" aria-controls="collapse-main-sub-nav-1" aria-expanded="false">
<span class="glyphicon glyphicon-chevron-down 
main-navigation-dropdown__chevron js-toggle-main-menu-item" aria-hidden="true">
</span>
</span>
<ul class="main-navigation-dropdown__submenu js-toggle-display-sub-menu" id="collapse-main-sub-nav-1">
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/content/journal/micro" class="main-navigation-dropdown__submenu-link" title="Microbiology">Microbiology</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/content/journal/jgv" class="main-navigation-dropdown__submenu-link" title="Journal of General Virology">Journal of General Virology</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/content/journal/jmm" class="main-navigation-dropdown__submenu-link" title="Journal of Medical Microbiology">Journal of Medical Microbiology</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/content/journal/mgen" class="main-navigation-dropdown__submenu-link" title="Microbial Genomics">Microbial Genomics</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/content/journal/ijsem" class="main-navigation-dropdown__submenu-link" title="International Journal of Systematic and Evolutionary Microbiology">International Journal of Systematic and Evolutionary Microbiology</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/content/journal/acmi" class="main-navigation-dropdown__submenu-link" title="Access Microbiology, an open research platform">Access Microbiology, an open research platform</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/content/journal/jmmcr" class="main-navigation-dropdown__submenu-link" title="JMM Case Reports">JMM Case Reports</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/content/collections" class="main-navigation-dropdown__submenu-link" title="Browse Collections">Browse Collections</a>
</li>
</ul>
</li>
<li class="main-navigation-dropdown__item active">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" title="About us" role="button" aria-label="About us expand or collapse" aria-controls="collapse-main-sub-nav-2" aria-expanded="false" class="main-navigation-dropdown__link js-link-toggle-main-sub-menu">About us</a>
<span class="main-navigation-dropdown__toggle-expand js-link-toggle-main-sub-menu" role="button" aria-label="About us expand or collapse" title="expand or collapse" aria-controls="collapse-main-sub-nav-2" aria-expanded="false">
<span class="glyphicon glyphicon-chevron-down 
main-navigation-dropdown__chevron js-toggle-main-menu-item" aria-hidden="true">
</span>
</span>
<ul class="main-navigation-dropdown__submenu js-toggle-display-sub-menu" id="collapse-main-sub-nav-2">
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://microbiologysociety.org/" class="main-navigation-dropdown__submenu-link" title="The Microbiology Society">The Microbiology Society</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/why-publish" class="main-navigation-dropdown__submenu-link" title="Why publish?">Why publish?</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/our-editorial-boards" class="main-navigation-dropdown__submenu-link" title="Joining our Editorial Boards">Joining our Editorial Boards</a>
</li>
</ul>
</li>
<li class="main-navigation-dropdown__item active">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" title="Information for authors" role="button" aria-label="Information for authors expand or collapse" aria-controls="collapse-main-sub-nav-3" aria-expanded="false" class="main-navigation-dropdown__link js-link-toggle-main-sub-menu">Information for authors</a>
<span class="main-navigation-dropdown__toggle-expand js-link-toggle-main-sub-menu" role="button" aria-label="Information for authors expand or collapse" title="expand or collapse" aria-controls="collapse-main-sub-nav-3" aria-expanded="false">
<span class="glyphicon glyphicon-chevron-down 
main-navigation-dropdown__chevron js-toggle-main-menu-item" aria-hidden="true">
</span>
</span>
<ul class="main-navigation-dropdown__submenu js-toggle-display-sub-menu" id="collapse-main-sub-nav-3">
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/article-types" class="main-navigation-dropdown__submenu-link" title="Article types">Article types</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/prepare-an-article" class="main-navigation-dropdown__submenu-link" title="Prepare an article">Prepare an article</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/submission-and-peer-review" class="main-navigation-dropdown__submenu-link" title="Submission and peer review">Submission and peer review</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/post-acceptance" class="main-navigation-dropdown__submenu-link" title="Post-acceptance and publication">Post-acceptance and publication</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/publishing-costs" class="main-navigation-dropdown__submenu-link" title="Open Access costs">Open Access costs</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/fee-free-open-access" class="main-navigation-dropdown__submenu-link" title="Fee-free Open Access">Fee-free Open Access</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/ethics-policies" class="main-navigation-dropdown__submenu-link" title="Ethics policies">Ethics policies</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/article-journal-metrics" class="main-navigation-dropdown__submenu-link" title="Impact and metrics">Impact and metrics</a>
</li>
</ul>
</li>
<li class="main-navigation-dropdown__item active">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" title="Information for librarians" role="button" aria-label="Information for librarians expand or collapse" aria-controls="collapse-main-sub-nav-4" aria-expanded="false" class="main-navigation-dropdown__link js-link-toggle-main-sub-menu">Information for librarians</a>
<span class="main-navigation-dropdown__toggle-expand js-link-toggle-main-sub-menu" role="button" aria-label="Information for librarians expand or collapse" title="expand or collapse" aria-controls="collapse-main-sub-nav-4" aria-expanded="false">
<span class="glyphicon glyphicon-chevron-down 
main-navigation-dropdown__chevron js-toggle-main-menu-item" aria-hidden="true">
</span>
</span>
<ul class="main-navigation-dropdown__submenu js-toggle-display-sub-menu" id="collapse-main-sub-nav-4">
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/subscribe-get-access" class="main-navigation-dropdown__submenu-link" title="Licensing and access options">Licensing and access options</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/license-agreements" class="main-navigation-dropdown__submenu-link" title="Licence agreements">Licence agreements</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/admin-tools" class="main-navigation-dropdown__submenu-link" title="Manage your subscription">Manage your subscription</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/publish-and-read" class="main-navigation-dropdown__submenu-link" title="Publish and Read">Publish and Read</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/fee-free-open-access" class="main-navigation-dropdown__submenu-link" title="List of Publish and Read institutions">List of Publish and Read institutions</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/library-faqs" class="main-navigation-dropdown__submenu-link" title="Library FAQs">Library FAQs</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/resources" class="main-navigation-dropdown__submenu-link" title="Resources for librarians">Resources for librarians</a>
</li>
</ul>
</li>
<li class="main-navigation-dropdown__item active">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" title="Information for reviewers" role="button" aria-label="Information for reviewers expand or collapse" aria-controls="collapse-main-sub-nav-5" aria-expanded="false" class="main-navigation-dropdown__link js-link-toggle-main-sub-menu">Information for reviewers</a>
<span class="main-navigation-dropdown__toggle-expand js-link-toggle-main-sub-menu" role="button" aria-label="Information for reviewers expand or collapse" title="expand or collapse" aria-controls="collapse-main-sub-nav-5" aria-expanded="false">
<span class="glyphicon glyphicon-chevron-down 
main-navigation-dropdown__chevron js-toggle-main-menu-item" aria-hidden="true">
</span>
</span>
<ul class="main-navigation-dropdown__submenu js-toggle-display-sub-menu" id="collapse-main-sub-nav-5">
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/review-with-us" class="main-navigation-dropdown__submenu-link" title="Review with us">Review with us</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/guidelines-for-reviewers" class="main-navigation-dropdown__submenu-link" title="Guidleines for reviewers">Guidleines for reviewers</a>
</li>
</ul>
</li>
<li class="main-navigation-dropdown__item active">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" title="Open Access" role="button" aria-label="Open Access expand or collapse" aria-controls="collapse-main-sub-nav-6" aria-expanded="false" class="main-navigation-dropdown__link js-link-toggle-main-sub-menu">Open Access</a>
<span class="main-navigation-dropdown__toggle-expand js-link-toggle-main-sub-menu" role="button" aria-label="Open Access expand or collapse" title="expand or collapse" aria-controls="collapse-main-sub-nav-6" aria-expanded="false">
<span class="glyphicon glyphicon-chevron-down 
main-navigation-dropdown__chevron js-toggle-main-menu-item" aria-hidden="true">
</span>
</span>
<ul class="main-navigation-dropdown__submenu js-toggle-display-sub-menu" id="collapse-main-sub-nav-6">
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/publish-oa" class="main-navigation-dropdown__submenu-link" title="Publish Open Access in our journals">Publish Open Access in our journals</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/publishing-costs" class="main-navigation-dropdown__submenu-link" title="Open Access costs">Open Access costs</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/publish-and-read" class="main-navigation-dropdown__submenu-link" title="Publish and Read">Publish and Read</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/oa-reporting" class="main-navigation-dropdown__submenu-link" title="Open Access reporting">Open Access reporting</a>
</li>
<li class="main-navigation-dropdown__submenu-item ">
<a href="https://www.microbiologyresearch.org/publish-and-read-FAQ" class="main-navigation-dropdown__submenu-link" title="Publish and Read FAQ">Publish and Read FAQ</a>
</li>
</ul>
</li>
<li class="main-navigation-dropdown__item active">
<a href="https://www.microbiologyresearch.org/publishing-fundamentals" title="Publishing Fundamentals" class="main-navigation-dropdown__link">Publishing Fundamentals</a>
</li>
<li class="main-navigation-dropdown__item active">
<a href="https://www.microbiologyresearch.org/frequently-asked-questions" title=" FAQs" class="main-navigation-dropdown__link"> FAQs</a>
</li>
<li class="main-navigation-dropdown__item active">
<a href="https://www.microbiologyresearch.org/contact-us" title="Contact us" class="main-navigation-dropdown__link">Contact us</a>
</li>
</ul>
</div>
</li>
</ul>
<div class="header-container__publication-menu">
<ul class="nav navbar-nav navbar-nav--publication-menu" aria-label="" role="menubar">
<li role="menuitem" class="button1 dropdown">
<a title="Publications" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="dropdown-toggle" data-toggle="dropdown" id="dropdownMenu0" role="button" aria-haspopup="true" aria-expanded="false">
Publications
<i class="fa fa-angle-down"></i>
</a>
<ul class="dropdown-menu" role="menu">
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/content/journal/micro" title="Microbiology" role="menuitem" tabindex="-1" class="megamenu-link">Microbiology</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/content/journal/jgv" title="Journal of General Virology" role="menuitem" tabindex="-1" class="megamenu-link">Journal of General Virology</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/content/journal/jmm" title="Journal of Medical Microbiology" role="menuitem" tabindex="-1" class="megamenu-link">Journal of Medical Microbiology</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/content/journal/mgen" title="Microbial Genomics" role="menuitem" tabindex="-1" class="megamenu-link">Microbial Genomics</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/content/journal/ijsem" title="International Journal of Systematic and Evolutionary Microbiology" role="menuitem" tabindex="-1" class="megamenu-link">International Journal of Systematic and Evolutionary Microbiology</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/content/journal/acmi" title="Access Microbiology, an open research platform" role="menuitem" tabindex="-1" class="megamenu-link">Access Microbiology, an open research platform</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/content/journal/jmmcr" title="JMM Case Reports" role="menuitem" tabindex="-1" class="megamenu-link">JMM Case Reports</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/content/collections" title="Browse Collections" role="menuitem" tabindex="-1" class="megamenu-link">Browse Collections</a>
</li>
</ul>
</li>
<li role="menuitem" class="button2 dropdown">
<a title="Information for authors" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="dropdown-toggle" data-toggle="dropdown" id="dropdownMenu1" role="button" aria-haspopup="true" aria-expanded="false">
Information for authors
<i class="fa fa-angle-down"></i>
</a>
<ul class="dropdown-menu" role="menu">
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/article-types" title="Article types" role="menuitem" tabindex="-1" class="megamenu-link">Article types</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/prepare-an-article" title="Prepare an article" role="menuitem" tabindex="-1" class="megamenu-link">Prepare an article</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/submission-and-peer-review" title="Submission and peer review" role="menuitem" tabindex="-1" class="megamenu-link">Submission and peer review</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/post-acceptance" title="Post-acceptance and publication" role="menuitem" tabindex="-1" class="megamenu-link">Post-acceptance and publication</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/publishing-costs" title="Open Access costs" role="menuitem" tabindex="-1" class="megamenu-link">Open Access costs</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/fee-free-open-access" title="Fee-free Open Access" role="menuitem" tabindex="-1" class="megamenu-link">Fee-free Open Access</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/ethics-policies" title="Ethics policies" role="menuitem" tabindex="-1" class="megamenu-link">Ethics policies</a>
</li>
<li role="presentation" class="journal-list ">
<a href="https://www.microbiologyresearch.org/article-journal-metrics" title="Impact and metrics" role="menuitem" tabindex="-1" class="megamenu-link">Impact and metrics</a>
</li>
</ul>
</li>
<li role="menuitem" class="button3 ">
<a href="https://microbiologysociety.org/" title="Microbiology Society">
Microbiology Society
</a>
</li>
</ul>
</div> 
</div> 
</div>
</header>
<div id="bellowheadercontainer" class="container no-padding container--jc__micro">
<button type="button" id="mobile-tools-button" data-parent="#bellowheadercontainer" class="navbar-toggle toggle-arrow" data-toggle="collapse" data-target="#tools-nav">
<span class="sr-only">Toggle navigation</span>
<span>Tools <i class="fa fa-angle-down"></i></span>
</button>
<aside class="col-xs-12 col-sm-12 col-md-3 pull-right left-padding sidebar-pub2web-container header-sidebar" aria-label="site sidebar containing page tools, access key and possibly adverts"> 
<div id="signInOrRegisterDialog"></div>
</aside>
<main class="col-xs-12 col-sm-12 col-md-9 content main-content-container js-main-content-container">
<input type="hidden" name="svgImageBackgroundColor" value="rgba(255,255,255,0.7)">
<div class="clearfix"></div>
<div class="article-cover">
<!-- If we have come from a search page, a list of search articles maybe present in the session and should be used to provide
the listItems for the next and previous bttns -->
<div class="title-box item-meta-data">
<div class="journal-volume-issue-container item-meta-data__journal-volume-issue-container">
<a href="https://www.microbiologyresearch.org/content/journal/micro" class="item-meta-data__journal-title-link">
<h3 title="Microbiology" class="journaltitle_micro item-meta-data__journal container__journal-title">
<img alt="Microbiology Society logo" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/2023_New_Microbiology_Logo.svg" class="img-responsive main-logo-image fr-fil fr-dib" style="margin-left: -10px; height: 80px;"></h3>
</a>
<a href="https://www.microbiologyresearch.org/content/journal/micro/163/7" title="" class="item-meta-data__issue-title-link">
<h4 class="item-meta-data__journal-issue" title="Volume 163, Issue 7">Volume 163, Issue 7</h4>
</a>
</div>
<div class="hidden-xs article-access-icon-and-access item-meta-data__accreditation">
<h4 class="article-type item-meta-data__type">Research Article</h4>
<h4 class="article-access item-meta-data__oa"> 
<span class="accesstext">Free</span>
</h4>
</div>
<h1 class="item-meta-data__item-title">
Mutational analysis of the MS2 lysis protein L
<img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_green_transparent.svg" alt="Free">
</h1>
<ul class="list-unstyled article-meta-data item-meta-data__article"> 
<li class="data-author article-meta-data__item item-meta-data__author">
<span class="meta-value authors"><a href="https://www.microbiologyresearch.org/search?value1=Karthik+R.+Chamakura&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Karthik R. Chamakura</a><sup>1​,2​</sup>, <a href="https://www.microbiologyresearch.org/search?value1=Garrett+B.+Edwards&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Garrett B. Edwards</a><sup>1​,2​,†​</sup>&nbsp;and <a href="https://www.microbiologyresearch.org/search?value1=Ry+Young&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Ry Young</a><sup>1​,2​</sup></span>
</li>
<li class="data-author article-meta-data__item item-meta-data__author-affiliations">
<div class="showhide spacing">
<a id="showHideAffiliation" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" role="button" class="std-display viewHideAffiliation js-showhide" aria-expanded="false" aria-controls="showHideAffiliationContent">
<span class="js-plus">
<i class="fa fa-plus-circle"></i>
<span class="underline">
View Affiliations
</span>
</span>
<span class="js-minus minus">
<i class="fa fa-minus-circle"></i>
<span class="underline">
Hide Affiliations
</span>
</span>
</a>
<div id="showHideAffiliationContent" class="js-description minus">
<div class="affiliation-details">
<span class="meta-key affiliations_label">
<span dir="auto">Affiliations:</span>
</span>
<!-- -->
<span class="meta-value affiliations">
<sup class="affiliations_number">1​</sup>
<a class="affilLinkToEncode" href="https://www.microbiologyresearch.org/search?option1=pub_affiliation&amp;value1=%22Center+for+Phage+Technology%2C+Texas+A%26amp%3BM+AgriLife%2C+Texas+A%26amp%3BM+University%2C+College+Station%2C+TX%2C+USA%22&amp;option912=resultCategory&amp;value912=ResearchPublicationContent">Center for Phage Technology, Texas A&amp;M AgriLife, Texas A&amp;M University, College Station, TX, USA</a>
</span>
<!-- , -->
<span class="meta-value affiliations">
<sup class="affiliations_number">2​</sup>
<a class="affilLinkToEncode" href="https://www.microbiologyresearch.org/search?option1=pub_affiliation&amp;value1=%22Department+of+Biochemistry+and+Biophysics%2C+Texas+A%26amp%3BM+University%2C+College+Station%2C+TX%2C+USA%22&amp;option912=resultCategory&amp;value912=ResearchPublicationContent">Department of Biochemistry and Biophysics, Texas A&amp;M University, College Station, TX, USA</a>
</span>
<!-- and -->
<span class="meta-value affiliations">
<sup class="affiliations_number">†​</sup>
<a class="affilLinkToEncode" href="https://www.microbiologyresearch.org/search?option1=pub_affiliation&amp;value1=%22Present+address%3A+Department+of+Chemistry+and+Biochemistry%2C+University+of+Colorado+Boulder%2C+Boulder%2C+CO%2C+USA.%22&amp;option912=resultCategory&amp;value912=ResearchPublicationContent">Present address: Department of Chemistry and Biochemistry, University of Colorado Boulder, Boulder, CO, USA.</a>
</span>
</div>
<span class="author-correspondence">
<xhtml:span xml:lang="en">
<span class="jp-bold">*Correspondence:</span> Ry Young, <a href="mailto:ryland@tamu.edu">ryland@tamu.edu</a> 
</xhtml:span>
</span>
<div class="institutionLinks">
<div class="instList">
</div>
</div>
</div>
</div>
</li>
<li class="article-meta-data__item item-meta-data__date-and-doi">
<span class="meta-container"> 
<span class="meta-key meta-container__label 
item-meta-data__pub-date-label"><span class="hidden-xs"></span>Published:</span>
<span class="meta-value meta-container__value item-meta-data__pub-date-value">
01 July 2017
</span>
</span>
<a href="https://doi.org/10.1099/mic.0.000485" class="externallink meta-container meta-container__value item-meta-data__doi meta-container--doi hidden-xs">https://doi.org/10.1099/mic.0.000485</a>
</li>
</ul>
</div>
</div>
<nav class="article-navigation-bar non-printed-section" aria-label="Article navigation">
<div class="article-navigation-bar__inner-container">
<ul class="nav navbar-nav navbar-nav--article-nav-left">
<li class="dropdown navbar-nav__item">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" data-toggle="dropdown" role="button" title="Navigate Article Resources" aria-haspopup="true" aria-expanded="false"><i class="material-icons hidden-xxs">info</i> 
Info</a> 
<ul class="dropdown-menu js-transformer-tabs">
<li class=" tab-one tabIcon" id="abstract_tab">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract_content" title="Article information" class=" tab-one" role="button" aria-pressed="false">
<span class="iconContainer">
<i class="fa-stack fa-lg">
<i class="fa fa-ellipsis-h fa-stack-1x"></i>
<i class="fa fa-file-o fa-stack-2x"></i>
</i>
<strong class="titleIcon">Article information </strong>
</span>
</a>
</li>
<li class="active tabIcon" id="html_tab">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#html_fulltext" title="Full-Text" class="active " role="button" aria-pressed="true">
<span class="iconContainer">
<i class="fa-stack fa-lg" aria-hidden="true">
<i class="fa fa-code fa-stack-1x" aria-hidden="true"></i>
<i class="fa fa-file-o fa-stack-2x" aria-hidden="true"></i>
</i>
<strong class="titleIcon">
Full-Text 
</strong>
</span>
</a>
</li>
<li class=" tabIcon" id="dataandmedia_tab">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#dataandmedia" title="Figures &amp; Tables" class=" " role="button" aria-pressed="false">
<span class="iconContainer">
<i class="fa fa-file-text-o"></i>
<strong class="titleIcon">
Figures and Tables 
</strong>
</span>
</a>
</li>
<li class=" tabIcon" id="references">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#referenceContainer" title="References" class=" " role="button" aria-pressed="false">
<span class="iconContainer">
<i class="fa-stack fa-lg" aria-hidden="true">
<i class="fa fa-bookmark fa-stack-1x" aria-hidden="true"></i>
<i class="fa fa-file-o fa-stack-2x" aria-hidden="true"></i>
</i>
<strong class="titleIcon">
References 
(36)
</strong>
</span>
</a>
</li>
<li class=" tabIcon" id="cite">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#cited" title="Cited By" class=" " role="button" aria-pressed="false">
<span class="iconContainer">
<i class="fa-stack fa-lg" aria-hidden="true">
<i class="fa fa-quote-right fa-stack-1x" aria-hidden="true"></i>
<i class="fa fa-file-o fa-stack-2x" aria-hidden="true"></i>
</i>
<strong class="titleIcon">
Cited By 
(12)
</strong>
</span>
</a>
</li>
<li class=" tabIcon" id="supplementary_Tab">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#supplementary_data" title="Supplementary Material" class=" " role="button" aria-pressed="false">
<span class="iconContainer">
<i class="fa fa-file-text-o"></i>
<strong class="titleIcon">
Supplementary Material 
</strong>
</span>
</a>
</li>
<li class=" tabIcon" id="metrics_tab">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#metrics_content" title="Metrics" class=" " role="button" aria-pressed="false">
<span class="iconContainer">
<i class="fa fa-pie-chart"></i>
<strong class="titleIcon">
Metrics 
</strong>
</span>
</a>
</li>
</ul> 
</li>
<li class="dropdown navbar-nav__item navbar-nav__item--ft-sections" id="sectionDropdownContainer">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" title="Navigate fulltext Sections" class="menu-sections dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" aria-disabled="false">
<i class="material-icons">list</i>
Sections
</a>
<ul class="dropdown-menu"><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul>
</li>
</ul>
<ul class="nav navbar-nav navbar-nav--article-nav-right pull-right">
<li class="navbar-nav__item hidden-xs">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485/sidebyside" class="js-sidebyside-open-iframe">
<i class="material-icons lens-view-icon">chrome_reader_mode</i> 
<span class="hidden-xxs">Side by side view</span>
</a>
</li>
<li class="navbar-nav__item">
<div class="accessfulltextlabel hidden">
Access full text
Article
</div>
<div class="download-pdf"><div class="access-options pdfItem-options">
<div class="ft-download-content ft-download-content--pdf">
<form method="POST" action="https://www.microbiologyresearch.org/deliver/fulltext/micro/163/7/961_micro000485.pdf?itemId=%2Fcontent%2Fjournal%2Fmicro%2F10.1099%2Fmic.0.000485&amp;mimeType=pdf&amp;containerItemId=content/journal/micro" target="/content/journal/micro/10.1099/mic.0.000485-pdf" data-title="Download" data-itemid="http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.000485" class="ft-download-content__form ft-download-content__form--pdf js-ft-download-form ">
<i class="fa fa-file-pdf-o 
access-options-icon" aria-hidden="true"></i>
<span class="hidden-xxs" title="Download">PDF</span> 
</form>
</div>
</div></div>
</li>
<li class="dropdown navbar-nav__item navbar-nav__item--keepopen" id="toolsDropdownContainer">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" role="button" id="toolsDropdown" aria-haspopup="true" aria-expanded="false" data-toggle="dropdown"><i class="material-icons">build</i> <span class="hidden-xxs">Tools</span>
</a>
<div class="dropdown-menu">
<div class="tools-nav tools-nav--keepopen">
<div class="panel-heading">
<h3 class="panel-title" id="toolsMenuHeadingInPage"><i class="fa fa-wrench" aria-hidden="true"></i> Tools</h3>
</div>
<div class="panel-body">
<ul class="list-unstyled" aria-labelledby="toolsMenuHeadingInPage">
<li class=" tools-item tools-recommend">
<div class="js-nli-createalert" data-toggle="modal" data-target="#signInToCreateAlert">
<a tabindex="0" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="js-popoverLink" data-container="body" data-toggle="popover" data-placement="top" data-content="Please sign in or register to use this feature" data-original-title="" title="">
<i class="fa fa-heart favouritesIcon fa-fw"></i>
Add to my favourites
</a>
</div> 
</li>
<li class="" role="none">
<div class="js-nli-createalert" data-toggle="modal" data-target="#signInToCreateAlert" role="listitem">
<a tabindex="0" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="js-popoverLink" data-container="body" data-toggle="popover" data-placement="top" data-content="You must be signed in to create this alert!" data-original-title="" title="">
<span class="fa fa-bell fa-fw"></span> Create Publication Alert
</a>
</div>
<div class="js-nli-createalert" data-toggle="modal" data-target="#signInToCreateAlert" role="listitem">
<a tabindex="0" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="js-popoverLink" data-container="body" data-toggle="popover" data-placement="top" data-content="You must be signed in to create this alert!" data-original-title="" title="">
<span class="fa fa-bell fa-fw"></span> Create Citation Alert
</a>
</div>
<div class="js-nli-createalert" data-toggle="modal" data-target="#signInToCreateAlert" role="listitem">
<a tabindex="0" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="js-popoverLink" data-container="body" data-toggle="popover" data-placement="top" data-content="You must be signed in to create this alert!" data-original-title="" title="">
<span class="fa fa-bell fa-fw"></span> Create Correction Alert
</a>
</div>
</li>
<li class="">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="toggle showhide collapsed js-dropdown-toggle" role="button" aria-expanded="false" data-toggle="collapse" title="Link to export citations" aria-controls="export-list0">
<i class="fa fa-paper-plane fa-fw"></i>
Export citation <span class="fa upDownCaret"></span>
</a>
<ul class="flat togglecontent hidden-js-li exportlist list-group collapse" id="export-list0">
<li class="list-group-item">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485/cite/bibtex" title="Link to export in BibTEX format" class="externallink">BibT<sub>E</sub>X</a>
</li>
<li class="list-group-item">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485/cite/endnote" title="Link to export in Endnote format" class="externallink">Endnote</a>
</li>
<li class="list-group-item">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485/cite/bibtex" title="Link to export in Zotero format" class="externallink">Zotero</a>
</li>
<li class="list-group-item">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485/cite/medline" title="Link to export in MEDLARS text" class="externallink">Medlars</a>
</li>
<li class="list-group-item">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485/cite/refworks" title="Link to export in RefWorks format" class="externallink">RefWorks</a>
</li>
<li class="list-group-item">
<a href="http://www.mendeley.com/import/?url=https%3A%2F%2Fwww.microbiologyresearch.org%2Fcontent%2Fjournal%2Fmicro%2F10.1099%2Fmic.0.000485" target="_blank" title="Link to export to Mendeley" class="externallink">Mendeley</a>
</li>
</ul>
</li>
<li class=" tools-item tools-recommend">
<a href="https://www.microbiologyresearch.org/content/journal/micro" title="Link to recommend this journal to a library" data-title="Microbiology" data-issn="1350-0872" data-eissn="1465-2080" id="recommendToLibrarian" data-subj="Microbiology Society"><i class="fa fa-thumbs-o-up fa-fw"></i>
Recommend to library</a>
</li>
<li class=" tools-item tools-permissions">
<a href="https://www.copyright.com/openurl?issn=1350-0872&amp;contentIdType=doi&amp;doi=10.1099%2Fmic.0.000485" title="Link to Reprints and Permissions page" class="toolsprite reprint-permission popup" target="_blank"><i class="fa fa-files-o fa-fw"></i>
Reprints &amp; Permissions</a>
</li>
<li>
</li>
</ul>
</div>
</div>
</div> 
</li>
<li class="dropdown navbar-nav__item navbar-nav__item--keepopen navbar-nav__item--share" id="shareDropdownContainer">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" role="button" id="shareDropdown" aria-haspopup="true" aria-expanded="false" data-toggle="dropdown"><i class="material-icons">share</i> <span class="hidden-xxs"> Share</span></a>
<div class="dropdown-menu ">
<div class="share-nav panel-default shareDropdown">
<div class="panel-heading">
<h3 class="panel-title"><i class="fa fa-share-alt-square"></i> Share</h3>
</div>
<div class="panel-body">
<div class="shareaholic-canvas shareaholic-resolved-canvas shareaholic-ui" data-app="share_buttons" data-app-id="33118683" id="shr_canvas1"><div translate="no">
  <div class="shareaholic-share-buttons-container shareaholic-ui shr-no-print shareaholic-flat shareaholic-round shareaholic- shareaholic-block shareaholic- shareaholic-horizontal shareaholic- shareaholic-left-align  shr-sharebuttons-inpage shareaholic-flat shareaholic-round shareaholic-block shareaholic-horizontal shareaholic-mini" style="">

    <div class="shareaholic-share-buttons-wrapper shareaholic-ui ">
      <div class="shareaholic-share-buttons-animation-wrapper ">
        <div class="shareaholic-share-buttons-heading shareaholic-custom">
          <div class="shareaholic-share-buttons-heading-text" style="line-height: normal; font-style: normal; font-size: 24px; font-family: &quot;Shadows Into Light&quot;; color: rgb(211, 69, 68); --darkreader-inline-color: #c55d5c;" data-darkreader-inline-color="">
            
          </div>
        </div>

        <ul class="shareaholic-share-buttons shareaholic-no-margin-padding-important">

          <li data-service="facebook" class="shareaholic-share-button" style="display:list-item;" title="Facebook">
            <div class="shareaholic-share-button-container shr-outline" role="button">
              <div class="shareaholic-share-button-sizing " style="">
                <a class="shareaholic-service-icon shareaholic-service-facebook shareaholic-service-default" style="" rel="nofollow" aria-label="Share to Facebook" tabindex="0" href="https://www.microbiologyresearch.org/#shr-facebook"></a>
                <span class="shr-share-button-verb" style="color: rgb(255, 255, 255) !important; --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color="">
                  Share
                </span>
              </div>
            </div>
          </li>
          <li data-service="twitter" class="shareaholic-share-button" style="display:list-item;" title="X (Twitter)">
            <div class="shareaholic-share-button-container shr-outline" role="button">
              <div class="shareaholic-share-button-sizing " style="">
                <a class="shareaholic-service-icon shareaholic-service-twitter shareaholic-service-default" style="" rel="nofollow" aria-label="Post to X (Twitter)" tabindex="0" href="https://www.microbiologyresearch.org/#shr-twitter"></a>
                <span class="shr-share-button-verb" style="color: rgb(255, 255, 255) !important; --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color="">
                  Post
                </span>
              </div>
            </div>
          </li>
          <li data-service="linkedin" class="shareaholic-share-button" style="display:list-item;" title="LinkedIn">
            <div class="shareaholic-share-button-container shr-outline" role="button">
              <div class="shareaholic-share-button-sizing " style="">
                <a class="shareaholic-service-icon shareaholic-service-linkedin shareaholic-service-default" style="" rel="nofollow" aria-label="Share to LinkedIn" tabindex="0" href="https://www.microbiologyresearch.org/#shr-linkedin"></a>
                <span class="shr-share-button-verb" style="color: rgb(255, 255, 255) !important; --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color="">
                  Share
                </span>
              </div>
            </div>
          </li>
          <li data-service="email_this" class="shareaholic-share-button" style="display:list-item;" title="Email This">
            <div class="shareaholic-share-button-container shr-outline" role="button">
              <div class="shareaholic-share-button-sizing " style="">
                <a class="shareaholic-service-icon shareaholic-service-email_this shareaholic-service-default" style="" rel="nofollow" aria-label="Mail to Email This" tabindex="0" href="https://www.microbiologyresearch.org/#shr-email_this"></a>
                <span class="shr-share-button-verb" style="color: rgb(255, 255, 255) !important; --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color="">
                  Mail
                </span>
              </div>
            </div>
          </li>
          <li data-service="print" class="shareaholic-share-button" style="display:list-item;" title="Print">
            <div class="shareaholic-share-button-container shr-outline" role="button">
              <div class="shareaholic-share-button-sizing " style="">
                <a class="shareaholic-service-icon shareaholic-service-print shareaholic-service-default" style="" rel="nofollow" aria-label="Print to Print" tabindex="0" href="https://www.microbiologyresearch.org/#shr-print"></a>
                <span class="shr-share-button-verb" style="color: rgb(255, 255, 255) !important; --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color="">
                  Print
                </span>
              </div>
            </div>
          </li>
          <li data-service="all" class="shareaholic-share-button shareaholic-share-button-all" style="display:list-item;" title="More Options">
            <div class="shareaholic-share-button-container shr-outline" role="button">
              <div class="shareaholic-share-button-sizing " style="">
                <a class="shareaholic-service-icon shareaholic-service-all shareaholic-service-default" style="" rel="nofollow" aria-label="Share to More Options" tabindex="0" href="https://www.microbiologyresearch.org/#shr-all"></a>
                <span class="shr-share-button-verb" style="color: rgb(255, 255, 255) !important; --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color="">
                  Share
                </span>
              </div>
            </div>
          </li>
      </ul></div>
    </div>
  </div>
</div>
<style id="shr-social-styles">.shr-debug{font-weight:bolder;cursor:pointer;color:#008000}.shareaholic-canvas{clear:both;-webkit-transition:none;-moz-transition:none;transition:none}.shareaholic-ui button,.shareaholic-ui .shareaholic-ui-button{border:0;box-shadow:none;text-shadow:none;font-size:12px;padding:6px 10px !important;border-radius:3px;background:transparent;display:inline-block;margin:2px;border:1px solid transparent;line-height:16px;color:#555;cursor:pointer;height:initial;text-transform:initial}.shareaholic-ui button:hover,.shareaholic-ui .shareaholic-ui-button:hover{color:#000;font-weight:inherit}.shareaholic-ui button.shareaholic-button-primary,.shareaholic-ui .shareaholic-ui-button.shareaholic-button-primary{background-color:#52a8e8;background-image:-webkit-linear-gradient(top,#52a8e8,#377ad0);border:1px solid #3f81ae;border-color:#3f81ae #2e68a1 #1f5496;box-shadow:inset 0 1px 0 0 #72baeb,0 1px 2px 0 #c3c3c3;color:#fff;font-weight:bold;text-shadow:0 -1px 1px #3275bb;-webkit-appearance:none}.shareaholic-ui button.shareaholic-ui-disabled,.shareaholic-ui .shareaholic-ui-button.shareaholic-ui-disabled{background-color:#ccc;background-image:none;border-color:#bbb;box-shadow:none;color:#999;cursor:default;opacity:.5;text-shadow:none}.shareaholic-ui button.shareaholic-icon-only,.shareaholic-ui .shareaholic-ui-button.shareaholic-icon-only{padding:0}.shareaholic-ui button.shareaholic-icon-only.shareaholic-icon-16,.shareaholic-ui .shareaholic-ui-button.shareaholic-icon-only.shareaholic-icon-16{padding:4px}.shareaholic-ui button.shareaholic-icon-only.shareaholic-icon-16 a,.shareaholic-ui .shareaholic-ui-button.shareaholic-icon-only.shareaholic-icon-16 a{height:16px;width:16px}.shareaholic-ui button.shareaholic-icon-only.shareaholic-icon-24,.shareaholic-ui .shareaholic-ui-button.shareaholic-icon-only.shareaholic-icon-24{padding:0}.shareaholic-ui button.shareaholic-icon-only.shareaholic-icon-24 a,.shareaholic-ui .shareaholic-ui-button.shareaholic-icon-only.shareaholic-icon-24 a{height:24px;width:24px}.shareaholic-ui button.shareaholic-medium,.shareaholic-ui .shareaholic-ui-button.shareaholic-medium{font-size:14px;padding:4px 15px}.shareaholic-ui .shareaholic-ui-icon{display:inline-block;vertical-align:middle;width:16px;height:16px;background-position:50% 50%;background-repeat:no-repeat}.shareaholic-ui .shareaholic-ui-icon.shareaholic-ui-loading-grey{background-image:url("https://cdn.openshareweb.com/v2/images_72e7af4e/spin16_666666_trans.gif")}.shareaholic-ui .shareaholic-ui-icon.shareaholic-ui-logo{width:85px;background-image:url("https://cdn.openshareweb.com/v2/images_72e7af4e/spin16_666666_trans.gif")}.shareaholic-ui input[type="text"],.shareaholic-ui textarea{border:1px solid #ddd;-webkit-transition:all .3s;-moz-transition:all .3s;transition:all .3s;background:#fff none;border-radius:3px;-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;font-size:14px;padding:10px 8px;color:#333}.shareaholic-ui input[type="text"]:focus,.shareaholic-ui textarea:focus{border-color:#3c9ee8;box-shadow:0 0 2px 1px #2695e8;outline:none}.shareaholic-tooltip{font-family:'lucida grande',tahoma,verdana,arial,sans-serif;background-color:#000;text-align:center;padding:3px 8px 3px 8px;color:#fff;font-weight:normal;font-size:12px;-moz-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;z-index:10000000}.shareaholic-media-buttons .shareaholic-share-buttons-container ul.shareaholic-share-buttons{padding:0 !important;margin:0 !important}.shareaholic-show-hide-arrow{height:12px;width:8px;opacity:.5}.shareaholic-hide-container{text-align:left;cursor:pointer}.shareaholic-hide-container.shareaholic-left-alignment{text-align:right}.shareaholic-hide-container.shareaholic-left-alignment .shareaholic-show-hide-arrow{-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg);transform:rotate(180deg);height:12px;width:8px}.shareaholic-show-container{width:100%;height:12px;padding-left:5px;position:absolute;top:15%;cursor:pointer}.shareaholic-show-container.shareaholic-right-alignment{padding-left:0}.shareaholic-show-container.shareaholic-right-alignment .shareaholic-show-hide-arrow{-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg);transform:rotate(180deg);margin-right:5px}.shareaholic-item-quit-left-animation{animation:exit .3s 1;animation-fill-mode:forwards}.shareaholic-item-enter-left-animation{animation:enter .3s 1;animation-fill-mode:forwards}.shareaholic-item-quit-right-animation{animation:exit_right .3s 1;animation-fill-mode:forwards}.shareaholic-item-enter-right-animation{animation:enter_right .3s 1;animation-fill-mode:forwards}.shr-element-hide{display:none;left:-100%}.shr-element-show{display:block;left:0}.shareaholic-share-arrow-version-1{transform:scale(.7,.7) translate(-115%,115%) rotateZ(20deg)}.shareaholic-share-arrow-version-1.shareaholic-mini,.shareaholic-share-arrow-version-1.shareaholic-rectangle{transform:scale(.5,.5) translate(-135%,135%) rotateZ(20deg)}.shareaholic-share-arrow-version-2{transform:scale(.7,.7) translate(-120%,100%) rotateZ(10deg)}.shareaholic-share-arrow-version-2.shareaholic-mini,.shareaholic-share-arrow-version-2.shareaholic-rectangle{transform:scale(.5,.5) translate(-150%,140%) rotateZ(10deg)}.shareaholic-share-arrow-version-3{transform:scale(.7,.7) translate(-120%,120%) rotateZ(15deg)}.shareaholic-share-arrow-version-3.shareaholic-mini,.shareaholic-share-arrow-version-3.shareaholic-rectangle{transform:scale(.5,.5) translate(-140%,140%) rotateZ(15deg)}.shareaholic-share-arrow-version-4{transform:scale(.8,.8) translate(-110%,95%)}.shareaholic-share-arrow-version-4.shareaholic-mini,.shareaholic-share-arrow-version-4.shareaholic-rectangle{transform:scale(.6,.6) translate(-125%,115%)}@keyframes exit{0%{left:0}100%{left:-80px}}@-webkit-keyframes exit{0%{left:0}100%{left:-80px}}@keyframes enter{0%{left:-40px}100%{left:0}}@-webkit-keyframes enter{0%{left:-40px}100%{left:0}}@keyframes exit_right{0%{left:0}100%{left:80px}}@-webkit-keyframes exit_right{0%{left:0}100%{left:80px}}@keyframes enter_right{0%{left:40px}100%{left:0}}@-webkit-keyframes enter_right{0%{left:40px}100%{left:0}}@font-face{font-family:'shareaholic-icons';font-display:block;src:url("https://cdn.openshareweb.com/v2/fonts_72e7af4e/shareaholic-icons.woff2") format("woff2"),url("https://cdn.openshareweb.com/v2/fonts_72e7af4e/shareaholic-icons.woff") format("woff"),url("https://cdn.openshareweb.com/v2/fonts_72e7af4e/shareaholic-icons.ttf") format("truetype"),url("https://cdn.openshareweb.com/v2/fonts_72e7af4e/shareaholic-icons.svg#shareaholic-icons") format("svg")}font-weight normal{font-style:normal}a.shareaholic-service-icon.shareaholic-service-icon.shareaholic-service-default{box-shadow:inset 0 3px 3px -3px #fff;border:1px solid rgba(0,0,0,0.15);border-radius:4px;background-image:none;box-sizing:border-box;position:relative;margin:0;padding:0;overflow:hidden;line-height:1.1em;color:#fff !important;display:block !important;cursor:pointer;font-size:24px;height:40px;width:40px;text-shadow:none;text-decoration:none}a.shareaholic-service-icon.shareaholic-service-icon.shareaholic-service-default:hover,a.shareaholic-service-icon.shareaholic-service-icon.shareaholic-service-default:active{filter:brightness(110%)}a.shareaholic-service-icon.shareaholic-service-icon:before{font-family:'shareaholic-icons' !important;font-style:normal !important;font-weight:normal !important;font-variant:normal !important;line-height:1.1em !important;text-transform:none !important;color:unset;position:absolute;top:7px;left:7px;speak:none;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;display:block}a.shareaholic-service-icon:after{display:block}.shareaholic-service-placeholder{border:3px dotted #d6d6d6;background-color:#f1f2f2}.shareaholic-share-buttons-container .shareaholic-share-button-container ul.shareaholic-share-buttons li.shareaholic-share-button:hover,.shareaholic-share-buttons-container .shareaholic-share-button-container ul.shareaholic-share-buttons li.shareaholic-share-button:active{filter:brightness(110%)}.shareaholic-share-buttons-container.shareaholic-mini li.shareaholic-share-button{background:rgba(0,0,0,0) !important;font-size:14px;height:20px;width:20px}.shareaholic-share-buttons-container.shareaholic-mini li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon{font-size:14px;height:20px;width:20px;border-radius:2px}.shareaholic-share-buttons-container.shareaholic-mini li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon:before{top:2px;left:2px}.shareaholic-share-buttons-container.shareaholic-flat li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon{box-shadow:none;border:none;border-radius:0}.shareaholic-share-buttons-container.shareaholic-flat li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon:before{top:3px;left:3px}.shareaholic-share-buttons-container.shareaholic-flat:not(.shareaholic-mini) li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon:before{left:8px;top:8px}.shareaholic-share-buttons-container.shareaholic-flat.shareaholic-mini:not(.shareaholic-mobile.shareaholic-bottom-align) li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon{font-size:14px}.shareaholic-share-buttons-container.shareaholic-flat.shareaholic-mini:not(.shareaholic-mobile.shareaholic-bottom-align) li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon:before{top:3px;left:3px}.shareaholic-share-buttons-container.shareaholic-flat.shareaholic-large li.shareaholic-share-button .shareaholic-share-button-container{height:70px}.shareaholic-share-buttons-container.shareaholic-flat.shareaholic-large li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon{font-size:32px;height:70px;width:70px}.shareaholic-share-buttons-container.shareaholic-flat.shareaholic-large li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon:before{top:19px;left:19px}.shareaholic-share-buttons-container.shareaholic-round li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon{font-size:24px;border-radius:50%;box-shadow:none;border:none}.shareaholic-share-buttons-container.shareaholic-round li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon:before{top:8px;left:8px}.shareaholic-share-buttons-container.shareaholic-round.shareaholic-mini li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon{font-size:12px;height:20px;width:20px;border-radius:50% !important}.shareaholic-share-buttons-container.shareaholic-round.shareaholic-mini li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon:before{top:4px;left:4px}.shareaholic-share-buttons-container.shareaholic-rectangle li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon{text-align:left;font-size:13px;height:20px;width:70px;border-radius:2px}.shareaholic-share-buttons-container.shareaholic-rectangle li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon:before{top:2px;left:3px}.shareaholic-share-buttons-container.shareaholic-rectangle.shareaholic-flat li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon:before{top:3px;left:4px}.shareaholic-share-buttons-container.shareaholic-rectangle.shareaholic-flat.shareaholic-round li.shareaholic-share-button .shareaholic-share-button-container{border-radius:0}.shareaholic-share-buttons-container.shareaholic-rectangle.shareaholic-flat.shareaholic-round li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon{border-radius:20px}.shareaholic-share-buttons-container.shareaholic-rectangle.shareaholic-flat.shareaholic-round li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-service-icon:before{top:3px;left:7px}.shareaholic-service-default.shareaholic-service-default,.shareaholic-service-default.shareaholic-service-default:hover{background-color:#94a9ac}a.shareaholic-service-default.shareaholic-service-default:before{content:'\e011'}.shareaholic-service-facebook.shareaholic-service-facebook,.shareaholic-service-facebook.shareaholic-service-facebook:hover{background-color:#0866ff}a.shareaholic-service-facebook.shareaholic-service-facebook:before{content:'\e01d'}.shareaholic-service-digg.shareaholic-service-digg,.shareaholic-service-digg.shareaholic-service-digg:hover{background-color:#080808}a.shareaholic-service-digg.shareaholic-service-digg:before{content:'\e013'}.shareaholic-service-twitter.shareaholic-service-twitter,.shareaholic-service-twitter.shareaholic-service-twitter:hover{background-color:#0f1419}a.shareaholic-service-twitter.shareaholic-service-twitter:before{content:'\e06f'}.shareaholic-service-instapaper.shareaholic-service-instapaper,.shareaholic-service-instapaper.shareaholic-service-instapaper:hover{background-color:#414042}a.shareaholic-service-instapaper.shareaholic-service-instapaper:before{content:'\e032'}.shareaholic-service-diigo.shareaholic-service-diigo,.shareaholic-service-diigo.shareaholic-service-diigo:hover{background-color:#0b5fa0}a.shareaholic-service-diigo.shareaholic-service-diigo:before{content:'\e014'}.shareaholic-service-meneame.shareaholic-service-meneame,.shareaholic-service-meneame.shareaholic-service-meneame:hover{background-color:#f56b36}a.shareaholic-service-meneame.shareaholic-service-meneame:before{content:'\e03f'}.shareaholic-service-reddit.shareaholic-service-reddit,.shareaholic-service-reddit.shareaholic-service-reddit:hover{background-color:#ff5700}a.shareaholic-service-reddit.shareaholic-service-reddit:before{content:'\e053'}.shareaholic-service-google_mail.shareaholic-service-google_mail,.shareaholic-service-google_mail.shareaholic-service-google_mail:hover{background-color:#ea4335}a.shareaholic-service-google_mail.shareaholic-service-google_mail:before{content:'\e02d'}.shareaholic-service-hotmail.shareaholic-service-hotmail,.shareaholic-service-hotmail.shareaholic-service-hotmail:hover{background-color:#0072c6}a.shareaholic-service-hotmail.shareaholic-service-hotmail:before{content:'\e02f'}.shareaholic-service-yahoo.shareaholic-service-yahoo,.shareaholic-service-yahoo.shareaholic-service-yahoo:hover{background-color:#9b2c96}a.shareaholic-service-yahoo.shareaholic-service-yahoo:before{content:'\e07c'}.shareaholic-service-aol_mail.shareaholic-service-aol_mail,.shareaholic-service-aol_mail.shareaholic-service-aol_mail:hover{background-color:#a70a4f}a.shareaholic-service-aol_mail.shareaholic-service-aol_mail:before{content:'\e006'}.shareaholic-service-fark.shareaholic-service-fark,.shareaholic-service-fark.shareaholic-service-fark:hover{background-color:#706da3}a.shareaholic-service-fark.shareaholic-service-fark:before{content:'\e021'}.shareaholic-service-tumblr.shareaholic-service-tumblr,.shareaholic-service-tumblr.shareaholic-service-tumblr:hover{background-color:#35465d}a.shareaholic-service-tumblr.shareaholic-service-tumblr:before{content:'\e06d'}.shareaholic-service-livejournal.shareaholic-service-livejournal,.shareaholic-service-livejournal.shareaholic-service-livejournal:hover{background-color:#00b0ea}a.shareaholic-service-livejournal.shareaholic-service-livejournal:before{content:'\e039'}.shareaholic-service-linkedin.shareaholic-service-linkedin,.shareaholic-service-linkedin.shareaholic-service-linkedin:hover{background-color:#0a66c2}a.shareaholic-service-linkedin.shareaholic-service-linkedin:before{content:'\e038'}.shareaholic-service-viadeo.shareaholic-service-viadeo,.shareaholic-service-viadeo.shareaholic-service-viadeo:hover{background-color:#000}a.shareaholic-service-viadeo.shareaholic-service-viadeo:before{content:'\e071'}.shareaholic-service-itunes_podcast.shareaholic-service-itunes_podcast,.shareaholic-service-itunes_podcast.shareaholic-service-itunes_podcast:hover{background-color:#794696}a.shareaholic-service-itunes_podcast.shareaholic-service-itunes_podcast:before{content:'\e033'}.shareaholic-service-evernote.shareaholic-service-evernote,.shareaholic-service-evernote.shareaholic-service-evernote:hover{background-color:#6bb130}a.shareaholic-service-evernote.shareaholic-service-evernote:before{content:'\e01c'}.shareaholic-service-stumpedia.shareaholic-service-stumpedia,.shareaholic-service-stumpedia.shareaholic-service-stumpedia:hover{background-color:#fd9d0f}a.shareaholic-service-stumpedia.shareaholic-service-stumpedia:before{content:'\e062'}.shareaholic-service-amazon.shareaholic-service-amazon,.shareaholic-service-amazon.shareaholic-service-amazon:hover{background-color:#000}a.shareaholic-service-amazon.shareaholic-service-amazon:before{content:'\e005'}.shareaholic-service-email_this.shareaholic-service-email_this,.shareaholic-service-email_this.shareaholic-service-email_this:hover{background-color:#939598}a.shareaholic-service-email_this.shareaholic-service-email_this:before{content:'\e01a'}.shareaholic-service-yc_hacker_news.shareaholic-service-yc_hacker_news,.shareaholic-service-yc_hacker_news.shareaholic-service-yc_hacker_news:hover{background-color:#fd7c3a}a.shareaholic-service-yc_hacker_news.shareaholic-service-yc_hacker_news:before{content:'\e07e'}.shareaholic-service-techmeme.shareaholic-service-techmeme,.shareaholic-service-techmeme.shareaholic-service-techmeme:hover{background-color:#174a61}a.shareaholic-service-techmeme.shareaholic-service-techmeme:before{content:'\e063'}.shareaholic-service-bit_ly.shareaholic-service-bit_ly,.shareaholic-service-bit_ly.shareaholic-service-bit_ly:hover{background-color:#ee6123}a.shareaholic-service-bit_ly.shareaholic-service-bit_ly:before{content:'\e00a'}.shareaholic-service-we_heart_it.shareaholic-service-we_heart_it,.shareaholic-service-we_heart_it.shareaholic-service-we_heart_it:hover{background-color:#fe7da3}a.shareaholic-service-we_heart_it.shareaholic-service-we_heart_it:before{content:'\e075'}.shareaholic-service-plurk.shareaholic-service-plurk,.shareaholic-service-plurk.shareaholic-service-plurk:hover{background-color:#cf682f}a.shareaholic-service-plurk.shareaholic-service-plurk:before{content:'\e04c'}.shareaholic-service-blogger_post.shareaholic-service-blogger_post,.shareaholic-service-blogger_post.shareaholic-service-blogger_post:hover{background-color:#fd792f}a.shareaholic-service-blogger_post.shareaholic-service-blogger_post:before{content:'\e00b'}.shareaholic-service-typepad_post.shareaholic-service-typepad_post,.shareaholic-service-typepad_post.shareaholic-service-typepad_post:hover{background-color:#becb4f}a.shareaholic-service-typepad_post.shareaholic-service-typepad_post:before{content:'\e070'}.shareaholic-service-tinyurl.shareaholic-service-tinyurl,.shareaholic-service-tinyurl.shareaholic-service-tinyurl:hover{background-color:#009}a.shareaholic-service-tinyurl.shareaholic-service-tinyurl:before{content:'\e069'}.shareaholic-service-wordpress_blog.shareaholic-service-wordpress_blog,.shareaholic-service-wordpress_blog.shareaholic-service-wordpress_blog:hover{background-color:#58585b}a.shareaholic-service-wordpress_blog.shareaholic-service-wordpress_blog:before{content:'\e079'}.shareaholic-service-printfriendly.shareaholic-service-printfriendly,.shareaholic-service-printfriendly.shareaholic-service-printfriendly:hover{background-color:#00b0f0}a.shareaholic-service-printfriendly.shareaholic-service-printfriendly:before{content:'\e04f'}.shareaholic-service-pocket.shareaholic-service-pocket,.shareaholic-service-pocket.shareaholic-service-pocket:hover{background-color:#ee4055}a.shareaholic-service-pocket.shareaholic-service-pocket:before{content:'\e04d'}.shareaholic-service-box_net.shareaholic-service-box_net,.shareaholic-service-box_net.shareaholic-service-box_net:hover{background-color:#1c88e9}a.shareaholic-service-box_net.shareaholic-service-box_net:before{content:'\e00d'}.shareaholic-service-hatena.shareaholic-service-hatena,.shareaholic-service-hatena.shareaholic-service-hatena:hover{background-color:#00a6db}a.shareaholic-service-hatena.shareaholic-service-hatena:before{content:'\e02e'}.shareaholic-service-yammer.shareaholic-service-yammer,.shareaholic-service-yammer.shareaholic-service-yammer:hover{background-color:#45b0d3}a.shareaholic-service-yammer.shareaholic-service-yammer:before{content:'\e07d'}.shareaholic-service-pinboard_in.shareaholic-service-pinboard_in,.shareaholic-service-pinboard_in.shareaholic-service-pinboard_in:hover{background-color:#0c52be}a.shareaholic-service-pinboard_in.shareaholic-service-pinboard_in:before{content:'\e04a'}.shareaholic-service-google_apps_mail.shareaholic-service-google_apps_mail,.shareaholic-service-google_apps_mail.shareaholic-service-google_apps_mail:hover{background-color:#58595b}a.shareaholic-service-google_apps_mail.shareaholic-service-google_apps_mail:before{content:'\e02b'}.shareaholic-service-wykop.shareaholic-service-wykop,.shareaholic-service-wykop.shareaholic-service-wykop:hover{background-color:#f48339}a.shareaholic-service-wykop.shareaholic-service-wykop:before{content:'\e07a'}.shareaholic-service-facebook_messenger.shareaholic-service-facebook_messenger,.shareaholic-service-facebook_messenger.shareaholic-service-facebook_messenger:hover{background-color:#0866ff}a.shareaholic-service-facebook_messenger.shareaholic-service-facebook_messenger:before{content:'\e020'}.shareaholic-service-buffer.shareaholic-service-buffer,.shareaholic-service-buffer.shareaholic-service-buffer:hover{background-color:#231f20}a.shareaholic-service-buffer.shareaholic-service-buffer:before{content:'\e00e'}.shareaholic-service-pinterest.shareaholic-service-pinterest,.shareaholic-service-pinterest.shareaholic-service-pinterest:hover{background-color:#bd081c}a.shareaholic-service-pinterest.shareaholic-service-pinterest:before{content:'\e04b'}.shareaholic-service-flipboard.shareaholic-service-flipboard,.shareaholic-service-flipboard.shareaholic-service-flipboard:hover{background-color:#b01e1f}a.shareaholic-service-flipboard.shareaholic-service-flipboard:before{content:'\e026'}.shareaholic-service-vk.shareaholic-service-vk,.shareaholic-service-vk.shareaholic-service-vk:hover{background-color:#4d75a3}a.shareaholic-service-vk.shareaholic-service-vk:before{content:'\e074'}.shareaholic-service-yummly.shareaholic-service-yummly,.shareaholic-service-yummly.shareaholic-service-yummly:hover{background-color:#e06331}a.shareaholic-service-yummly.shareaholic-service-yummly:before{content:'\e081'}.shareaholic-service-houzz.shareaholic-service-houzz,.shareaholic-service-houzz.shareaholic-service-houzz:hover{background-color:#4dbc15}a.shareaholic-service-houzz.shareaholic-service-houzz:before{content:'\e030'}.shareaholic-service-instagram.shareaholic-service-instagram,.shareaholic-service-instagram.shareaholic-service-instagram:hover{background-color:#e4405f}a.shareaholic-service-instagram.shareaholic-service-instagram:before{content:'\e031'}.shareaholic-service-rss.shareaholic-service-rss,.shareaholic-service-rss.shareaholic-service-rss:hover{background-color:#fd882a}a.shareaholic-service-rss.shareaholic-service-rss:before{content:'\e055'}.shareaholic-service-youtube.shareaholic-service-youtube,.shareaholic-service-youtube.shareaholic-service-youtube:hover{background-color:#cd201f}a.shareaholic-service-youtube.shareaholic-service-youtube:before{content:'\e080'}.shareaholic-service-vimeo.shareaholic-service-vimeo,.shareaholic-service-vimeo.shareaholic-service-vimeo:hover{background-color:#1ab7ea}a.shareaholic-service-vimeo.shareaholic-service-vimeo:before{content:'\e073'}.shareaholic-service-flickr.shareaholic-service-flickr,.shareaholic-service-flickr.shareaholic-service-flickr:hover{background-color:#ff0084}a.shareaholic-service-flickr.shareaholic-service-flickr:before{content:'\e025'}.shareaholic-service-foursquare.shareaholic-service-foursquare,.shareaholic-service-foursquare.shareaholic-service-foursquare:hover{background-color:#f94877}a.shareaholic-service-foursquare.shareaholic-service-foursquare:before{content:'\e027'}.shareaholic-service-yelp.shareaholic-service-yelp,.shareaholic-service-yelp.shareaholic-service-yelp:hover{background-color:#c41200}a.shareaholic-service-yelp.shareaholic-service-yelp:before{content:'\e07f'}.shareaholic-service-bloglovin.shareaholic-service-bloglovin,.shareaholic-service-bloglovin.shareaholic-service-bloglovin:hover{background-color:#4cd1fa}a.shareaholic-service-bloglovin.shareaholic-service-bloglovin:before{content:'\e00c'}.shareaholic-service-dribbble.shareaholic-service-dribbble,.shareaholic-service-dribbble.shareaholic-service-dribbble:hover{background-color:#ea4c89}a.shareaholic-service-dribbble.shareaholic-service-dribbble:before{content:'\e018'}.shareaholic-service-odnoklassniki.shareaholic-service-odnoklassniki,.shareaholic-service-odnoklassniki.shareaholic-service-odnoklassniki:hover{background-color:#e67e17}a.shareaholic-service-odnoklassniki.shareaholic-service-odnoklassniki:before{content:'\e045'}.shareaholic-service-xing.shareaholic-service-xing,.shareaholic-service-xing.shareaholic-service-xing:hover{background-color:#026466}a.shareaholic-service-xing.shareaholic-service-xing:before{content:'\e07b'}.shareaholic-service-weibo.shareaholic-service-weibo,.shareaholic-service-weibo.shareaholic-service-weibo:hover{background-color:#d82829}a.shareaholic-service-weibo.shareaholic-service-weibo:before{content:'\e077'}.shareaholic-service-mixi.shareaholic-service-mixi,.shareaholic-service-mixi.shareaholic-service-mixi:hover{background-color:#d1ad5a}a.shareaholic-service-mixi.shareaholic-service-mixi:before{content:'\e043'}.shareaholic-service-kindle_it.shareaholic-service-kindle_it,.shareaholic-service-kindle_it.shareaholic-service-kindle_it:hover{background-color:#f90}a.shareaholic-service-kindle_it.shareaholic-service-kindle_it:before{content:'\e035'}.shareaholic-service-onenote.shareaholic-service-onenote,.shareaholic-service-onenote.shareaholic-service-onenote:hover{background-color:#80397b}a.shareaholic-service-onenote.shareaholic-service-onenote:before{content:'\e046'}.shareaholic-service-whatsapp.shareaholic-service-whatsapp,.shareaholic-service-whatsapp.shareaholic-service-whatsapp:hover{background-color:#29a71a}a.shareaholic-service-whatsapp.shareaholic-service-whatsapp:before{content:'\e078'}.shareaholic-service-sms.shareaholic-service-sms,.shareaholic-service-sms.shareaholic-service-sms:hover{background-color:#1598d5}a.shareaholic-service-sms.shareaholic-service-sms:before{content:'\e05b'}.shareaholic-service-etsy.shareaholic-service-etsy,.shareaholic-service-etsy.shareaholic-service-etsy:hover{background-color:#e06331}a.shareaholic-service-etsy.shareaholic-service-etsy:before{content:'\e01b'}.shareaholic-service-github.shareaholic-service-github,.shareaholic-service-github.shareaholic-service-github:hover{background-color:#181616}a.shareaholic-service-github.shareaholic-service-github:before{content:'\e02a'}.shareaholic-service-tripadvisor.shareaholic-service-tripadvisor,.shareaholic-service-tripadvisor.shareaholic-service-tripadvisor:hover{background-color:#34e0a1}a.shareaholic-service-tripadvisor.shareaholic-service-tripadvisor:before{content:'\e06b'}.shareaholic-service-slideshare.shareaholic-service-slideshare,.shareaholic-service-slideshare.shareaholic-service-slideshare:hover{background-color:#0177b5}a.shareaholic-service-slideshare.shareaholic-service-slideshare:before{content:'\e05a'}.shareaholic-service-stackoverflow.shareaholic-service-stackoverflow,.shareaholic-service-stackoverflow.shareaholic-service-stackoverflow:hover{background-color:#fe7a15}a.shareaholic-service-stackoverflow.shareaholic-service-stackoverflow:before{content:'\e05f'}.shareaholic-service-about_me.shareaholic-service-about_me,.shareaholic-service-about_me.shareaholic-service-about_me:hover{background-color:#054a76}a.shareaholic-service-about_me.shareaholic-service-about_me:before{content:'\e000'}.shareaholic-service-behance.shareaholic-service-behance,.shareaholic-service-behance.shareaholic-service-behance:hover{background-color:#000}a.shareaholic-service-behance.shareaholic-service-behance:before{content:'\e009'}.shareaholic-service-soundcloud.shareaholic-service-soundcloud,.shareaholic-service-soundcloud.shareaholic-service-soundcloud:hover{background-color:#f30}a.shareaholic-service-soundcloud.shareaholic-service-soundcloud:before{content:'\e05d'}.shareaholic-service-ebay.shareaholic-service-ebay,.shareaholic-service-ebay.shareaholic-service-ebay:hover{background-color:#f00}a.shareaholic-service-ebay.shareaholic-service-ebay:before{content:'\e019'}.shareaholic-service-five_hundred_px.shareaholic-service-five_hundred_px,.shareaholic-service-five_hundred_px.shareaholic-service-five_hundred_px:hover{background-color:#1a171b}a.shareaholic-service-five_hundred_px.shareaholic-service-five_hundred_px:before{content:'\e024'}.shareaholic-service-bbb.shareaholic-service-bbb,.shareaholic-service-bbb.shareaholic-service-bbb:hover{background-color:#002b80}a.shareaholic-service-bbb.shareaholic-service-bbb:before{content:'\e008'}.shareaholic-service-skype.shareaholic-service-skype,.shareaholic-service-skype.shareaholic-service-skype:hover{background-color:#00aff0}a.shareaholic-service-skype.shareaholic-service-skype:before{content:'\e058'}.shareaholic-service-google_classroom.shareaholic-service-google_classroom,.shareaholic-service-google_classroom.shareaholic-service-google_classroom:hover{background-color:#25a667}a.shareaholic-service-google_classroom.shareaholic-service-google_classroom:before{content:'\e02c'}.shareaholic-service-zillow_digs.shareaholic-service-zillow_digs,.shareaholic-service-zillow_digs.shareaholic-service-zillow_digs:hover{background-color:#06c}a.shareaholic-service-zillow_digs.shareaholic-service-zillow_digs:before{content:'\e001'}.shareaholic-service-disqus.shareaholic-service-disqus,.shareaholic-service-disqus.shareaholic-service-disqus:hover{background-color:#2e9fff}a.shareaholic-service-disqus.shareaholic-service-disqus:before{content:'\e016'}.shareaholic-service-feedly.shareaholic-service-feedly,.shareaholic-service-feedly.shareaholic-service-feedly:hover{background-color:#2bb24c}a.shareaholic-service-feedly.shareaholic-service-feedly:before{content:'\e023'}.shareaholic-service-spotify.shareaholic-service-spotify,.shareaholic-service-spotify.shareaholic-service-spotify:hover{background-color:#23cf5f}a.shareaholic-service-spotify.shareaholic-service-spotify:before{content:'\e05e'}.shareaholic-service-baidu.shareaholic-service-baidu,.shareaholic-service-baidu.shareaholic-service-baidu:hover{background-color:#006cb7}a.shareaholic-service-baidu.shareaholic-service-baidu:before{content:'\e007'}.shareaholic-service-line_me.shareaholic-service-line_me,.shareaholic-service-line_me.shareaholic-service-line_me:hover{background-color:#00c300}a.shareaholic-service-line_me.shareaholic-service-line_me:before{content:'\e037'}.shareaholic-service-print.shareaholic-service-print,.shareaholic-service-print.shareaholic-service-print:hover{background-color:#58595b}a.shareaholic-service-print.shareaholic-service-print:before{content:'\e04e'}.shareaholic-service-mix.shareaholic-service-mix,.shareaholic-service-mix.shareaholic-service-mix:hover{background-color:#e06331}a.shareaholic-service-mix.shareaholic-service-mix:before{content:'\e042'}.shareaholic-service-telegram.shareaholic-service-telegram,.shareaholic-service-telegram.shareaholic-service-telegram:hover{background-color:#2ca5e0}a.shareaholic-service-telegram.shareaholic-service-telegram:before{content:'\e064'}.shareaholic-service-slack.shareaholic-service-slack,.shareaholic-service-slack.shareaholic-service-slack:hover{background-color:#563c82}a.shareaholic-service-slack.shareaholic-service-slack:before{content:'\e059'}.shareaholic-service-patreon.shareaholic-service-patreon,.shareaholic-service-patreon.shareaholic-service-patreon:hover{background-color:#f96854}a.shareaholic-service-patreon.shareaholic-service-patreon:before{content:'\e048'}.shareaholic-service-snapchat.shareaholic-service-snapchat,.shareaholic-service-snapchat.shareaholic-service-snapchat:hover{background-color:#ffdf00}a.shareaholic-service-snapchat.shareaholic-service-snapchat:before{content:'\e05c'}.shareaholic-service-medium.shareaholic-service-medium,.shareaholic-service-medium.shareaholic-service-medium:hover{background-color:#000}a.shareaholic-service-medium.shareaholic-service-medium:before{content:'\e03d'}.shareaholic-service-strava.shareaholic-service-strava,.shareaholic-service-strava.shareaholic-service-strava:hover{background-color:#fc4c02}a.shareaholic-service-strava.shareaholic-service-strava:before{content:'\e061'}.shareaholic-service-meetup.shareaholic-service-meetup,.shareaholic-service-meetup.shareaholic-service-meetup:hover{background-color:#e0393e}a.shareaholic-service-meetup.shareaholic-service-meetup:before{content:'\e03e'}.shareaholic-service-viber.shareaholic-service-viber,.shareaholic-service-viber.shareaholic-service-viber:hover{background-color:#7c529e}a.shareaholic-service-viber.shareaholic-service-viber:before{content:'\e072'}.shareaholic-service-trello.shareaholic-service-trello,.shareaholic-service-trello.shareaholic-service-trello:hover{background-color:#0079bf}a.shareaholic-service-trello.shareaholic-service-trello:before{content:'\e06a'}.shareaholic-service-wechat.shareaholic-service-wechat,.shareaholic-service-wechat.shareaholic-service-wechat:hover{background-color:#7bb32e}a.shareaholic-service-wechat.shareaholic-service-wechat:before{content:'\e076'}.shareaholic-service-twitch.shareaholic-service-twitch,.shareaholic-service-twitch.shareaholic-service-twitch:hover{background-color:#9146ff}a.shareaholic-service-twitch.shareaholic-service-twitch:before{content:'\e06e'}.shareaholic-service-mewe.shareaholic-service-mewe,.shareaholic-service-mewe.shareaholic-service-mewe:hover{background-color:#007da1}a.shareaholic-service-mewe.shareaholic-service-mewe:before{content:'\e040'}.shareaholic-service-parler.shareaholic-service-parler,.shareaholic-service-parler.shareaholic-service-parler:hover{background-color:#892e5e}a.shareaholic-service-parler.shareaholic-service-parler:before{content:'\e047'}.shareaholic-service-tiktok.shareaholic-service-tiktok,.shareaholic-service-tiktok.shareaholic-service-tiktok:hover{background-color:#000}a.shareaholic-service-tiktok.shareaholic-service-tiktok:before{content:'\e068'}.shareaholic-service-gab.shareaholic-service-gab,.shareaholic-service-gab.shareaholic-service-gab:hover{background-color:#30ce7d}a.shareaholic-service-gab.shareaholic-service-gab:before{content:'\e028'}.shareaholic-service-copy_link.shareaholic-service-copy_link,.shareaholic-service-copy_link.shareaholic-service-copy_link:hover{background-color:#555}a.shareaholic-service-copy_link.shareaholic-service-copy_link:before{content:'\e010'}.shareaholic-service-microsoft_teams.shareaholic-service-microsoft_teams,.shareaholic-service-microsoft_teams.shareaholic-service-microsoft_teams:hover{background-color:#5059c9}a.shareaholic-service-microsoft_teams.shareaholic-service-microsoft_teams:before{content:'\e041'}.shareaholic-service-mail_ru.shareaholic-service-mail_ru,.shareaholic-service-mail_ru.shareaholic-service-mail_ru:hover{background-color:#356fac}a.shareaholic-service-mail_ru.shareaholic-service-mail_ru:before{content:'\e03b'}.shareaholic-service-kakao.shareaholic-service-kakao,.shareaholic-service-kakao.shareaholic-service-kakao:hover{background-color:#fee500}a.shareaholic-service-kakao.shareaholic-service-kakao:before{content:'\e034'}.shareaholic-service-stocktwits.shareaholic-service-stocktwits,.shareaholic-service-stocktwits.shareaholic-service-stocktwits:hover{background-color:#218dff}a.shareaholic-service-stocktwits.shareaholic-service-stocktwits:before{content:'\e060'}.shareaholic-service-discord.shareaholic-service-discord,.shareaholic-service-discord.shareaholic-service-discord:hover{background-color:#5865f2}a.shareaholic-service-discord.shareaholic-service-discord:before{content:'\e015'}.shareaholic-service-douban.shareaholic-service-douban,.shareaholic-service-douban.shareaholic-service-douban:hover{background-color:#071}a.shareaholic-service-douban.shareaholic-service-douban:before{content:'\e017'}.shareaholic-service-renren.shareaholic-service-renren,.shareaholic-service-renren.shareaholic-service-renren:hover{background-color:#005eac}a.shareaholic-service-renren.shareaholic-service-renren:before{content:'\e054'}.shareaholic-service-diaspora.shareaholic-service-diaspora,.shareaholic-service-diaspora.shareaholic-service-diaspora:hover{background-color:#2e3436}a.shareaholic-service-diaspora.shareaholic-service-diaspora:before{content:'\e012'}.shareaholic-service-naver.shareaholic-service-naver,.shareaholic-service-naver.shareaholic-service-naver:hover{background-color:#07bc5e}a.shareaholic-service-naver.shareaholic-service-naver:before{content:'\e044'}.shareaholic-service-qzone.shareaholic-service-qzone,.shareaholic-service-qzone.shareaholic-service-qzone:hover{background-color:#2b82d9}a.shareaholic-service-qzone.shareaholic-service-qzone:before{content:'\e051'}.shareaholic-service-koo.shareaholic-service-koo,.shareaholic-service-koo.shareaholic-service-koo:hover{background-color:#facd18}a.shareaholic-service-koo.shareaholic-service-koo:before{content:'\e036'}.shareaholic-service-mastodon.shareaholic-service-mastodon,.shareaholic-service-mastodon.shareaholic-service-mastodon:hover{background-color:#563acc}a.shareaholic-service-mastodon.shareaholic-service-mastodon:before{content:'\e03c'}.shareaholic-service-threads.shareaholic-service-threads,.shareaholic-service-threads.shareaholic-service-threads:hover{background-color:#000}a.shareaholic-service-threads.shareaholic-service-threads:before{content:'\e065'}.shareaholic-service-truthsocial.shareaholic-service-truthsocial,.shareaholic-service-truthsocial.shareaholic-service-truthsocial:hover{background-color:#6700ed}a.shareaholic-service-truthsocial.shareaholic-service-truthsocial:before{content:'\e06c'}.shareaholic-service-gettr.shareaholic-service-gettr,.shareaholic-service-gettr.shareaholic-service-gettr:hover{background-color:#e50000}a.shareaholic-service-gettr.shareaholic-service-gettr:before{content:'\e029'}.shareaholic-service-threema.shareaholic-service-threema,.shareaholic-service-threema.shareaholic-service-threema:hover{background-color:#1a1a1a}a.shareaholic-service-threema.shareaholic-service-threema:before{content:'\e067'}.shareaholic-service-rakuten.shareaholic-service-rakuten,.shareaholic-service-rakuten.shareaholic-service-rakuten:hover{background-color:#bf0000}a.shareaholic-service-rakuten.shareaholic-service-rakuten:before{content:'\e052'}.shareaholic-service-shareaholic.shareaholic-service-shareaholic,.shareaholic-service-shareaholic.shareaholic-service-shareaholic:hover{background-color:#95d600}a.shareaholic-service-shareaholic.shareaholic-service-shareaholic:before{content:'\e056'}.shareaholic-service-shareaholic_archive.shareaholic-service-shareaholic_archive,.shareaholic-service-shareaholic_archive.shareaholic-service-shareaholic_archive:hover{background-color:#ca0814}a.shareaholic-service-shareaholic_archive.shareaholic-service-shareaholic_archive:before{content:'\e057'}.shareaholic-service-favorite.shareaholic-service-favorite,.shareaholic-service-favorite.shareaholic-service-favorite:hover{background-color:#ca0814}a.shareaholic-service-favorite.shareaholic-service-favorite:before{content:'\e022'}.shareaholic-service-paypal.shareaholic-service-paypal,.shareaholic-service-paypal.shareaholic-service-paypal:hover{background-color:#080808}a.shareaholic-service-paypal.shareaholic-service-paypal:before{content:'\e049'}.shareaholic-service-buymeacoffee.shareaholic-service-buymeacoffee,.shareaholic-service-buymeacoffee.shareaholic-service-buymeacoffee:hover{background-color:#080808}a.shareaholic-service-buymeacoffee.shareaholic-service-buymeacoffee:before{content:'\e00f'}.shareaholic-service-all.shareaholic-service-all,.shareaholic-service-all.shareaholic-service-all:hover{background-color:#45a147}a.shareaholic-service-all.shareaholic-service-all:before{content:'\e002'}.shareaholic-service-all_ios.shareaholic-service-all_ios,.shareaholic-service-all_ios.shareaholic-service-all_ios:hover{background-color:#080808}a.shareaholic-service-all_ios.shareaholic-service-all_ios:before{content:'\e004'}.shareaholic-service-all_droid.shareaholic-service-all_droid,.shareaholic-service-all_droid.shareaholic-service-all_droid:hover{background-color:#080808}a.shareaholic-service-all_droid.shareaholic-service-all_droid:before{content:'\e003'}.shareaholic-service-three_dots.shareaholic-service-three_dots,.shareaholic-service-three_dots.shareaholic-service-three_dots:hover{background-color:#080808}a.shareaholic-service-three_dots.shareaholic-service-three_dots:before{content:'\e066'}.shareaholic-service-facebook_like.shareaholic-service-facebook_like,.shareaholic-service-facebook_like.shareaholic-service-facebook_like:hover{background-color:#0866ff}a.shareaholic-service-facebook_like.shareaholic-service-facebook_like:before{content:'\e01f'}.shareaholic-service-facebook_alt.shareaholic-service-facebook_alt,.shareaholic-service-facebook_alt.shareaholic-service-facebook_alt:hover{background-color:#0866ff}a.shareaholic-service-facebook_alt.shareaholic-service-facebook_alt:before{content:'\e01e'}.shareaholic-floated-share-buttons-container{display:none}@media only screen and (min-width:700px) and (orientation:landscape){.shareaholic-floated-share-buttons-container{display:inherit}}@media only screen and (min-width:400px) and (orientation:portrait){.shareaholic-floated-share-buttons-container{display:inherit}}.shareaholic-share-buttons-container{margin:0 auto;position:relative;font-family:'lucida grande',tahoma,verdana,arial,sans-serif;-webkit-transition:all .25s;-moz-transition:all .25s;transition:all .25s}.shareaholic-share-buttons-container li.shareaholic-total-count{font-size:10px !important;line-height:1.4em;margin:12px 20px 0 0 !important;padding:0 !important;text-align:center;height:40px;left:14px;top:-7px;position:relative;float:left;list-style:none !important;background:none !important;display:list-item}.shareaholic-share-buttons-container li.shareaholic-total-count:before{display:none}.shareaholic-share-buttons-container li.shareaholic-total-count .shareaholic-total-count-bg{width:100%;height:100%;position:absolute}.shareaholic-share-buttons-container li.shareaholic-total-count .shareaholic-total-share-count{position:relative;top:4px;font-size:14px;font-weight:600;font-variant-numeric:tabular-nums;min-width:45px}.shareaholic-share-buttons-container li.shareaholic-total-count .shareaholic-total-share-count.shareaholic-no-label{top:0 !important}.shareaholic-share-buttons-container li.shareaholic-total-count .shareaholic-shares-label{position:relative;top:4px;padding:0 2px}.shareaholic-share-buttons-container.shareaholic-diglett li.shareaholic-total-count{height:30px;top:3px}.shareaholic-share-buttons-container.shareaholic-diglett li.shareaholic-total-count .shareaholic-total-share-count{top:1px}.shareaholic-share-buttons-container.shareaholic-diglett li.shareaholic-total-count .shareaholic-shares-label{top:2px}.shareaholic-share-buttons-container.shareaholic-diglett.shareaholic-vertical li.shareaholic-total-count{margin-bottom:5px}.shareaholic-share-buttons-container.shareaholic-diglett.shareaholic-vertical li.shareaholic-total-count .shareaholic-total-share-count{top:0}.shareaholic-share-buttons-container.shareaholic-vertical li.shareaholic-total-count{float:none;left:0;margin:0 auto !important}.shareaholic-share-buttons-container.shareaholic-vertical.shareaholic-top-counter li.shareaholic-total-count{top:0}.shareaholic-share-buttons-container.shareaholic-rectangle:not(.shareaholic-vertical) li.shareaholic-total-count{height:20px;top:-7px;padding:0 5px}.shareaholic-share-buttons-container.shareaholic-rectangle:not(.shareaholic-vertical) li.shareaholic-total-count .shareaholic-total-share-count{display:inline;vertical-align:middle;line-height:20px;top:auto}.shareaholic-share-buttons-container.shareaholic-rectangle:not(.shareaholic-vertical) li.shareaholic-total-count .shareaholic-shares-label{display:inline;vertical-align:middle;line-height:20px;top:auto}.shareaholic-share-buttons-container.shareaholic-rectangle:not(.shareaholic-vertical).shareaholic-top-counter li.shareaholic-total-count{top:28px}.shareaholic-share-buttons-container.shareaholic-mini:not(.shareaholic-vertical) li.shareaholic-total-count{height:20px;top:-7px;padding:0 5px}.shareaholic-share-buttons-container.shareaholic-mini:not(.shareaholic-vertical) li.shareaholic-total-count .shareaholic-total-share-count{display:inline;vertical-align:middle;line-height:20px;top:auto}.shareaholic-share-buttons-container.shareaholic-mini:not(.shareaholic-vertical) li.shareaholic-total-count .shareaholic-shares-label{display:inline;vertical-align:middle;line-height:20px;top:auto}.shareaholic-share-buttons-container.shareaholic-top-counter li.shareaholic-total-count{top:28px}.shareaholic-share-buttons-container.shareaholic-floated li.shareaholic-total-count{margin-bottom:-7px !important;line-height:40px}.shareaholic-share-buttons-container.shareaholic-floated li.shareaholic-total-count .shareaholic-total-share-count{top:-6px}.shareaholic-share-buttons-container.shareaholic-floated li.shareaholic-total-count .shareaholic-shares-label{top:-32px}.shareaholic-share-buttons-container.shareaholic-mobile li.shareaholic-total-count{float:left;top:0;left:0;margin:0 !important}.shareaholic-share-buttons-container.shareaholic-mobile.shareaholic-inset-counter li.shareaholic-total-count{height:56px}.shareaholic-share-buttons-container.shareaholic-mobile.shareaholic-inset-counter li.shareaholic-total-count .shareaholic-total-share-count{top:0}.shareaholic-share-buttons-container.shareaholic-mobile.shareaholic-inset-counter li.shareaholic-total-count .shareaholic-shares-label{top:-24px}.shareaholic-share-buttons-container .shareaholic-share-buttons-wrapper{display:inline-block;text-align:left;-webkit-transition:none;-moz-transition:none;transition:none}.shareaholic-share-buttons-container .shareaholic-share-buttons-wrapper-with-headline{margin-left:25px}.shareaholic-share-buttons-container.shareaholic-center-align{text-align:center}.shareaholic-share-buttons-container.shareaholic-center-align:not(.shareaholic-side-counter) .shareaholic-share-buttons-wrapper{text-align:center}.shareaholic-share-buttons-container.shareaholic-center-align:not(.shareaholic-side-counter) .shareaholic-share-buttons-wrapper-with-headline{text-align:left}.shareaholic-share-buttons-container.shareaholic-center-align:not(.shareaholic-side-counter) ul.shareaholic-share-buttons{display:inline-block}.shareaholic-share-buttons-container.shareaholic-left-align{text-align:left}.shareaholic-share-buttons-container.shareaholic-right-align{text-align:right}.shareaholic-share-buttons-container ul.shareaholic-share-buttons{float:none !important;background:none !important;list-style:none;overflow:hidden;margin:0 0 0 10px !important;padding:0 !important;line-height:0;-moz-user-select:-moz-none;-khtml-user-select:none;-webkit-user-select:none;-o-user-select:none;user-select:none;display:block}.shareaholic-share-buttons-container ul.shareaholic-share-buttons.shareaholic-no-margin-padding-important{padding:0 !important;margin:0 !important}.shareaholic-share-buttons-container.shareaholic-vertical ul.shareaholic-share-buttons{overflow:visible;height:auto;display:inline-block}.shareaholic-share-buttons-container.shareaholic-vertical ul.shareaholic-share-buttons li.shareaholic-share-button{float:none !important;width:fit-content}.shareaholic-share-buttons-container.shareaholic-vertical ul.shareaholic-share-buttons li.shareaholic-share-button .shareaholic-share-button-sizing{float:none}.shareaholic-share-buttons-container.shareaholic-vertical.shareaholic-top-counter li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter{display:block}.shareaholic-share-buttons-container.shareaholic-vertical.shareaholic-top-counter li.shareaholic-share-button .shareaholic-share-button-counter{display:none}.shareaholic-share-buttons-container.shareaholic-vertical.shareaholic-side-counter li.shareaholic-share-button.shareaholic-has-shares{width:87px;display:inline-block}.shareaholic-share-buttons-container.shareaholic-vertical.shareaholic-side-counter li.shareaholic-share-button .shareaholic-share-button-counter{float:right !important}.shareaholic-share-buttons-container.shareaholic-horizontal ul.shareaholic-share-buttons li.shareaholic-share-button{float:left !important}.shareaholic-share-buttons-container .shareaholic-custom{margin-left:15px}.shareaholic-share-buttons-container .shareaholic-share-buttons-heading{text-align:left !important;text-indent:0;position:relative}.shareaholic-share-buttons-container .shareaholic-share-buttons-heading img.shareaholic-share-buttons-heading-image{margin:0 0 -30px 0 !important;float:none !important;border:0 !important;outline:none !important;box-shadow:none !important;background:none !important;padding:0 !important}.shareaholic-share-buttons-container.shareaholic-mini:not(.shareaholic-mobile.shareaholic-bottom-align) li.shareaholic-share-button{width:20px !important;height:20px}.shareaholic-share-buttons-container.shareaholic-mini li.shareaholic-share-button .shareaholic-share-button-counter{height:20px;width:35px !important;line-height:20px !important;font-size:11px !important;border-radius:3px}.shareaholic-share-buttons-container.shareaholic-mini.shareaholic-side-counter li.shareaholic-share-button.shareaholic-has-shares{width:62px !important}.shareaholic-share-buttons-container.shareaholic-mini.shareaholic-side-counter li.shareaholic-share-button .shareaholic-share-button-counter{display:none}.shareaholic-share-buttons-container.shareaholic-mini.shareaholic-top-counter li.shareaholic-share-button{height:56px}.shareaholic-share-buttons-container.shareaholic-large li.shareaholic-share-button{width:70px !important;height:70px}.shareaholic-share-buttons-container.shareaholic-large.shareaholic-vertical.shareaholic-inset-counter li.shareaholic-share-button.shareaholic-has-shares{height:100%}.shareaholic-share-buttons-container.shareaholic-large.shareaholic-vertical.shareaholic-inset-counter li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter{line-height:32px !important}.shareaholic-share-buttons-container.shareaholic-rectangle ul.shareaholic-share-buttons li.shareaholic-share-button{width:70px !important;height:20px;box-sizing:content-box !important;-webkit-box-sizing:content-box !important;-moz-box-sizing:content-box !important}.shareaholic-share-buttons-container.shareaholic-rectangle ul.shareaholic-share-buttons li.shareaholic-share-button .shareaholic-share-button-sizing:hover,.shareaholic-share-buttons-container.shareaholic-rectangle ul.shareaholic-share-buttons li.shareaholic-share-button .shareaholic-share-button-sizing:active{filter:brightness(110%)}.shareaholic-share-buttons-container.shareaholic-rectangle ul.shareaholic-share-buttons li.shareaholic-share-button a.shareaholic-service-icon:hover{filter:none}.shareaholic-share-buttons-container.shareaholic-rectangle ul.shareaholic-share-buttons li.shareaholic-share-button .shr-share-button-verb{display:block;position:absolute;top:0;left:28px;line-height:20px !important;font-weight:600}.shareaholic-share-buttons-container.shareaholic-rectangle ul.shareaholic-share-buttons li.shareaholic-share-button .shareaholic-share-button-counter{height:20px;width:35px !important;line-height:20px !important;font-size:11px;border-radius:3px}.shareaholic-share-buttons-container.shareaholic-rectangle.shareaholic-vertical ul.shareaholic-share-buttons li.shareaholic-share-button{float:none !important}.shareaholic-share-buttons-container.shareaholic-rectangle.shareaholic-vertical ul.shareaholic-share-buttons li.shareaholic-share-button .shareaholic-share-button-container{margin:0 !important}.shareaholic-share-buttons-container.shareaholic-rectangle.shareaholic-vertical.shareaholic-top-counter li.shareaholic-share-button{height:auto !important}.shareaholic-share-buttons-container.shareaholic-rectangle.shareaholic-side-counter li.shareaholic-share-button.shareaholic-has-shares{width:112px !important}.shareaholic-share-buttons-container.shareaholic-rectangle.shareaholic-side-counter li.shareaholic-share-button .shareaholic-share-button-counter{line-height:20px !important;font-size:11px !important;display:none}.shareaholic-share-buttons-container.shareaholic-rectangle.shareaholic-top-counter li.shareaholic-share-button{height:auto !important}.shareaholic-share-buttons-container.shareaholic-flat li.shareaholic-share-button .shareaholic-share-button-counter{border-radius:0}.shareaholic-share-buttons-container.shareaholic-flat.shareaholic-badge-counter li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter{background:#d60000 !important;box-shadow:none !important}.shareaholic-share-buttons-container.shareaholic-flat.shareaholic-inset-counter li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-container{padding-bottom:16px !important}.shareaholic-share-buttons-container.shareaholic-flat.shareaholic-inset-counter li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-container .shareaholic-share-button-counter{z-index:1;display:block;color:#fff !important;bottom:0;line-height:23px;position:absolute;margin:0 !important;background:transparent !important;visibility:visible;font-size:12px !important}.shareaholic-share-buttons-container.shareaholic-round{background-color:inherit;background-image:none}.shareaholic-share-buttons-container.shareaholic-round li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter{border-radius:0 40px 40px 0 !important;padding:0 7px 0 20px !important;margin:0 0 0 -20px !important;line-height:40px !important;z-index:0}.shareaholic-share-buttons-container.shareaholic-round li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter:after{display:none}.shareaholic-share-buttons-container.shareaholic-round.shareaholic-mini li.shareaholic-share-button .shareaholic-share-button-counter{padding:0 7px 0 10px !important;line-height:20px !important}.shareaholic-share-buttons-container.shareaholic-round.shareaholic-rectangle li.shareaholic-share-button .shareaholic-share-button-counter{padding:0 7px 0 10px}.shareaholic-share-buttons-container.shareaholic-floated{position:fixed;z-index:10000000;top:15%;left:0;line-height:0 !important}.shareaholic-share-buttons-container.shareaholic-floated ul.shareaholic-share-buttons{margin:0 !important;padding:0 !important}.shareaholic-share-buttons-container.shareaholic-floated ul.shareaholic-share-buttons li.shareaholic-total-count{background-color:#fff}.shareaholic-share-buttons-container.shareaholic-floated ul.shareaholic-share-buttons li.shareaholic-share-button{padding:0 !important;font-size:0;line-height:0}.shareaholic-share-buttons-container.shareaholic-floated ul.shareaholic-share-buttons li.shareaholic-share-button .shareaholic-service-icon{border-radius:0 !important}.shareaholic-share-buttons-container.shareaholic-floated ul.shareaholic-share-buttons li.shareaholic-share-button .shareaholic-share-button-counter{padding:0;width:100%}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-right-align{left:auto;right:0}.shareaholic-share-buttons-container.shareaholic-floated .shareaholic-share-buttons-heading{display:none}.shareaholic-share-buttons-container.shareaholic-floated .shareaholic-share-button-sizing{width:100%}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-force-relative{position:relative}.shareaholic-share-buttons-container.shareaholic-wrapped.shareaholic-left-align{float:left}.shareaholic-share-buttons-container.shareaholic-wrapped.shareaholic-right-align{float:right}.shareaholic-share-buttons-container.shareaholic-wrapped.shareaholic-center-align{float:none}.shareaholic-share-buttons-container.shareaholic-diglett ul.shareaholic-share-buttons{display:flex !important;flex-direction:row}.shareaholic-share-buttons-container.shareaholic-diglett li.shareaholic-share-button{height:40px;width:60px !important;padding:5px 0 !important;text-align:center}.shareaholic-share-buttons-container.shareaholic-diglett li.shareaholic-share-button .shareaholic-share-button-container{position:relative;height:40px;width:60px;margin:0 auto}.shareaholic-share-buttons-container.shareaholic-diglett li.shareaholic-share-button .shareaholic-share-button-container:after{background:url("https://cdn.openshareweb.com/v2/images_72e7af4e/share-buttons/share-button-shadow.png") 0 100% no-repeat;content:" ";height:15px;width:60px;position:absolute;left:0;bottom:0;display:block;z-index:100}@media (min--moz-device-pixel-ratio:1.3),(-o-min-device-pixel-ratio:2.6/2),(-webkit-min-device-pixel-ratio:1.3),(min-device-pixel-ratio:1.3),(min-resolution:1.3dppx){.shareaholic-share-buttons-container.shareaholic-diglett li.shareaholic-share-button .shareaholic-share-button-container:after{background-image:url("https://cdn.openshareweb.com/v2/images_72e7af4e/share-buttons/share-button-shadow@2x.png");background-size:60px 39px}}.shareaholic-share-buttons-container.shareaholic-diglett li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-share-button-sizing{float:none}.shareaholic-share-buttons-container.shareaholic-diglett li.shareaholic-share-button .shareaholic-share-button-container .shareaholic-share-button-sizing a{background-image:none;top:10px;height:30px;border-radius:4px 4px 0 0}.shareaholic-share-buttons-container.shareaholic-diglett li.shareaholic-share-button:hover .shareaholic-share-button-sizing a{top:5px;opacity:1;height:35px}.shareaholic-share-buttons-container.shareaholic-diglett.shareaholic-vertical ul.shareaholic-share-buttons{flex-direction:column}.shareaholic-share-buttons-container.shareaholic-diglett.shareaholic-vertical li.shareaholic-share-button{width:60px !important;margin:0 auto !important}.shareaholic-share-buttons-container.shareaholic-diglett.shareaholic-vertical li.shareaholic-share-button .shareaholic-share-button-container{margin:0 !important}.shareaholic-share-buttons-container.shareaholic-diglett.shareaholic-badge-counter li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter{top:0;right:0}.shareaholic-share-buttons-container.shareaholic-diglett.shareaholic-badge-counter li.shareaholic-share-button.shareaholic-has-shares:hover .shareaholic-share-button-counter{top:-5px}.shareaholic-share-buttons-container.shareaholic-badge-counter li.shareaholic-share-button.shareaholic-has-shares{width:-webkit-min-content}.shareaholic-share-buttons-container.shareaholic-badge-counter li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter{display:block;position:absolute;visibility:visible;width:auto;height:auto;top:-2px;right:-3px;text-align:center;min-width:8px !important;padding:1px 4px !important;color:#fff !important;font-family:'Open Sans',arial,sans-serif !important;font-size:10px !important;font-weight:600 !important;background:#d60000 !important;background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0%,#fd7c7e),color-stop(100%,#d60000)) !important;background-image:-webkit-linear-gradient(top,#fd7c7e,#d60000) !important;background-image:-moz-linear-gradient(top,#fd7c7e,#d60000);background-image:-ms-linear-gradient(top,#fd7c7e,#d60000);background-image:-o-linear-gradient(top,#fd7c7e,#d60000);background-image:linear-gradient(top,#fd7c7e,#d60000);-webkit-border-radius:15px !important;border-radius:15px !important;box-shadow:0 2px 2px rgba(0,0,0,0.4);text-shadow:0 -1px 0 rgba(0,0,0,0.2);line-height:14px !important;border:2px solid #fff !important;z-index:1}.shareaholic-share-buttons-container.shareaholic-badge-counter.shareaholic-vertical li.shareaholic-share-button{width:40px;margin:0 auto !important;padding:5px 0 !important}.shareaholic-share-buttons-container.shareaholic-side-counter li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter{float:right !important;visibility:visible;display:inline-block;box-sizing:content-box;-webkit-box-sizing:content-box;-moz-box-sizing:content-box}.shareaholic-share-buttons-container.shareaholic-side-counter li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter:after{top:50%;right:100%;margin:-5px 0 0 0;border-right-color:#e6e7e9}.shareaholic-share-buttons-container.shareaholic-top-counter li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter{visibility:visible}.shareaholic-share-buttons-container.shareaholic-top-counter li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter:after{left:50%;top:100%;margin:0 0 0 -5px;border-top-color:#e6e7e9}.shareaholic-share-buttons-container.shareaholic-top-counter li.shareaholic-share-button .shareaholic-share-button-counter{margin:0 0 7px 0 !important;display:block;visibility:hidden;width:40px !important;height:28px !important;font-size:14px !important;line-height:28px !important}.shareaholic-share-buttons-container.shareaholic-top-counter.shareaholic-vertical li.shareaholic-share-button{margin:0 auto !important;width:40px}.shareaholic-share-buttons-container.shareaholic-top-counter.shareaholic-rectangle li.shareaholic-share-button.shareaholic-has-shares .shareaholic-share-button-counter{line-height:28px !important;width:auto !important}.shareaholic-share-buttons-container li.shareaholic-share-button{list-style:none !important;background:none !important;float:left !important;width:auto;margin:0 !important;padding:4px !important;position:relative;cursor:pointer;box-sizing:content-box;overflow:visible;text-indent:0;clear:none;border:none}.shareaholic-share-buttons-container li.shareaholic-share-button .shareaholic-share-button-sizing{float:left;position:relative;display:inline-block;overflow:visible !important;border:none}.shareaholic-share-buttons-container li.shareaholic-share-button:before{content:"" !important;padding:0 !important;margin:0 !important;display:inline !important;border:none;position:static}.shareaholic-share-buttons-container li.shareaholic-share-button .shareaholic-share-button-counter{margin:0 0 0 7px !important;font-family:'Open Sans',arial,sans-serif !important;width:40px;vertical-align:top;position:relative;text-align:center;background:#e6e7e9 !important;color:#666 !important;font-size:14px !important;font-weight:400;border-radius:4px;line-height:40px;border:0 solid #ccc;display:none;visibility:hidden;cursor:pointer}.shareaholic-share-buttons-container li.shareaholic-share-button .shareaholic-share-button-counter:after{content:"";display:inline-block;position:absolute;border:5px solid transparent}.shareaholic-share-buttons-container li.shareaholic-share-button .shareaholic-share-button-counter:before{content:"";display:inline-block;position:absolute;border:6px solid transparent}.shareaholic-share-buttons-container li.shareaholic-share-button .shr-share-button-verb{display:none;color:#fff !important;font-size:11px !important}.shareaholic-follow-buttons-container ul.shareaholic-share-buttons{margin:0 !important}.shareaholic-follow-buttons-container.shareaholic-center-align .shareaholic-share-buttons-heading{text-align:center !important}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile{display:block}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-top-align{top:0 !important}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-bottom-align{top:inherit;margin-bottom:0;bottom:0;left:0}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-slide-down{bottom:-40px}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-slide-up{margin-top:-40px}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-inset-counter li.shareaholic-has-shares .shareaholic-share-button-container{padding-bottom:0 !important}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-inset-counter li.shareaholic-has-shares .shareaholic-service-icon{top:-16px}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-inset-counter.shareaholic-slide-down{bottom:-56px}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-inset-counter.shareaholic-slide-up{margin-top:-56px}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-inset-counter.shareaholic-top-align{top:16px !important}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-inset-counter.shareaholic-top-align li{top:-16px}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-inset-counter .shareaholic-share-button-sizing{padding-top:16px}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile.shareaholic-inset-counter .shareaholic-service-icon{top:-8px}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile ul.shareaholic-share-buttons{overflow:visible;height:auto;display:inline-block}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile ul.shareaholic-share-buttons li.shareaholic-share-button{float:left !important;width:auto !important}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile ul.shareaholic-share-buttons li.shareaholic-share-button:hover,.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile ul.shareaholic-share-buttons li.shareaholic-share-button:active{opacity:1;filter:brightness(110%);z-index:1000000}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-mobile ul.shareaholic-share-buttons li.shareaholic-share-button .shareaholic-share-button-sizing a{margin:0 auto !important}.shareaholic-share-buttons-animation-wrapper{position:relative}.shareaholic-share-arrow{position:absolute;bottom:15px;left:0}.shareaholic-share-buttons-container.shareaholic-floated:not(.shareaholic-mini) .shareaholic-share-button-sizing a.shareaholic-service-icon{margin:0 auto !important}.shareaholic-share-buttons-container.shareaholic-floated.shareaholic-right-align.shareaholic-mini .shareaholic-service-icon{float:right}.shareaholic-share-buttons-container.shareaholic-floated .shareaholic-share-button:hover,.shareaholic-share-buttons-container.shareaholic-floated .shareaholic-share-button:active{filter:brightness(110%)}.shareaholic-share-buttons-container.shareaholic-floated a.shareaholic-service-icon:hover{filter:none}.shr-share-button-invisible{visibility:hidden}.shareaholic-media-overlay{animation:shr-media-overlay-fadein ease .8s}.shareaholic-media-overlay .shareaholic-share-buttons-container{height:0}.shareaholic-media-overlay li.shareaholic-share-button{padding:0 !important}.shareaholic-media-overlay li.shareaholic-share-button:hover{opacity:1 !important}.shareaholic-media-overlay li.shareaholic-share-button .shareaholic-share-button-container{opacity:1 !important}.shareaholic-media-overlay li.shareaholic-share-button .shareaholic-service-icon:hover{opacity:1 !important}.shareaholic-media-overlay ul.shareaholic-share-buttons{padding:0 !important;margin:0 !important}.shareaholic-media-overlay .shareaholic-shareable_images{width:100%;height:100%}.shareaholic-media-overlay .shareaholic-shareable_images .shareaholic-share-buttons-container{width:100%;height:100%}@keyframes shr-media-overlay-fadein{0%{opacity:0}99%{opacity:1}}@-webkit-keyframes shr-media-overlay-fadein{0%{opacity:0}99%{opacity:1}}</style><style class="darkreader darkreader--sync" media="screen"></style></div>
</div>
</div>
</div> 
</li>
</ul>
</div>
</nav> 
<input type="hidden" name="svgImageBackgroundColor" value="rgba(255,255,255,0.7)">
<section id="tabbedpages" class="tabs icon-image-content">
<div id="abstract_content" class=" tab-pane abstract tabbedsection">
<div class="articleabstract">
<h3 dir="auto">Abstract</h3>
<div dir="auto">
<div class="description"><p>Small single-stranded nucleic acid phages effect lysis by expressing a single protein, the amurin, lacking muralytic enzymatic activity. Three amurins have been shown to act like ‘protein antibiotics’ by inhibiting cell-wall biosynthesis. However, the L lysis protein of the canonical ssRNA phage MS2, a 75 aa polypeptide, causes lysis by an unknown mechanism without affecting net peptidoglycan synthesis. To identify residues important for lytic function, randomly mutagenized alleles of <span class="jp-italic">L</span> were generated, cloned into an inducible plasmid and the transformants were selected on agar containing the inducer. From a total of 396 clones, 67 were unique single base-pair changes that rendered L non-functional, of which 44 were missense mutants and 23 were nonsense mutants. Most of the non-functional missense alleles that accumulated in levels comparable to the wild-type allele are localized in the C-terminal half of L, clustered in and around an LS dipeptide sequence. The LS motif was used to align <span class="jp-italic">L</span> genes from ssRNA phages lacking any sequence similarity to MS2 or to each other. This alignment revealed a conserved domain structure, in terms of charge, hydrophobic character and predicted helical content. None of the missense mutants affected membrane-association of L. Several of the <span class="jp-italic">L</span> mutations in the central domains were highly conservative and recessive, suggesting a defect in a heterotypic protein–protein interaction, rather than in direct disruption of the bilayer structure, as had been previously proposed for L.</p></div>
</div>
</div>
<ul class="list-inline border-right-list article-dates">
<li>
Received: 
<time datetime="2017-03-13">13/03/2017</time>
</li>
<li>
Accepted:
<time datetime="2017-05-15">15/05/2017</time>
</li>
<li>
Published Online:
<time datetime="2017-07-01">01/07/2017</time>
</li>
</ul> 
<div class="bottom-side-nav">
<span>
<strong dir="auto">
Keyword(s):
</strong>
</span>
<a href="https://www.microbiologyresearch.org/search?value1=%22bacteriophage%22&amp;option1=pub_keyword" title="bacteriophage">bacteriophage</a>
, 
<a href="https://www.microbiologyresearch.org/search?value1=%22Escherichia+coli%22&amp;option1=pub_keyword" title="Escherichia coli">Escherichia coli</a>
and 
<a href="https://www.microbiologyresearch.org/search?value1=%22lysis%22&amp;option1=pub_keyword" title="lysis">lysis</a>
</div>
<div class="copyright-info">
<span dir="auto">
© 2017 The Authors
</span>
</div>
</div>
<div id="metrics_content" class="tab-pane metrics hidden-js-div tabbedsection">
<div class="itemMetricsLoading">
<img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/spinner.gif" alt="Loading">
<p>Article metrics loading...</p>
</div>
<div class="metricsContainer retrieveMetrics">
<div class="hiddenjsdiv metricsItemId">/content/journal/micro/10.1099/mic.0.000485</div>
<div class="hiddenjsdiv metricsPubDate">2017-07-01</div>
<div class="hiddenjsdiv metricsEndDate">2024-03-15</div>
<div class="hiddenjsdiv" id="graph1Settings" data-nodata="No metrics data to plot.">
</div>
<div class="hiddenjsdiv" id="graph2Settings" data-nodata="No metrics data to plot.">
</div>
<div class="hiddenjsdiv " id="graph3Settings" data-nodata="No metrics data to plot.">
</div>
<div class="metricsDetails">
</div>
</div>
</div>
<div id="html_fulltext" class="htmlfulltext hidden-js-div tabbedsection tab-pane active">
<form name="pptForm" class="hidden pptForm" id="pptDwnld" action="https://www.microbiologyresearch.org/contentexport/powerpoint/powerpoint.action" method="POST">
<input type="hidden" name="itemId" value="">
<input type="hidden" name="title" value="">
<input type="hidden" name="caption" value="">
<input type="hidden" name="copyright" value="">
<input type="hidden" name="citation" value="">
<input type="hidden" name="imagePath" value="">
<input type="hidden" name="logoPath" value="jp/pub2web.gif">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="pptDownload">Download as PowerPoint</a>
</form>

<span class="showPPT" data-showppt="true"></span>
<div id="itemFullTextId" class="itemFullTextHtml" data-fulltexturl="/deliver/fulltext/micro/163/7/961.html?itemId=/content/journal/micro/10.1099/mic.0.000485&amp;mimeType=html&amp;fmt=ahah" data-fulltextunavailablemsg="&lt;span dir=&quot;auto&quot;&gt;The full text of this item is not currently available.&lt;/span&gt;" data-fulltextexpandlinktext="Toggle display:" data-hideinlinetablesbydefault="false" data-showreferenceoverlaypopup="true"><!--?xml version="1.0" encoding="UTF-8"?--><div xmlns:mml="http://www.w3.org/1998/Math/MathML" id="html-body">

<div id="article-level-0-front-and-body">
<ul xmlns:xs="http://www.w3.org/2001/XMLSchema" class="flat fm article-header-metadata"><li></li><li>ISSN: <span class="issn">1350-0872</span>, Online ISSN: <span class="oissn">1465-2080</span></li><li>DOI: <span class="doi">http://dx.doi.org/10.1099/mic.0.000485</span></li><li>Volume <span class="volume">163</span>, Issue <span class="issue">7</span>, pages <span class="first-page">961</span>-<span class="last-page">969</span></li><li><span class="copyright">© 2017 The Authors</span>&nbsp;</li></ul><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection article-title-and-authors"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="top-1" id="top-1">
                                &nbsp;
                            </a></div><div class="menuButton">Go to section...</div><div class="clearer">&nbsp;</div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul></div></div><h1><span class="article-title">Mutational analysis of the MS2 lysis protein L</span></h1><ul class="flat authors">Karthik R. Chamakura<span class="au-label">1</span><span class="au-label">2</span>; Garrett B. Edwards<span class="au-label">1</span><span class="au-label">2</span><span class="au-label">†</span>; Ry Young<span class="au-label">1</span><span class="au-label">2</span></ul><ul class="flat affiliations"><strong>1: </strong><span class="au-label">1​</span>Center for Phage Technology, Texas A&amp;M AgriLife, Texas A&amp;M University, College Station, TX, USA; <strong>2: </strong><span class="au-label">2​</span>Department of Biochemistry and Biophysics, Texas A&amp;M University, College Station, TX, USA; <strong>3: </strong><span class="au-label">†​</span>Present address: Department of Chemistry and Biochemistry, University of Colorado Boulder, Boulder, CO, USA.</ul><div class="correspondence">*Correspondence: Ry Young, <a href="mailto:ryland@tamu.edu">ryland@tamu.edu</a></div><span class="author-footnote" id="FN1"><p xmlns:f="http://www.example.org/functions"><span class="fn-label">FN1</span>A supplementary figure and two supplementary tables are available with the online Supplementary Material.</p></span><div class="clearer">&nbsp;</div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection article-abstract"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="abstract-1" id="abstract-1">ABSTRACT</a></div><div class="menuButton">Go to section...</div><div class="clearer">&nbsp;</div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul></div></div><p>Small single-stranded nucleic acid phages effect lysis by expressing a single protein, the amurin, lacking muralytic enzymatic activity. Three amurins have been shown to act like ‘protein antibiotics’ by inhibiting cell-wall biosynthesis. However, the L lysis protein of the canonical ssRNA phage MS2, a 75 aa polypeptide, causes lysis by an unknown mechanism without affecting net peptidoglycan synthesis. To identify residues important for lytic function, randomly mutagenized alleles of <span class="jp-italic">L</span> were generated, cloned into an inducible plasmid and the transformants were selected on agar containing the inducer. From a total of 396 clones, 67 were unique single base-pair changes that rendered L non-functional, of which 44 were missense mutants and 23 were nonsense mutants. Most of the non-functional missense alleles that accumulated in levels comparable to the wild-type allele are localized in the C-terminal half of L, clustered in and around an LS dipeptide sequence. The LS motif was used to align <span class="jp-italic">L</span> genes from ssRNA phages lacking any sequence similarity to MS2 or to each other. This alignment revealed a conserved domain structure, in terms of charge, hydrophobic character and predicted helical content. None of the missense mutants affected membrane-association of L. Several of the <span class="jp-italic">L</span> mutations in the central domains were highly conservative and recessive, suggesting a defect in a heterotypic protein–protein interaction, rather than in direct disruption of the bilayer structure, as had been previously proposed for L.</p>
<span class="tl-lowest-section">Keywords</span><ul class="flat keywords"><li>bacteriophage, </li><li>lysis, </li><li><span class="jp-italic">Escherichia coli</span></li></ul><div class="clearer">&nbsp;</div></div><span xmlns:xs="http://www.w3.org/2001/XMLSchema" class="tl-lowest-section">Author Notes</span><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="author-footnote-below-abstract" id="FN1">A supplementary figure and two supplementary tables are available with the online Supplementary Material.</div>



<div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="s1-1" id="s1-1">Abbreviations</a></div><div class="menuButton">Go to section...</div><div class="clearer">&nbsp;</div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul></div></div><p>PG, peptidoglycan; pmf, proton motive force; TMD, transmembrane domain.</p>
<div class="clearer">&nbsp;</div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="s1-2" id="s1-2">Introduction</a></div><div class="menuButton">Go to section...</div><div class="clearer">&nbsp;</div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul></div></div><p>Small lytic phages with single-stranded nucleic acid genomes achieve lysis and release of the progeny virions by the expression of a single gene [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R1" data-original-title="" title="">1</a></span>]. These phages can be broadly classified into ssDNA and ssRNA phages. The ssDNA phage φX174 is the prototype of the ubiquitous family <span class="jp-italic">Microviridae</span>, with a 5.4 kb genome and 10 genes. In the φX174 genome, the lysis gene <span class="jp-italic">E</span> is entirely embedded within the +1 reading frame of the essential assembly gene <span class="jp-italic">D</span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R2" data-original-title="" title="">2</a></span>]. The ssRNA phages defined the family <span class="jp-italic">Leviviridae</span> and were traditionally further subdivided into two genera, <span class="jp-italic">Allolevivirus</span> and <span class="jp-italic">Levivirus</span>, represented by the prototype male-specific coliphages Qβ and MS2 (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F1">Fig. 1a</a></span>). These phages are the simplest viruses, with 3.5–4.3 kb genomes and three core genes encoding an RNA-dependent RNA polymerase or replicase (Rep), a major capsid protein (Coat) and Mat, the maturation or attachment protein (one molecule per virion, named as A in MS2, A<span class="jp-sub">2</span> in Qβ). The two genera differ in the structure and role of a fourth essential gene in each case, with the Alloleviviruses having the <span class="jp-italic">A1</span> gene, a translational read-through extension of <span class="jp-italic">coat</span>, and the Leviviruses having the <span class="jp-italic">L</span> lysis gene, with a reading frame overlapping and out-of-frame with the end of <span class="jp-italic">coat</span> and the beginning of <span class="jp-italic">rep</span>. Moreover, the Mat protein of Qβ (A<span class="jp-sub">2</span>) was found to be necessary and sufficient for lysis [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R3" data-original-title="" title="">3</a></span>].</p>
<!--splitRids is F1--><!--currentRid is F1--><!--current position is 11--><!--check position is 11--><p xmlns:f="http://www.example.org/functions"></p><div class="figure html-fulltext-responsive-figure" id="F1"><div class="caption"><span class="captionLabel"><span class="label">Fig. 1.</span></span><span class="captionText"><span>Cysteine scanning and alignment of L homologues identifies the importance of the conserved LS motif. (a) Genome maps of four ssRNA phages representing the four known genetic architectures of lysis genes. The lysis genes are highlighted in grey. (b) Lysis profiles of various cysteine scanning mutants. ▲, <span class="jp-italic">L</span>; ⊞, <span class="jp-italic">L<sup>C29S</sup></span>; □, <span class="jp-italic">L<sup>S9C</sup></span>; ◊, <span class="jp-italic">L<sup>S15C</sup></span>; ♦, <span class="jp-italic">L<sup>S35C</sup></span>; Δ, <span class="jp-italic">L<sup>S49C</sup></span>; X, <span class="jp-italic">L<sup>S58C</sup></span>; ○, <span class="jp-italic">L<sup>L73C</sup></span>. (c) L-like lysis proteins from different leviviruses are aligned with respect to the conserved LS motif (yellow), preceded by a stretch of hydrophobic residues (underlined) and highly basic N-termini. Basic and acidic residues are highlighted in red and blue, respectively. GenBank accession numbers for the lysis proteins shown in (c) are as follows: MS2 (CAA23990.1), M12 (AAF19634.1), fr (CAA33137.1), GA (CAA27498.1), JP34 (AAA72211.1), KU1 (AAF67675.1), Hgal1 (YP_007237174.1), C1 (YP_007237128.1), AP205 (NP_085469.1), PP7 (NP_042306.1), PRR1 (YP_717670.1).</span>
</span></div><div class="image"><a class="media-link" href="https://www.microbiologyresearch.org/docserver/fulltext/micro/163/7/micro000485-f1.gif" id="/content/micro/10.1099/mic.0.000485.F1" data-fancybox="gallery1"><img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/micro000485-f1_thmb.gif" border="0" alt="Fig. 1."><p><span class="figure-duplicate-label">Fig. 1.<br><br></span>Click to view</p></a></div><div class="clearer">&nbsp;</div><div class="downloadAsPptContainer hidden">
<form name="pptFigForm" class="pptForm" action="https://www.microbiologyresearch.org/contentexport/powerpoint/powerpoint.action" method="POST">
<input type="hidden" name="itemId" value="http://instance.metastore.ingenta.com/content/micro/10.1099/mic.0.000485.F1">
<input type="hidden" name="title" value="Fig. 1.">
<input type="hidden" name="caption" value="Cysteine scanning and alignment of L homologues identifies the importance of the conserved LS motif. (a) Genome maps of four ssRNA phages representing the four known genetic architectures of lysis genes. The lysis genes are highlighted in grey. (b) Lysis profiles of various cysteine scanning mutants. ▲, L; ⊞, LC29S
; □, LS9C
; ◊, LS15C
; ♦, LS35C
; Δ, LS49C
; X, LS58C
; ○, LL73C
. (c) L-like lysis proteins from different leviviruses are aligned with respect to the conserved LS motif (yellow), preceded by a stretch of hydrophobic residues (underlined) and highly basic N-termini. Basic and acidic residues are highlighted in red and blue, respectively. GenBank accession numbers for the lysis proteins shown in (c) are as follows: MS2 (CAA23990.1), M12 (AAF19634.1), fr (CAA33137.1), GA (CAA27498.1), JP34 (AAA72211.1), KU1 (AAF67675.1), Hgal1 (YP_007237174.1), C1 (YP_007237128.1), AP205 (NP_085469.1), PP7 (NP_042306.1), PRR1 (YP_717670.1).">
<input type="hidden" name="copyright" value="© 2017 The Authors">
<input type="hidden" name="citation" value="">
<input type="hidden" name="imagePath" value="/fulltext/micro/163/7/micro000485-f1.gif">
<input type="hidden" name="logoPath" value="jp/pub2web.gif">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="downloadPpt js-downloadppt">Download as PowerPoint</a>
</form>
</div></div><p></p><p>The genetic simplicity of the ssDNA and ssRNA phages made it straightforward to define the cistron required for lysis. The term amurin has been proposed for these lysis proteins, which lack muralytic activity but somehow subvert peptidoglycan (PG) integrity [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R1" data-original-title="" title="">1</a></span>]. The mechanism by which amurins effect lysis remained controversial for decades. One model supported by multiple reports was that φX174 E formed a ‘membrane tunnel’ that traversed the entire envelope and allowed release of the cytosolic contents, including the assembled virions [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R4" data-original-title="" title="">4, 5</a></span>]. Ultimately a genetic approach based on selection of host mutants resistant to the induction of the amurin genes cloned under a plasmid-borne inducible promoter was successful [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R6" data-original-title="" title="">6</a></span>]. Remarkably, both the φX174 E and Qβ A<span class="jp-sub">2</span> amurins were found to be specific inhibitors of enzymes of the conserved pathway for murein precursor biosynthesis: A<span class="jp-sub">2</span> inhibits MurA, the first committed step of the pathway, whereas E inhibits MraY, which catalyses the formation of the first lipid-linked precursor [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R7" data-original-title="" title="">7–9</a></span>]. More recently, a Levivirus specific for the conjugational pilus of an IncM R-factor was shown to have a lysis cistron, <span class="jp-italic">lys<sup xmlns:f="http://www.example.org/functions">M</sup></span>, that evolved in a genomic location different from the position of <span class="jp-italic">L</span> astride the <span class="jp-italic">coat–rep</span> interface in MS2 [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R10" data-original-title="" title="">10</a></span>]. By use of a similar approach, it was shown that Lys<sup xmlns:f="http://www.example.org/functions">M</sup> is a specific inhibitor of MurJ, the Lipid II flippase of <span class="jp-italic">Escherichia coli</span> (K. R. Chamakura, L. T. Sham, R. M. Davis, L. Min, H. Cho, N. Ruiz, T. G. Bernhardt, R. F. Young, unpublished results). Thus, for all three of these small phages, the expression of these proteins, either from plasmid-cloned cistrons or in the context of phage infection, caused blockage of the flow of murein precursors, generally leading to lysis as a result of failed septation events. The obvious similarity to murein-specific chemotherapeutic agents has led to these lysis proteins being designated as ‘protein antibiotics’ [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R1" data-original-title="" title="">1</a></span>].</p>
<p>Despite this success with three single gene lysis systems, the function of the original amurin, MS2 L, remains unclear. L is a 75-amino acid protein that has been reported to be present in the membrane fractions [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R11" data-original-title="" title="">11</a></span>]. Work from the van Duin group using plasmid-borne L genes showed that the highly basic N-terminal half of the protein is dispensable for lytic activity, whereas C-terminal truncations were found to be non-functional [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R12" data-original-title="" title="">12</a></span>]. An experiment with a synthetic peptide comprising the C-terminal 25 amino acids of L was reported to dissipate proton motive force (pmf) of <span class="jp-italic">E. coli</span> inverted membrane vesicles and cause fluorescent dye leakage in reconstituted liposomes [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R13" data-original-title="" title="">13</a></span>]. Surprisingly, immuno-electron microscopy studies using an antibody raised against a hybrid L protein, suggested that L was enriched in zones of adhesion between inner and outer membranes, also known as Bayer’s bridges [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R14" data-original-title="" title="">14</a></span>]. Moreover, biochemical analysis of the murein in cells lysed by L was reported to have a decreased average chain length of glycan strands and altered cross-linking [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R15" data-original-title="" title="">15</a></span>]. These disparate observations were not easily reconciled into a clear conceptual framework. However, it was proposed that L causes pmf-depleting lesions in the inner membrane, thereby somehow activating host autolytic enzymes such as lytic transglycoslyases and D-D endopeptidases [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R15" data-original-title="" title="">15</a></span>]. Although this ‘autolysis’ remains as an attractive general model for L function, neither an operational schema nor molecular details for such a pathway have been forthcoming.</p>
<p>Recently a genetics-based approach, analogous to the approach used for E and A<span class="jp-sub">2</span>, was employed to address the host target of the L amurin. Although a cellular target was not identified, it was demonstrated that the lytic function of L requires an interaction with the host chaperone DnaJ, and that this requirement could be by passed by deletion of the basic N-terminal domain [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R16" data-original-title="" title="">16</a></span>]. Here we report a comprehensive genetic analysis and identify residues and regions of L that are critical for lysis. Results are discussed in terms of a model for L function.</p>
<div class="clearer">&nbsp;</div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="s1-3" id="s1-3">Methods</a></div><div class="menuButton">Go to section...</div><div class="clearer">&nbsp;</div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul></div></div><div id="s2-3-1">
<span class="tl-lowest-section">Bacterial strains, culture growth, plasmids and reagents</span>
<p>The bacterial strains used in this study are described in <span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#T1">Table 1</a></span> and the primers are listed in Table S1 (available in the online Supplementary Material). LB broth and LB agar were used as growth medium and were supplemented with appropriate antibiotics and inducers. When indicated, ampicillin (Amp), chloramphenicol (Cam) and arabinose were added to the growth media at concentrations of 100 µg ml<sup xmlns:f="http://www.example.org/functions">−1</sup>, 10 µg ml<sup xmlns:f="http://www.example.org/functions">−1</sup> and 0.2–0.4 % (w/v), respectively. Bacterial growth and lysis were monitored as previously described [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R17" data-original-title="" title="">17</a></span>]. The <span class="jp-italic">his6-L<sup xmlns:f="http://www.example.org/functions">syn</sup></span> gene from pKC12 (<span class="jp-italic">bla araC P<span class="jp-sub">ara </span></span>::<span class="jp-italic"> his6-L<sup xmlns:f="http://www.example.org/functions">syn</sup></span>) was amplified with primers KC19 and KC31 using Phusion DNA polymerase (New England Biolabs). The PCR product was gel-purified and digested with enzymes KpnI and HindIII and ligated into pBAD33 digested with the same enzymes to generate pKC17 (<span class="jp-italic">cat araC P<span class="jp-sub">ara </span></span>::<span class="jp-italic"> his6-L<sup xmlns:f="http://www.example.org/functions">syn</sup></span>). Plasmids pKC18, pKC19, pKC20 and pKC21 were constructed by site-directed mutagenesis of the plasmid pBAD24 L with primers KC407, KC408, KC409 and KC410, respectively. Rabbit polyclonal serum raised against MS2 L peptide ‘TPASTNRRRPFKHEDC’ and goat anti-rabbit-HRP (ThermoFisher Scientific) were used as primary and secondary antibodies, respectively. Unless otherwise indicated, all chemicals were purchased from Sigma-Aldrich.</p>
<!--splitRids is T1--><!--currentRid is T1--><!--current position is 26--><!--check position is 26--><div class="table-caption-container js-table-caption-container" data-webid="/content/micro/10.1099/mic.0.000485.T1" id="T1"><h5 class="table-label">Table 1.</h5><p><span>Strains and plasmids used in this study</span>
</p><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="toggle-table js-toggle-table is-ajax-triggered">Toggle display:Table 1.&nbsp;<i class="fa fa-table" aria-hidden="true">         </i>&nbsp;<i class="fa fa-caret-down" aria-hidden="true"></i></a>         <a href="https://www.microbiologyresearch.org/content/micro/10.1099/mic.0.000485.T1?fmt=ahah&amp;fullscreen=true" target="_blank" class="open-table-fullscreen js-open-table pull-right">Open Table 1. fullscreen <i class="fa fa-expand" aria-hidden="true"></i></a></div><div class="table-container" data-id="T1" style="">
<table class="html-fulltext-inline-table js-html-fulltext-inline-table">
<caption>
<h5 class="table-label">Table 1.</h5>
<p>
<span>Strains and plasmids used in this study</span>
</p>
</caption>
<thead>
<tr>
<th align="left" colspan="1" rowspan="1">Strain/plasmid</th>
<th align="center" colspan="1" rowspan="1">Relevant genotype or description</th>
<th align="center" colspan="1" rowspan="1">Reference</th>
</tr>
</thead>
<tbody><tr>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-bold">Strain</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> XL1-Blue</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">recA endA1 gyrA96 thi hsdR17 supE44 relA1 lac</span> [F′ :: Tn<span class="jp-italic">10 proA+B+lacIq</span>
<br><br>
<span class="jp-italic">Δ(lacZ)M15</span>]</td>
<td align="left" colspan="1" rowspan="1" valign="top">Stratagene</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> MG1655</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">ilvG- rfb</span>-50 <span class="jp-italic">rph</span>-1</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R34">34</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> TB28</td>
<td align="left" colspan="1" rowspan="1" valign="top">MG1655 <span class="jp-italic">lacIZYA &lt;&gt;frt</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R35">35</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-bold">Plasmid</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pBAD24</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">bla araC P<span class="jp-sub">ara</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R22">22</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pBAD33</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">cat araC P<span class="jp-sub">ara</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R22">22</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pQ</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R36">36</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pRE-<span class="jp-italic">L</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">L</span> gene from MS2 clone under the lambda late promoter pR'</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R36">36</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pRE-<span class="jp-italic">L<span class="jp-sub">C29S</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">pRE-<span class="jp-italic">L</span> with C29S mutation</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R36">36</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pRE-<span class="jp-italic">L<span class="jp-sub">C29S, S9C</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">pRE-<span class="jp-italic">L<span class="jp-sub">C29S</span>
</span> with S9C mutation</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R36">36</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pRE-<span class="jp-italic">L<span class="jp-sub">C29S, S15C</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">pRE-<span class="jp-italic">L<span class="jp-sub">C29S</span>
</span> with S15C mutation</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R36">36</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pRE-<span class="jp-italic">L<span class="jp-sub">C29S, S35C</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">pRE-<span class="jp-italic">L<span class="jp-sub">C29S</span>
</span> with S35C mutation</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R36">36</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pRE-<span class="jp-italic">L<span class="jp-sub">C29S, S49C</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">pRE-<span class="jp-italic">L<span class="jp-sub">C29S</span>
</span> with S49C mutation</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R36">36</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pRE-<span class="jp-italic">L<span class="jp-sub">C29S, S58C</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">pRE-<span class="jp-italic">L<span class="jp-sub">C29S</span>
</span> with S58C mutation</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R36">36</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pRE-<span class="jp-italic">L<span class="jp-sub">C29S, L73C</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">pRE-<span class="jp-italic">L<span class="jp-sub">C29S</span>
</span> with L73C mutation</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R36">36</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pRE-<span class="jp-italic">L<span class="jp-sub">A45E</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">pRE-<span class="jp-italic">L</span> with A45E mutation</td>
<td align="left" colspan="1" rowspan="1" valign="top">[<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R36">36</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pKC12</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">cat araC P<span class="jp-sub">ara </span>
</span>:: <span class="jp-italic">his6-L<span class="jp-sup">syn</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="xref">
<span class="jp-sup">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R16">[16</a>
</span>
</span>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pKC17</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">cat araC P<span class="jp-sub">ara </span>
</span>:: <span class="jp-italic">his6-L<span class="jp-sup">syn</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">This study</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pBAD24 <span class="jp-italic">L</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">bla araC P<span class="jp-sub">ara </span>
</span>:: <span class="jp-italic">L</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">This study</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pKC18</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">bla araC P<span class="jp-sub">ara </span>
</span>:: <span class="jp-italic">L<span class="jp-sup">L44I</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">This study</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pKC19</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">bla araC P<span class="jp-sub">ara </span>
</span>:: <span class="jp-italic">L<span class="jp-sup">A45V</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">This study</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pKC20</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">bla araC P<span class="jp-sub">ara </span>
</span>:: <span class="jp-italic">L<span class="jp-sup">L48I</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">This study</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"> pKC21</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<span class="jp-italic">bla araC P<span class="jp-sub">ara </span>
</span>:: <span class="jp-italic">L<span class="jp-sup">L48V</span>
</span>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">This study</td>
</tr>
</tbody></table>
</div></div>
<div id="s2-3-2">
<span class="tl-lowest-section">Selection of non-functional mutants</span>
<p>The <span class="jp-italic">L</span> gene was randomly mutagenized using GeneMorph II Random Mutagenesis kit (Agilent Technologies) as per the instructions provided with the kit and cloned into pBAD24 [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R16" data-original-title="" title="">16</a></span>]. The ligated plasmids were transformed into XL1-Blue and plated on LB agar plates supplemented with Amp and arabinose (0.2 %, w/v). Colonies that survived on the inducer plates were isolated, grown in liquid culture, and the plasmid DNA was extracted using a Qiagen mini-prep kit, and sequenced at Eton Biosciences (San Diego) with primers KC30 and KC31. The plasmids with a single-missense change in the <span class="jp-italic">L</span> gene were transformed into TB28 and the lysis profiles of the mutants were compared with that of the wild-type allele. Mutants that did not cause lysis in liquid culture were scored as non-functional alleles of <span class="jp-italic">L</span>.</p>
</div>
<div id="s2-3-3">
<span class="tl-lowest-section">Monitoring accumulation of <span class="jp-italic">L</span> allele products</span>
<p>To quantify the expression of mutant alleles of <span class="jp-italic">L</span>, a 1 ml sample was collected at 40 min post-induction and mixed with 111 µl cold 100 % TCA as previously described [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R18" data-original-title="" title="">18</a></span>]. The TCA precipitates were collected by centrifugation at 13 000 r.p.m. for 10 min in a micro centrifuge. The pellets were washed three times with 1 ml cold acetone and air-dried. The dried pellets were resuspended in 2× sample loading buffer with β-mercaptoethanol and boiled for 10 min. Normalized amounts of protein (~0.4 OD<span class="jp-sub">550</span> units) were analysed by SDS-PAGE and Western blotting as previously described [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R18" data-original-title="" title="">18</a></span>]. The antibodies against MS2 L and goat anti-rabbit-HRP were used at a 1 : 3000 dilution.</p>
</div>
<div id="s2-3-4">
<span class="tl-lowest-section">Membrane fractionation</span>
<p>Cultures of TB28 (500 ml) harbouring plasmids with <span class="jp-italic">L</span> alleles were grown to an OD<span class="jp-sub">550</span> ~0.2, induced with arabinose, and the cultures were collected at 40 min post-induction by centrifugation at 10 000 <span class="jp-bold"><span class="jp-italic">g</span></span> (Sorvall LYNX 6000 Superspeed Centrifuge) for 10 min. Pelleted cells were resuspended in ~3 ml PBS (pH 7.2) supplemented with Protease Inhibitor Cocktail (Sigma; 1 µl/35 OD<span class="jp-sub">550</span> units of original culture) and lysed by passing three times through an Aminco French Pressure cell at 16 000 psi. After intact cells were cleared from the lysate by centrifuging at 10 000 <span class="jp-bold"><span class="jp-italic">g</span></span>, a 500 µl volume of the supernatant was saved as the total fraction. The rest of the supernatant (~2.5 ml) was centrifuged at 100 000 <span class="jp-bold"><span class="jp-italic">g</span></span> in a TLA100.3 rotor (Beckman TL100 centrifuge) for 1 h and the supernatant and membrane pellet were collected as the soluble fractions and membrane, respectively. The three fractions were normalized to ~0.4 OD<span class="jp-sub">550</span> units, mixed with 2× sample loading buffer, boiled at 100 °C for 10 min, resolved on SDS-PAGE gels and analysed by Western blotting as described above.</p>
</div>
<div class="clearer">&nbsp;</div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="s1-4" id="s1-4">Results</a></div><div class="menuButton">Go to section...</div><div class="clearer">&nbsp;</div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul></div></div><div id="s2-4-1">
<span class="tl-lowest-section">Discovery of a serine residue essential for function allows alignment of diverse L-like amurins</span>
<p>In order to facilitate both cysteine-accessibility studies and also experiments with hetero-bifunctional thiol-specific cross-linkers to identify binding partners, we conducted site-directed mutagenesis on gene <span class="jp-italic">L</span> to create a variety of single-Cys alleles, including five Ser-Cys substitutions in the background L<sup xmlns:f="http://www.example.org/functions">C29S</sup> where the single natural Cys codon was replaced by Ser. Although most of the substitutions had no effect on L lytic function, a single change, S49C, conferred an absolute lysis defect, both in the L<sup xmlns:f="http://www.example.org/functions">C29S</sup> (not shown) and the parental L contexts (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F1">Fig. 1b</a></span>), without affecting L accumulation (not shown). We noticed that this essential Ser residue could be used as part of a Leu-Ser dipeptide motif (LS motif) for aligning the lysis protein sequences, not only from the closely related F-specific Leviviruses but also from Leviviruses that use different conjugational or Type IV pili for infection (<span class="jp-italic">E. coli</span> phages Hgal1 and C1; <span class="jp-italic">Pseudomonas</span> phages PP7 and PRR1) and from the <span class="jp-italic">Acinetobacter</span> <span class="jp-italic">baumannii</span> phage AP205, where the lysis gene evolved in a distinct genomic location (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F1">Fig. 1a, c</a></span>). These lysis genes have been shown to function in <span class="jp-italic">E. coli</span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R19" data-original-title="" title="">19–21</a></span>]. In this alignment, although sequence similarity could not be generally detected, organizing the sequences using the LS motif allowed a putative domain structure for L-like amurins to be assigned: (1) a positively charged N-terminus (4–36 aa, 3–6 net positive charges); (2) a hydrophobic sequence rich in aromatic and large beta-branched aliphatic residues (10–17 aa); (3) the LS dipeptide; and (4) a phage-specific C-terminal domain of variable length (4 to 31 residues) but either neutral or containing one net charge. Based on the previous truncation analysis of L and our results indicating the N-terminal domain confers a dependency on the host DnaJ chaperone, Domain 1 would be dispensable [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R16" data-original-title="" title="">16</a></span>].</p>
<!--splitRids is F1--><!--currentRid is F1--><!--current position is 46--><!--check position is 11--><!--splitRids is F1--><!--currentRid is F1--><!--current position is 47--><!--check position is 11--></div>
<div id="s2-4-2">
<span class="tl-lowest-section">Near-saturating mutational analysis for L lytic function</span>
<p>To interrogate the new L domain structure for functional significance, a mutational analysis involving selection for non-lethal <span class="jp-italic">L</span> alleles was implemented. The general approach was to create a mutagenized library of PCR-mutagenized <span class="jp-italic">L</span> alleles in a plasmid in which the lysis gene was cloned under a tightly regulated arabinose-inducible pBAD promoter [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R22" data-original-title="" title="">22</a></span>]. The transformants carrying this library were plated for surviving cells under inducing conditions. The plasmid DNAs from 396 survivors were extracted and sequenced. Several false positives also survived on the inducer plates, probably due to the lower concentration of arabinose (0.2 %, w/v) in the inducer plates. Nineteen of these alleles were wild-type and three had only silent mutations. Of the remaining 374 alleles, only 139 had a single base change, with the rest having frameshifts or multiple mutations. Of these single base change alleles, 36 mutant alleles had no lysis defect in liquid culture and were scored as false positives, leaving 103 lysis-defective, single base change alleles (Table S2). A summary of this analysis based on a total of 67 unique non-functional single mutation alleles obtained from the lysis-defect selection, as well as a mutant allele obtained by site-directed mutagenesis, is depicted in <span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F2">Fig. 2</a></span> and Table S2. The mutant hunt was close to saturation, as could be judged from the frequency of repeat alleles that were identified late in the procedure. Moreover, of 29 possible nonsense mutations that could be accessed by single base changes in the <span class="jp-italic">L</span> gene, 23 were obtained, distributed throughout the cistron. Generally, the mutational analysis supports the domain concept proposed above. Since Domain 1 has been previously reported to be non-essential, missense alleles in this region conferring a lysis defect were not expected. Accordingly, three of the missense alleles we did obtain in Domain 1 were changes to the start codon (M1I, M1T), which ablate L production entirely, and T3I, T3S, P6L, Q8L and C29R, which exhibited a severe accumulation defect (Table S2). Since the C29S allele is fully functional, we suspect that C29R is also lysis-defective due to proteolytic instability, but because this position is within the epitope used for raising the antibody, this could not be confirmed. Only one allele, Q33H, was lysis-defective and exhibited normal accumulation (Table S2 and <span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F3">Fig. 3</a></span>). The simplest interpretation is that even in the dispensable N-terminal Domain 1, missense changes can occur that block function; e.g. by preventing DnaJ from activating L through interaction with Domain 1. In any case, by far the highest incidence of missense changes with lysis-defective phenotypes was obtained in the LS dipeptide motif, and in the regions of Domains 2 and 4 surrounding the LS sequence, supporting the importance of the motif and its context. Moreover, a significant proportion of the inactivating missense changes were conservative, especially in terms of hydrophobic and polar character, e.g. L44V, F47L, F47Y, S49T (similar to S49C), F51L and L56F. The inactivating effect of such substitutions, coupled with their undiminished levels of protein accumulation (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F3">Fig. 3</a></span>), suggests that L makes a specific protein–protein contact that depends on these mutationally sensitive residues, rather than acting as a membrane-disrupting peptide as previously suggested [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R13" data-original-title="" title="">13</a></span>]. To further test the sensitivity of these positions to other conservative changes, we generated four additional alleles (L44I, A45V, L48I and L48V) by site-directed mutagenesis and followed their lysis profiles in liquid cultures (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F4">Fig. 4a</a></span>). Interestingly, none of these changes blocked the lytic function of L, even L44I, despite the absolute lysis defect associated with L44V. The delayed onset of lysis did not uniformly correlate with the intracellular levels of L (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F4">Fig. 4b</a></span>). Most of the other mutations in Domains 2 and 4 more distal to the key LS motif represent more drastic changes in side-chain character. In addition, all of the alleles tested were recessive to wild-type <span class="jp-italic">L</span>, even when the mutant allele was expressed from a higher-copy plasmid (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F5">Fig. 5</a></span>), suggesting that L lytic function does not require homo-oligomerization but instead requires a heterotypic interaction with a host protein. Finally, all of the defective alleles tested showed unperturbed association with the membrane fraction (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F6">Fig. 6</a></span>). Taken together, the results of the mutational analysis indicate that L has a host membrane protein target and interacts with it through the LS motif and the nearby residues in essential Domains 2 and 4.</p>
<!--splitRids is F2--><!--currentRid is F2--><!--current position is 51--><!--check position is 51--><!--splitRids is F3--><!--currentRid is F3--><!--current position is 52--><!--check position is 52--><!--splitRids is F3--><!--currentRid is F3--><!--current position is 53--><!--check position is 52--><!--splitRids is F4--><!--currentRid is F4--><!--current position is 55--><!--check position is 55--><!--splitRids is F4--><!--currentRid is F4--><!--current position is 56--><!--check position is 55--><!--splitRids is F5--><!--currentRid is F5--><!--current position is 57--><!--check position is 57--><!--splitRids is F6--><!--currentRid is F6--><!--current position is 58--><!--check position is 58--><p xmlns:f="http://www.example.org/functions"></p><div class="figure html-fulltext-responsive-figure" id="F2"><div class="caption"><span class="captionLabel"><span class="label">Fig. 2.</span></span><span class="captionText"><span>Mutational analysis of MS2 <span class="jp-italic">L</span>. The L primary structure is represented as in <span class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F1">Fig. 1(c</a></span>). The four domains of L are represented by numbered boxes. Missense alleles with lysis defects but without a defect in protein accumulation are indicated above the L sequence. Missense changes that do not affect lytic function are indicated below the L sequence. Green asterisks indicate all possible codon positions where a nonsense mutation could be accessed by a single nucleotide change; underlined asterisks indicate positions where no nonsense mutants were obtained in the mutagenesis. Basic and acidic residues are highlighted in red and blue, respectively.</span>
</span></div><div class="image"><a class="media-link" href="https://www.microbiologyresearch.org/docserver/fulltext/micro/163/7/micro000485-f2.gif" id="/content/micro/10.1099/mic.0.000485.F2" data-fancybox="gallery1"><img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/micro000485-f2_thmb.gif" border="0" alt="Fig. 2."><p><span class="figure-duplicate-label">Fig. 2.<br><br></span>Click to view</p></a></div><div class="clearer">&nbsp;</div><div class="downloadAsPptContainer hidden">
<form name="pptFigForm" class="pptForm" action="https://www.microbiologyresearch.org/contentexport/powerpoint/powerpoint.action" method="POST">
<input type="hidden" name="itemId" value="http://instance.metastore.ingenta.com/content/micro/10.1099/mic.0.000485.F2">
<input type="hidden" name="title" value="Fig. 2.">
<input type="hidden" name="caption" value="Mutational analysis of MS2 L. The L primary structure is represented as in 
Fig. 1(c
). The four domains of L are represented by numbered boxes. Missense alleles with lysis defects but without a defect in protein accumulation are indicated above the L sequence. Missense changes that do not affect lytic function are indicated below the L sequence. Green asterisks indicate all possible codon positions where a nonsense mutation could be accessed by a single nucleotide change; underlined asterisks indicate positions where no nonsense mutants were obtained in the mutagenesis. Basic and acidic residues are highlighted in red and blue, respectively.">
<input type="hidden" name="copyright" value="© 2017 The Authors">
<input type="hidden" name="citation" value="">
<input type="hidden" name="imagePath" value="/fulltext/micro/163/7/micro000485-f2.gif">
<input type="hidden" name="logoPath" value="jp/pub2web.gif">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="downloadPpt js-downloadppt">Download as PowerPoint</a>
</form>
</div></div><p></p><p xmlns:f="http://www.example.org/functions"></p><div class="figure html-fulltext-responsive-figure" id="F3"><div class="caption"><span class="captionLabel"><span class="label">Fig. 3.</span></span><span class="captionText"><span>Immunoblots of mutant alleles of <span class="jp-italic">L</span>. Protein samples from various <span class="jp-italic">L</span> mutants were collected 5 min prior to the onset of lysis, normalized to OD<span class="jp-sub">550</span> ~0.4 units and immunoblotted with anti-L antibody. The wild-type (WT), empty vector (Vec) and the mutant alleles are indicated directly above each blot. The molecular mass standard in kDa is represented on the left. The bands corresponding to L are indicated by an arrow on the right.</span>
</span></div><div class="image"><a class="media-link" href="https://www.microbiologyresearch.org/docserver/fulltext/micro/163/7/micro000485-f3.gif" id="/content/micro/10.1099/mic.0.000485.F3" data-fancybox="gallery1"><img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/micro000485-f3_thmb.gif" border="0" alt="Fig. 3."><p><span class="figure-duplicate-label">Fig. 3.<br><br></span>Click to view</p></a></div><div class="clearer">&nbsp;</div><div class="downloadAsPptContainer hidden">
<form name="pptFigForm" class="pptForm" action="https://www.microbiologyresearch.org/contentexport/powerpoint/powerpoint.action" method="POST">
<input type="hidden" name="itemId" value="http://instance.metastore.ingenta.com/content/micro/10.1099/mic.0.000485.F3">
<input type="hidden" name="title" value="Fig. 3.">
<input type="hidden" name="caption" value="Immunoblots of mutant alleles of L. Protein samples from various L mutants were collected 5 min prior to the onset of lysis, normalized to OD550 ~0.4 units and immunoblotted with anti-L antibody. The wild-type (WT), empty vector (Vec) and the mutant alleles are indicated directly above each blot. The molecular mass standard in kDa is represented on the left. The bands corresponding to L are indicated by an arrow on the right.">
<input type="hidden" name="copyright" value="© 2017 The Authors">
<input type="hidden" name="citation" value="">
<input type="hidden" name="imagePath" value="/fulltext/micro/163/7/micro000485-f3.gif">
<input type="hidden" name="logoPath" value="jp/pub2web.gif">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="downloadPpt js-downloadppt">Download as PowerPoint</a>
</form>
</div></div><p></p><p xmlns:f="http://www.example.org/functions"></p><div class="figure html-fulltext-responsive-figure" id="F4"><div class="caption"><span class="captionLabel"><span class="label">Fig. 4.</span></span><span class="captionText"><span>Conservative changes in domains 2 and 3 result in varied lysis profiles. (a) Lysis profiles of isoleucine and valine substitution mutants of residues at positions 44, 45 and 48 of L. ○, <span class="jp-italic">pBAD24 (empty vector)</span>; □, <span class="jp-italic">L</span>; ◊, <span class="jp-italic">L<sup>L44I</sup></span>; X, <span class="jp-italic">L<sup>A45V</sup></span>; ■, <span class="jp-italic">L<sup>L48I</sup></span>; Δ, <span class="jp-italic">L<sup>L48V</sup></span>. (b) Protein samples were collected at the time point indicated by an arrow in (a) and immunoblotted with anti-L antibody. The molecular mass standard (in kDa) and the bands corresponding to L are indicated on the left and right sides of the blot, respectively. The various substitution mutants, wild-type (WT) and empty vector control (Vec) are indicated directly above the blot.</span>
</span></div><div class="image"><a class="media-link" href="https://www.microbiologyresearch.org/docserver/fulltext/micro/163/7/micro000485-f4.gif" id="/content/micro/10.1099/mic.0.000485.F4" data-fancybox="gallery1"><img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/micro000485-f4_thmb.gif" border="0" alt="Fig. 4."><p><span class="figure-duplicate-label">Fig. 4.<br><br></span>Click to view</p></a></div><div class="clearer">&nbsp;</div><div class="downloadAsPptContainer hidden">
<form name="pptFigForm" class="pptForm" action="https://www.microbiologyresearch.org/contentexport/powerpoint/powerpoint.action" method="POST">
<input type="hidden" name="itemId" value="http://instance.metastore.ingenta.com/content/micro/10.1099/mic.0.000485.F4">
<input type="hidden" name="title" value="Fig. 4.">
<input type="hidden" name="caption" value="Conservative changes in domains 2 and 3 result in varied lysis profiles. (a) Lysis profiles of isoleucine and valine substitution mutants of residues at positions 44, 45 and 48 of L. ○, pBAD24 (empty vector); □, L; ◊, LL44I
; X, LA45V
; ■, LL48I
; Δ, LL48V
. (b) Protein samples were collected at the time point indicated by an arrow in (a) and immunoblotted with anti-L antibody. The molecular mass standard (in kDa) and the bands corresponding to L are indicated on the left and right sides of the blot, respectively. The various substitution mutants, wild-type (WT) and empty vector control (Vec) are indicated directly above the blot.">
<input type="hidden" name="copyright" value="© 2017 The Authors">
<input type="hidden" name="citation" value="">
<input type="hidden" name="imagePath" value="/fulltext/micro/163/7/micro000485-f4.gif">
<input type="hidden" name="logoPath" value="jp/pub2web.gif">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="downloadPpt js-downloadppt">Download as PowerPoint</a>
</form>
</div></div><p></p><p xmlns:f="http://www.example.org/functions"></p><div class="figure html-fulltext-responsive-figure" id="F5"><div class="caption"><span class="captionLabel"><span class="label">Fig. 5.</span></span><span class="captionText"><span><span class="jp-italic">L</span> mutant alleles are recessive. The lysis profiles of pBAD24 <span class="jp-italic">L</span> alleles and pKC17 (<span class="jp-italic">his6-L</span>) co-transformants in the TB28 background. Culture growth was monitored by measuring OD<span class="jp-sub">550</span>; cultures were induced at time 0, and OD<span class="jp-sub">550</span> was measured at indicated time points. ○, pBAD24 <span class="jp-italic">L</span> and pKC17; □, pBAD24 <span class="jp-italic">L<sup>K50E</sup></span> and pKC17; ◊, pBAD24 <span class="jp-italic">L<sup>L44V</sup></span> and pKC17; X, pBAD24 <span class="jp-italic">L<sup>F47Y</sup></span> and pKC17; ♦, pBAD24 <span class="jp-italic">L<sup>S49T</sup></span> and pKC17.</span>
</span></div><div class="image"><a class="media-link" href="https://www.microbiologyresearch.org/docserver/fulltext/micro/163/7/micro000485-f5.gif" id="/content/micro/10.1099/mic.0.000485.F5" data-fancybox="gallery1"><img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/micro000485-f5_thmb.gif" border="0" alt="Fig. 5."><p><span class="figure-duplicate-label">Fig. 5.<br><br></span>Click to view</p></a></div><div class="clearer">&nbsp;</div><div class="downloadAsPptContainer hidden">
<form name="pptFigForm" class="pptForm" action="https://www.microbiologyresearch.org/contentexport/powerpoint/powerpoint.action" method="POST">
<input type="hidden" name="itemId" value="http://instance.metastore.ingenta.com/content/micro/10.1099/mic.0.000485.F5">
<input type="hidden" name="title" value="Fig. 5.">
<input type="hidden" name="caption" value="
L mutant alleles are recessive. The lysis profiles of pBAD24 L alleles and pKC17 (his6-L) co-transformants in the TB28 background. Culture growth was monitored by measuring OD550; cultures were induced at time 0, and OD550 was measured at indicated time points. ○, pBAD24 L and pKC17; □, pBAD24 LK50E
and pKC17; ◊, pBAD24 LL44V
and pKC17; X, pBAD24 LF47Y
and pKC17; ♦, pBAD24 LS49T
and pKC17.">
<input type="hidden" name="copyright" value="© 2017 The Authors">
<input type="hidden" name="citation" value="">
<input type="hidden" name="imagePath" value="/fulltext/micro/163/7/micro000485-f5.gif">
<input type="hidden" name="logoPath" value="jp/pub2web.gif">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="downloadPpt js-downloadppt">Download as PowerPoint</a>
</form>
</div></div><p></p><p xmlns:f="http://www.example.org/functions"></p><div class="figure html-fulltext-responsive-figure" id="F6"><div class="caption"><span class="captionLabel"><span class="label">Fig. 6.</span></span><span class="captionText"><span>Both wild-type and mutant alleles of L are membrane-associated. Cell lysates from wild-type and mutant alleles (L44V and S49T) were fractionated into soluble and membrane fractions by high-speed ultra-centrifugation. The normalized fractions were analysed by immunoblotting with anti-L antibody. The molecular mass standard in kDa is represented on the left. Total (T), soluble (S) and membrane (M) fractions are indicated above the blot.</span>
</span></div><div class="image"><a class="media-link" href="https://www.microbiologyresearch.org/docserver/fulltext/micro/163/7/micro000485-f6.gif" id="/content/micro/10.1099/mic.0.000485.F6" data-fancybox="gallery1"><img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/micro000485-f6_thmb.gif" border="0" alt="Fig. 6."><p><span class="figure-duplicate-label">Fig. 6.<br><br></span>Click to view</p></a></div><div class="clearer">&nbsp;</div><div class="downloadAsPptContainer hidden">
<form name="pptFigForm" class="pptForm" action="https://www.microbiologyresearch.org/contentexport/powerpoint/powerpoint.action" method="POST">
<input type="hidden" name="itemId" value="http://instance.metastore.ingenta.com/content/micro/10.1099/mic.0.000485.F6">
<input type="hidden" name="title" value="Fig. 6.">
<input type="hidden" name="caption" value="Both wild-type and mutant alleles of L are membrane-associated. Cell lysates from wild-type and mutant alleles (L44V and S49T) were fractionated into soluble and membrane fractions by high-speed ultra-centrifugation. The normalized fractions were analysed by immunoblotting with anti-L antibody. The molecular mass standard in kDa is represented on the left. Total (T), soluble (S) and membrane (M) fractions are indicated above the blot.">
<input type="hidden" name="copyright" value="© 2017 The Authors">
<input type="hidden" name="citation" value="">
<input type="hidden" name="imagePath" value="/fulltext/micro/163/7/micro000485-f6.gif">
<input type="hidden" name="logoPath" value="jp/pub2web.gif">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="downloadPpt js-downloadppt">Download as PowerPoint</a>
</form>
</div></div><p></p></div>
<div class="clearer">&nbsp;</div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="s1-5" id="s1-5">Discussion</a></div><div class="menuButton">Go to section...</div><div class="clearer">&nbsp;</div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul></div></div><div id="s2-5-1">
<span class="tl-lowest-section">New perspective on L mode of action</span>
<p>Despite its early identification as the lysis protein in one of the most intensively studied phages of all time, MS2 L has long eluded operational or mechanistic characterization at the molecular level. Until recently, the extant literature on L focused on a consensus in which its essential C-terminal domain acted as a membrane-disrupting polypeptide that subverted the energization of the bi-layer and somehow induced an autolytic response involving unknown host proteins [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R13" data-original-title="" title="">13</a></span>]. We have interrogated L function by genetic analysis based on analysis of survivors of L expression from a plasmid vector. First, this approach revealed that the host DnaJ chaperone was required for L function, interacting with the N-terminal, highly basic domain of L [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R16" data-original-title="" title="">16</a></span>]. Both this domain and thus the DnaJ interaction were shown to be dispensable, and a model was proposed in which the N-terminus acted as a regulatory domain that inhibited the interaction of L with a target protein. In this scenario, then, DnaJ acts not as the target but instead binds the regulatory domain and displaces it from its inhibitory position. Here, in the next phase of our genetic approach, we have probed the full length of the L protein by mutational analysis, revealing a distinct domain structure with differential mutational sensitivity and highlighting a dipeptide motif, Leu48-Ser49, that is conserved in heterologous amurins. In addition, many of the missense changes found to abolish lytic function without affecting accumulation or membrane-localization of L are very conservative and indicative of a protein–protein interface (e.g. L44V is defective but L44I is fully functional). Taken together, these data support a model in which the LS motif and its neighbouring subdomains form the core of an essential heterotypic protein–protein interaction domain. Moreover, the fact that all the inactivating mutations are recessive suggests that L does not act as an oligomeric lytic factor, as would be the case for models in which L is proposed to form membrane-permeating lesions.</p>
</div>
<div id="s2-5-2">
<span class="tl-lowest-section">Evolution of L-like amurins</span>
<p>The study of ssRNA phages began with the F-pilus-specific RNA phages of <span class="jp-italic">E. coli</span>, and indeed the original classification of these Leviviridae into two genera, the Leviviruses and Alloleviviruses. This was largely dependent on the fact that the former had an independent lysis gene <span class="jp-italic">L</span>, overlapping the <span class="jp-italic">coat</span> and <span class="jp-italic">rep</span> genes, whereas the latter did not, instead effecting lysis with a secondary function of the Mat protein (A<span class="jp-sub">2</span> in Qβ) [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R7" data-original-title="" title="">7, 23</a></span>]. Many Leviviridae, either MS2-like or Qβ-like based on the arrangement of genes, were sequenced, and in all the MS2-like phages, the L sequences were related (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F1">Fig. 1c</a></span>). However, more recently the genome sequences of seven Leviviridae specific for other retractable pili have become available, four (PRR1, Hgal1, M and C-1) for conjugational pili of R-factor plasmids [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R10" data-original-title="" title="">10, 19, 20</a></span>], two for polar pili (PP7 for <span class="jp-italic">Pseudomonas</span>; phiCb5 for <span class="jp-italic">Caulobacter crescentus</span>) [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R24" data-original-title="" title="">24</a></span>] and one for a Type IV pilus (AP205 for <span class="jp-italic">A. baumannii</span>) [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R21" data-original-title="" title="">21</a></span>]. Although there is no significant sequence similarity, the arrangements of the genes in PRR1, Hgal1, C-1 and PP7 all resemble that of MS2, with a small ORF overlapping the <span class="jp-italic">coat-rep</span> gene junction. In each case, the small gene was identified as the lysis gene by testing the induction of a plasmid-based clone. However, in the other three phages, AP205, M and phiCb5, there was no ORF at the <span class="jp-italic">coat-rep</span> junction [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R10" data-original-title="" title="">10, 21, 24</a></span>]. In AP205 and M, the lysis gene was identified by cloning and testing for inducible lysis, and in both cases, its position was novel: in AP205, it was a separate gene at the 5′ end of the genome, whereas in M, it was a gene that is embedded out of frame at a different position within <span class="jp-italic">rep</span>. Very recently, the lysis protein of the latter, Lys<sup xmlns:f="http://www.example.org/functions">M</sup>, was shown to be an inhibitor of MurJ, the Lipid II flippase of <span class="jp-italic">E. coli</span>, and thus, operationally, resembles the A<span class="jp-sub">2</span> (Mat) protein of Qβ, a specific inhibitor of MurA, the first enzyme in the peptidoglycan biosynthesis pathway; in both cases, lysis is effected by inhibition of cell-wall synthesis. The work we have reported here, with the identification of the LS motif and the domain structure of L, clearly groups all the lysis proteins found encoded at the <span class="jp-italic">coat-rep</span> junction and the AP205 lysis protein with MS2 L. Moreover, the implication is that the L-mediated pathway to lysis, which operates without affecting net cell-wall biosynthesis, interacts with a conserved target common to these diverse Gram-negative genera. In addition, since the <span class="jp-italic">L</span> gene in AP205 is located in a completely different place in the genome compared to the <span class="jp-italic">L</span> genes straddling the <span class="jp-italic">coat-rep</span> boundary, it seems clear that the conservation of the LS motif and the domain structure derives from convergent evolution that occurred after speciation of a progenitor Leviviridae to the respective retractable pili. This may also be true for the L proteins in PRR1, Hgal1, C-1 and PP7, which, although occupying the same <span class="jp-italic">coat-rep</span> junction region of their respective genomes, share no sequence homology. In support of this notion, other work has shown that for MS2, the region of <span class="jp-italic">rep</span> within which the key hydrophobic and LS elements are encoded in the out-of-frame <span class="jp-italic">L</span> cistron in MS2 is not conserved, so the emergence of <span class="jp-italic">L</span>-like genes in this region may simply reflect the limited availability of genomic space for sequence variation.</p>
<!--splitRids is F1--><!--currentRid is F1--><!--current position is 63--><!--check position is 11--></div>
<div id="s2-5-3">
<span class="tl-lowest-section">Comparative analysis of the domain structure of L proteins</span>
<p>Based on our mutational analysis of MS2 L, we have postulated four domains as significant for lytic function of L-like amurins, including the dipeptide sequence LS as a completely conserved motif (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F2">Fig. 2</a></span>). Assuming the LS sequence has to function from the same molecular context in all of the L-like proteins, then a comparative analysis of the putative domain structure in unrelated L-like amurins may make it possible to infer properties of subcellular localization, membrane topology and potential target proteins. We have chosen, as a comparison, a set of six L sequences from F-specific <span class="jp-italic">Leviviridae</span>, which we designate as the homologous set, and the five L-like proteins from <span class="jp-italic">Leviviridae</span> specific for other retractable pili, here designated as the heterologous set (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F1">Fig. 1c</a></span>). Confidence for the meaningfulness of this comparison stems from the fact that the L sequences from four of the five heterologous phages (Hgal1, C1, AP205 and PRR1) have been shown to be necessary and sufficient for lysis if induced from a plasmid clone in <span class="jp-italic">E. coli</span>. Preliminary results from this laboratory have demonstrated similar lytic capacity for the cloned <span class="jp-italic">L</span> gene of PP7 [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R16" data-original-title="" title="">16</a></span>].</p>
<!--splitRids is F2--><!--currentRid is F2--><!--current position is 68--><!--check position is 51--><!--splitRids is F1--><!--currentRid is F1--><!--current position is 69--><!--check position is 11--><p>The simplest conclusions from the alignment pertain to the non-essential Domain 1, where there are clearly common characteristics for all the L proteins. In all cases, Domain 1 is marked by a high content of charged residues, biased towards basic residues. Despite a wide variation in size, from seven residues in AP205 to 36 in MS2 L, Domain 1 in all L proteins has a net predicted charge of +3 to +6. The simplest notion is that, although L is firmly associated with the membrane, Domain 1 is disposed in the cytosol, consistent with its ability to bind DnaJ. Unstructured basic domains have been identified in other amurins [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R25" data-original-title="" title="">25, 26</a></span>] and may serve in initial localization to the surface of the membrane.</p>
<p>The simplest approach to a comparative analysis requires the assumption that Domain 1 will prove to be non-essential for the heterologous L proteins. On this basis, the alignment of L with the five heterologous L sequences was used for seeding secondary structure analysis of L using the Jpred algorithm [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R27" data-original-title="" title="">27</a></span>], resulting in a high-confidence prediction for alpha-helical structure from Leu37 to the C-terminus of the molecule (Fig. S1). This predicted alpha-helical domain coincides exactly with Domains 2–4, the essential domains for L function. Moreover, the TMHMM algorithm, widely used for transmembrane domain (TMD) prediction, predicts with uniformly high confidence that L and its five functional homologues are integral membrane proteins, which agrees with the behaviour of L in subcellular fractionation experiments, even when its N-terminal domain is in complex with DnaJ [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R16" data-original-title="" title="">16</a></span>]. In each case, the predicted membrane-embedded domain begins at the start of Domain 2 and extends through the LS motif and into Domain 4. In addition, a by-product of the <span class="jp-italic">L</span> mutagenesis was the isolation of two Pro substitution alleles, L44P and A45P, with normal lysis function (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#F2">Fig. 2</a></span>). Although proline residues are severe helix-breakers in solvent-exposed structures, they are often tolerated at the ends of TMDs [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R28" data-original-title="" title="">28</a></span>], suggesting that this domain is embedded in the bi-layer. Taken together, these considerations suggest a structural model in which all of the essential residues in L are in a helical structure that spans the bilayer. However, the existence of a TMD is not easily reconciled with the detailed primary structure of the L proteins from the F-specific homologous set. In each of these cases, there is a charged residue (Lys50 in L) adjacent to the LS motif. Unbalanced charged residues are rarely found in TMDs in bacterial integral membrane proteins, except in some transport proteins where there is solvent exposure deep in the membrane-embedded helices [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R29" data-original-title="" title="">29</a></span>].</p>
<!--splitRids is F2--><!--currentRid is F2--><!--current position is 74--><!--check position is 51--><p>Thus, although MS2 L, and, by extension, the other L proteins are undoubtedly membrane proteins, canonical requirements for a simple TMD cannot be met. Moreover, even for the heterologous L sequences, where it is not necessary to embed a charged residue to accommodate a TMD, the predicted TMD would also place the LS motif within the bi-layer, where specific interactions presumably including the H-bonding capacity of the serine hydroxyl group would be hard to envision unless the membrane is unusually distorted. However, recent findings suggest that membrane distortion might be integral to normal cell-wall growth. To account for homeostatic growth of the PG sacculus, models for the recruitment of the PG biosynthetic complexes to sites of negative membrane curvature have been proposed and supported by <span class="jp-italic">in vivo</span> and <span class="jp-italic">in silico</span> experiments [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R30" data-original-title="" title="">30, 31</a></span>]. Disruption of such a system by L interacting with a membrane-embedded target protein would account for the ability of L proteins to effect lysis without inhibiting net incorporation of PG into the existing wall, as has been reported for MS2 L [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#R32" data-original-title="" title="">32, 33</a></span>]. Thus interaction of L with its target, perhaps via attraction to distorted membrane, would lead to a lytic catastrophe. To address this general hypothesis for L lysis, experiments are underway to assess the distribution of L molecules relative to the sites of PG biosynthesis.</p>
</div>
<div class="clearer">&nbsp;</div></div>
</div>

<div id="article-level-0-back">
<div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="ack1" id="ack1">Funding information</a></div><div class="menuButton">Go to section...</div><div class="clearer">&nbsp;</div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul></div></div><p>This work was supported by Public Health Service grant GM27099 and by the Center for Phage Technology at Texas A&amp;M University, jointly sponsored by Texas A&amp;M AgriLife.</p>
<div class="clearer">&nbsp;</div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="ack2" id="ack2">Conflicts of interest</a></div><div class="menuButton">Go to section...</div><div class="clearer">&nbsp;</div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul></div></div><p>The authors declare that there are no conflicts of interest.</p>
<div class="clearer">&nbsp;</div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="ack3" id="ack3">Acknowledgements</a></div><div class="menuButton">Go to section...</div><div class="clearer">&nbsp;</div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul></div></div><p>The constructs, preliminary results and analysis pertaining to the cysteine alleles of <span class="jp-italic">L</span> have been previously reported in the dissertation of Dr Brenley McIntosh. The clerical assistance of Daisy Wilbert at Center for Phage Technology is highly appreciated.</p>
<div class="clearer">&nbsp;</div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="references-1" id="references-1">References</a></div><div class="menuButton">Go to section...</div><div class="clearer">&nbsp;</div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#abstract-1">ABSTRACT</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-1">Abbreviations</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-2">Introduction</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-3">Methods</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-4">Results</a></li><li><a class="menuLink body-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#s1-5">Discussion</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack2">Conflicts of interest</a></li><li><a class="menuLink ack-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#ack3">Acknowledgements</a></li><li><a class="menuLink ref-section-link" href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#references-1">References</a></li></ul></div></div><span class="references"><ol id="articlereference">
<li class="refbody" id="R1">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R1"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">Wang</span> <span class="reference-given-names">IN</span>, <span class="reference-surname">Struck</span> <span class="reference-given-names">DK</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Breaking free: "protein antibiotics" and phage lysis. </span> <span class="reference-source">Res Microbiol</span> <span class="reference-year">2002</span>; 153:<span class="reference-fpage">493–</span><span class="reference-lpage">501</span><a target="xrefwindow" href="https://doi.org/10.1016/S0923-2508(02)01330-X"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/12437210">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Breaking+free%3A+%22protein+antibiotics%22+and+phage+lysis&amp;author=TG+Bernhardt&amp;author=IN+Wang&amp;author=DK+Struck&amp;author=R+Young&amp;journal=Res+Microbiol&amp;volume=153&amp;doi=10.1016%2FS0923-2508%2802%2901330-X&amp;pages=493-501&amp;publication_year=2002&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R2">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R2"></a><div class="citation"> <span class="reference-surname">Sanger</span> <span class="reference-given-names">F</span>, <span class="reference-surname">Coulson</span> <span class="reference-given-names">AR</span>, <span class="reference-surname">Friedmann</span> <span class="reference-given-names">T</span>, <span class="reference-surname">Air</span> <span class="reference-given-names">GM</span>, <span class="reference-surname">Barrell</span> <span class="reference-given-names">BG</span><span class="reference-etal"> et al.</span> <span class="reference-article-title"> The nucleotide sequence of bacteriophage φX174. </span> <span class="reference-source">J Mol Biol</span> <span class="reference-year">1978</span>; 125:<span class="reference-fpage">225–</span><span class="reference-lpage">246</span><a target="xrefwindow" href="https://doi.org/10.1016/0022-2836(78)90346-7"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/731693">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=The+nucleotide+sequence+of+bacteriophage+%CF%86X174&amp;author=F+Sanger&amp;author=AR+Coulson&amp;author=T+Friedmann&amp;author=GM+Air&amp;author=BG+Barrell&amp;journal=J+Mol+Biol&amp;volume=125&amp;doi=10.1016%2F0022-2836%2878%2990346-7&amp;pages=225-246&amp;publication_year=1978&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R3">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R3"></a><div class="citation"> <span class="reference-surname">Winter</span> <span class="reference-given-names">RB</span>, <span class="reference-surname">Gold</span> <span class="reference-given-names">L</span>. <span class="reference-article-title"> Overproduction of bacteriophage Qβ maturation (A<sub>2</sub>) protein leads to cell lysis. </span> <span class="reference-source">Cell</span> <span class="reference-year">1983</span>; 33:<span class="reference-fpage">877–</span><span class="reference-lpage">885</span><a target="xrefwindow" href="https://doi.org/10.1016/0092-8674(83)90030-2"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/6871998">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Overproduction+of+bacteriophage+Q%CE%B2+maturation+%28A2%29+protein+leads+to+cell+lysis&amp;author=RB+Winter&amp;author=L+Gold&amp;journal=Cell&amp;volume=33&amp;doi=10.1016%2F0092-8674%2883%2990030-2&amp;pages=877-885&amp;publication_year=1983&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R4">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R4"></a><div class="citation"> <span class="reference-surname">Lubitz</span> <span class="reference-given-names">W</span>, <span class="reference-surname">Halfmann</span> <span class="reference-given-names">G</span>, <span class="reference-surname">Plapp</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Lysis of <span class="reference-italic">Escherichia coli</span> after infection with φX174 depends on the regulation of the cellular autolytic system. </span> <span class="reference-source">J Gen Microbiol</span> <span class="reference-year">1984</span>; 130:<span class="reference-fpage">1079–</span><span class="reference-lpage">1087</span><a target="xrefwindow" href="https://doi.org/10.1099/00221287-130-5-1079"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/6236279">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Lysis+of+Escherichia+coli+after+infection+with+%CF%86X174+depends+on+the+regulation+of+the+cellular+autolytic+system&amp;author=W+Lubitz&amp;author=G+Halfmann&amp;author=R+Plapp&amp;journal=J+Gen+Microbiol&amp;volume=130&amp;doi=10.1099%2F00221287-130-5-1079&amp;pages=1079-1087&amp;publication_year=1984&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R5">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R5"></a><div class="citation"> <span class="reference-surname">Witte</span> <span class="reference-given-names">A</span>, <span class="reference-surname">Wanner</span> <span class="reference-given-names">G</span>, <span class="reference-surname">Bläsi</span> <span class="reference-given-names">U</span>, <span class="reference-surname">Halfmann</span> <span class="reference-given-names">G</span>, <span class="reference-surname">Szostak</span> <span class="reference-given-names">M</span><span class="reference-etal"> et al.</span> <span class="reference-article-title"> Endogenous transmembrane tunnel formation mediated by φX174 lysis protein E. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">1990</span>; 172:<span class="reference-fpage">4109–</span><span class="reference-lpage">4114</span><a target="xrefwindow" href="https://doi.org/10.1128/jb.172.7.4109-4114.1990"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/2141836">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Endogenous+transmembrane+tunnel+formation+mediated+by+%CF%86X174+lysis+protein+E&amp;author=A+Witte&amp;author=G+Wanner&amp;author=U+Bl%C3%A4si&amp;author=G+Halfmann&amp;author=M+Szostak&amp;journal=J+Bacteriol&amp;volume=172&amp;doi=10.1128%2Fjb.172.7.4109-4114.1990&amp;pages=4109-4114&amp;publication_year=1990&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R6">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R6"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">Roof</span> <span class="reference-given-names">WD</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Genetic evidence that the bacteriophage φX174 lysis protein inhibits cell wall synthesis. </span> <span class="reference-source">Proc Natl Acad Sci USA</span> <span class="reference-year">2000</span>; 97:<span class="reference-fpage">4297–</span><span class="reference-lpage">4302</span><a target="xrefwindow" href="https://doi.org/10.1073/pnas.97.8.4297"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/10760296">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Genetic+evidence+that+the+bacteriophage+%CF%86X174+lysis+protein+inhibits+cell+wall+synthesis&amp;author=TG+Bernhardt&amp;author=WD+Roof&amp;author=R+Young&amp;journal=Proc+Natl+Acad+Sci+USA&amp;volume=97&amp;doi=10.1073%2Fpnas.97.8.4297&amp;pages=4297-4302&amp;publication_year=2000&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R7">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R7"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">Wang</span> <span class="reference-given-names">IN</span>, <span class="reference-surname">Struck</span> <span class="reference-given-names">DK</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> A protein antibiotic in the phage Qβ virion: diversity in lysis targets. </span> <span class="reference-source">Science</span> <span class="reference-year">2001</span>; 292:<span class="reference-fpage">2326–</span><span class="reference-lpage">2329</span><a target="xrefwindow" href="https://doi.org/10.1126/science.1058289"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/11423662">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=A+protein+antibiotic+in+the+phage+Q%CE%B2+virion%3A+diversity+in+lysis+targets&amp;author=TG+Bernhardt&amp;author=IN+Wang&amp;author=DK+Struck&amp;author=R+Young&amp;journal=Science&amp;volume=292&amp;doi=10.1126%2Fscience.1058289&amp;pages=2326-2329&amp;publication_year=2001&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R8">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R8"></a><div class="citation"> <span class="reference-surname">Zheng</span> <span class="reference-given-names">Y</span>, <span class="reference-surname">Struck</span> <span class="reference-given-names">DK</span>, <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Genetic analysis of MraY inhibition by the φX174 protein E. </span> <span class="reference-source">Genetics</span> <span class="reference-year">2008</span>; 180:<span class="reference-fpage">1459–</span><span class="reference-lpage">1466</span><a target="xrefwindow" href="https://doi.org/10.1534/genetics.108.093443"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/18791230">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Genetic+analysis+of+MraY+inhibition+by+the+%CF%86X174+protein+E&amp;author=Y+Zheng&amp;author=DK+Struck&amp;author=TG+Bernhardt&amp;author=R+Young&amp;journal=Genetics&amp;volume=180&amp;doi=10.1534%2Fgenetics.108.093443&amp;pages=1459-1466&amp;publication_year=2008&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R9">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R9"></a><div class="citation"> <span class="reference-surname">Zheng</span> <span class="reference-given-names">Y</span>, <span class="reference-surname">Struck</span> <span class="reference-given-names">DK</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Purification and functional characterization of φX174 lysis protein E. </span> <span class="reference-source">Biochemistry</span> <span class="reference-year">2009</span>; 48:<span class="reference-fpage">4999–</span><span class="reference-lpage">5006</span><a target="xrefwindow" href="https://doi.org/10.1021/bi900469g"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/19379010">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Purification+and+functional+characterization+of+%CF%86X174+lysis+protein+E&amp;author=Y+Zheng&amp;author=DK+Struck&amp;author=R+Young&amp;journal=Biochemistry&amp;volume=48&amp;doi=10.1021%2Fbi900469g&amp;pages=4999-5006&amp;publication_year=2009&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R10">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R10"></a><div class="citation"> <span class="reference-surname">Rumnieks</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Tars</span> <span class="reference-given-names">K</span>. <span class="reference-article-title"> Diversity of pili-specific bacteriophages: genome sequence of IncM plasmid-dependent RNA phage M. </span> <span class="reference-source">BMC Microbiol</span> <span class="reference-year">2012</span>; 12:<span class="reference-fpage">277</span><a target="xrefwindow" href="https://doi.org/10.1186/1471-2180-12-277"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/23176223">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Diversity+of+pili-specific+bacteriophages%3A+genome+sequence+of+IncM+plasmid-dependent+RNA+phage+M&amp;author=J+Rumnieks&amp;author=K+Tars&amp;journal=BMC+Microbiol&amp;volume=12&amp;doi=10.1186%2F1471-2180-12-277&amp;pages=277&amp;publication_year=2012&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R11">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R11"></a><div class="citation"> <span class="reference-surname">Beremand</span> <span class="reference-given-names">MN</span>, <span class="reference-surname">Blumenthal</span> <span class="reference-given-names">T</span>. <span class="reference-article-title"> Overlapping genes in RNA phage: a new protein implicated in lysis. </span> <span class="reference-source">Cell</span> <span class="reference-year">1979</span>; 18:<span class="reference-fpage">257–</span><span class="reference-lpage">266</span><a target="xrefwindow" href="https://doi.org/10.1016/0092-8674(79)90045-X"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/387256">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Overlapping+genes+in+RNA+phage%3A+a+new+protein+implicated+in+lysis&amp;author=MN+Beremand&amp;author=T+Blumenthal&amp;journal=Cell&amp;volume=18&amp;doi=10.1016%2F0092-8674%2879%2990045-X&amp;pages=257-266&amp;publication_year=1979&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R12">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R12"></a><div class="citation"> <span class="reference-surname">Berkhout</span> <span class="reference-given-names">B</span>, <span class="reference-surname">de Smit</span> <span class="reference-given-names">MH</span>, <span class="reference-surname">Spanjaard</span> <span class="reference-given-names">RA</span>, <span class="reference-surname">Blom</span> <span class="reference-given-names">T</span>, <span class="reference-surname">van Duin</span> <span class="reference-given-names">J</span>. <span class="reference-article-title"> The amino terminal half of the MS2-coded lysis protein is dispensable for function: implications for our understanding of coding region overlaps. </span> <span class="reference-source">EMBO J</span> <span class="reference-year">1985</span>; 4:<span class="reference-fpage">3315–</span><span class="reference-lpage">3320</span><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/3912168">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=The+amino+terminal+half+of+the+MS2-coded+lysis+protein+is+dispensable+for+function%3A+implications+for+our+understanding+of+coding+region+overlaps&amp;author=B+Berkhout&amp;author=MH+de+Smit&amp;author=RA+Spanjaard&amp;author=T+Blom&amp;author=J+van+Duin&amp;journal=EMBO+J&amp;volume=4&amp;pages=3315-3320&amp;publication_year=1985&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R13">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R13"></a><div class="citation"> <span class="reference-surname">Goessens</span> <span class="reference-given-names">WH</span>, <span class="reference-surname">Driessen</span> <span class="reference-given-names">AJ</span>, <span class="reference-surname">Wilschut</span> <span class="reference-given-names">J</span>, <span class="reference-surname">van Duin</span> <span class="reference-given-names">J</span>. <span class="reference-article-title"> A synthetic peptide corresponding to the C-terminal 25 residues of phage MS2 coded lysis protein dissipates the protonmotive force in <span class="reference-italic">Escherichia coli</span> membrane vesicles by generating hydrophilic pores. </span> <span class="reference-source">Embo J</span> <span class="reference-year">1988</span>; 7:<span class="reference-fpage">867–</span><span class="reference-lpage">873</span><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/2840287">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=A+synthetic+peptide+corresponding+to+the+C-terminal+25+residues+of+phage+MS2+coded+lysis+protein+dissipates+the+protonmotive+force+in+Escherichia+coli+membrane+vesicles+by+generating+hydrophilic+pores&amp;author=WH+Goessens&amp;author=AJ+Driessen&amp;author=J+Wilschut&amp;author=J+van+Duin&amp;journal=Embo+J&amp;volume=7&amp;pages=867-873&amp;publication_year=1988&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R14">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R14"></a><div class="citation"> <span class="reference-surname">Walderich</span> <span class="reference-given-names">B</span>, <span class="reference-surname">Höltje</span> <span class="reference-given-names">JV</span>. <span class="reference-article-title"> Specific localization of the lysis protein of bacteriophage MS2 in membrane adhesion sites of <span class="reference-italic">Escherichia coli</span>. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">1989</span>; 171:<span class="reference-fpage">3331–</span><span class="reference-lpage">3336</span><a target="xrefwindow" href="https://doi.org/10.1128/jb.171.6.3331-3336.1989"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/2656650">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Specific+localization+of+the+lysis+protein+of+bacteriophage+MS2+in+membrane+adhesion+sites+of+Escherichia+coli&amp;author=B+Walderich&amp;author=JV+H%C3%B6ltje&amp;journal=J+Bacteriol&amp;volume=171&amp;doi=10.1128%2Fjb.171.6.3331-3336.1989&amp;pages=3331-3336&amp;publication_year=1989&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R15">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R15"></a><div class="citation"> <span class="reference-surname">Walderich</span> <span class="reference-given-names">B</span>, <span class="reference-surname">Ursinus-Wössner</span> <span class="reference-given-names">A</span>, <span class="reference-surname">van Duin</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Höltje</span> <span class="reference-given-names">JV</span>. <span class="reference-article-title"> Induction of the autolytic system of <span class="reference-italic">Escherichia coli</span> by specific insertion of bacteriophage MS2 lysis protein into the bacterial cell envelope. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">1988</span>; 170:<span class="reference-fpage">5027–</span><span class="reference-lpage">5033</span><a target="xrefwindow" href="https://doi.org/10.1128/jb.170.11.5027-5033.1988"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/3053640">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Induction+of+the+autolytic+system+of+Escherichia+coli+by+specific+insertion+of+bacteriophage+MS2+lysis+protein+into+the+bacterial+cell+envelope&amp;author=B+Walderich&amp;author=A+Ursinus-W%C3%B6ssner&amp;author=J+van+Duin&amp;author=JV+H%C3%B6ltje&amp;journal=J+Bacteriol&amp;volume=170&amp;doi=10.1128%2Fjb.170.11.5027-5033.1988&amp;pages=5027-5033&amp;publication_year=1988&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R16">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R16"></a><div class="citation"> <span class="reference-surname">Chamakura</span> <span class="reference-given-names">KR</span>, <span class="reference-surname">Tran</span> <span class="reference-given-names">JS</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> MS2 lysis of <span class="reference-italic">Escherichia coli</span> depends on host chaperone DnaJ. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">2017</span>; 199:<span class="reference-elocation-id">e00058-17</span><a target="xrefwindow" href="https://doi.org/10.1128/JB.00058-17"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/28396351">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=MS2+lysis+of+Escherichia+coli+depends+on+host+chaperone+DnaJ&amp;author=KR+Chamakura&amp;author=JS+Tran&amp;author=R+Young&amp;journal=J+Bacteriol&amp;volume=199&amp;doi=10.1128%2FJB.00058-17&amp;publication_year=2017&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R17">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R17"></a><div class="citation"> <span class="reference-surname">Smith</span> <span class="reference-given-names">DL</span>, <span class="reference-surname">Chang</span> <span class="reference-given-names">CY</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> The λ holin accumulates beyond the lethal triggering concentration under hyperexpression conditions. </span> <span class="reference-source">Gene Expr</span> <span class="reference-year">1998</span>; 7:<span class="reference-fpage">39–</span><span class="reference-lpage">52</span><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/9572396">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=The+%CE%BB+holin+accumulates+beyond+the+lethal+triggering+concentration+under+hyperexpression+conditions&amp;author=DL+Smith&amp;author=CY+Chang&amp;author=R+Young&amp;journal=Gene+Expr&amp;volume=7&amp;pages=39-52&amp;publication_year=1998&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R18">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R18"></a><div class="citation"> <span class="reference-surname">Moussa</span> <span class="reference-given-names">SH</span>, <span class="reference-surname">Kuznetsov</span> <span class="reference-given-names">V</span>, <span class="reference-surname">Tran</span> <span class="reference-given-names">TA</span>, <span class="reference-surname">Sacchettini</span> <span class="reference-given-names">JC</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Protein determinants of phage T4 lysis inhibition. </span> <span class="reference-source">Protein Sci</span> <span class="reference-year">2012</span>; 21:<span class="reference-fpage">571–</span><span class="reference-lpage">582</span><a target="xrefwindow" href="https://doi.org/10.1002/pro.2042"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/22389108">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Protein+determinants+of+phage+T4+lysis+inhibition&amp;author=SH+Moussa&amp;author=V+Kuznetsov&amp;author=TA+Tran&amp;author=JC+Sacchettini&amp;author=R+Young&amp;journal=Protein+Sci&amp;volume=21&amp;doi=10.1002%2Fpro.2042&amp;pages=571-582&amp;publication_year=2012&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R19">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R19"></a><div class="citation"> <span class="reference-surname">Ruokoranta</span> <span class="reference-given-names">TM</span>, <span class="reference-surname">Grahn</span> <span class="reference-given-names">AM</span>, <span class="reference-surname">Ravantti</span> <span class="reference-given-names">JJ</span>, <span class="reference-surname">Poranen</span> <span class="reference-given-names">MM</span>, <span class="reference-surname">Bamford</span> <span class="reference-given-names">DH</span>. <span class="reference-article-title"> Complete genome sequence of the broad host range single-stranded RNA phage PRR1 places it in the <span class="reference-italic">Levivirus</span> genus with characteristics shared with Alloleviviruses. </span> <span class="reference-source">J Virol</span> <span class="reference-year">2006</span>; 80:<span class="reference-fpage">9326–</span><span class="reference-lpage">9330</span><a target="xrefwindow" href="https://doi.org/10.1128/JVI.01005-06"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/16940544">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Complete+genome+sequence+of+the+broad+host+range+single-stranded+RNA+phage+PRR1+places+it+in+the+Levivirus+genus+with+characteristics+shared+with+Alloleviviruses&amp;author=TM+Ruokoranta&amp;author=AM+Grahn&amp;author=JJ+Ravantti&amp;author=MM+Poranen&amp;author=DH+Bamford&amp;journal=J+Virol&amp;volume=80&amp;doi=10.1128%2FJVI.01005-06&amp;pages=9326-9330&amp;publication_year=2006&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R20">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R20"></a><div class="citation"> <span class="reference-surname">Kannoly</span> <span class="reference-given-names">S</span>, <span class="reference-surname">Shao</span> <span class="reference-given-names">Y</span>, <span class="reference-surname">Wang</span> <span class="reference-given-names">IN</span>. <span class="reference-article-title"> Rethinking the evolution of single-stranded RNA (ssRNA) bacteriophages based on genomic sequences and characterizations of two R-plasmid-dependent ssRNA phages, C-1 and Hgal1. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">2012</span>; 194:<span class="reference-fpage">5073–</span><span class="reference-lpage">5079</span><a target="xrefwindow" href="https://doi.org/10.1128/JB.00929-12"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/22821966">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Rethinking+the+evolution+of+single-stranded+RNA+%28ssRNA%29+bacteriophages+based+on+genomic+sequences+and+characterizations+of+two+R-plasmid-dependent+ssRNA+phages%2C+C-1+and+Hgal1&amp;author=S+Kannoly&amp;author=Y+Shao&amp;author=IN+Wang&amp;journal=J+Bacteriol&amp;volume=194&amp;doi=10.1128%2FJB.00929-12&amp;pages=5073-5079&amp;publication_year=2012&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R21">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R21"></a><div class="citation"> <span class="reference-surname">Klovins</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Overbeek</span> <span class="reference-given-names">GP</span>, <span class="reference-surname">van den Worm</span> <span class="reference-given-names">SH</span>, <span class="reference-surname">Ackermann</span> <span class="reference-given-names">HW</span>, <span class="reference-surname">van Duin</span> <span class="reference-given-names">J</span>. <span class="reference-article-title"> Nucleotide sequence of a ssRNA phage from <span class="reference-italic">Acinetobacter</span>: kinship to coliphages. </span> <span class="reference-source">J Gen Virol</span> <span class="reference-year">2002</span>; 83:<span class="reference-fpage">1523–</span><span class="reference-lpage">1533</span><a target="xrefwindow" href="https://doi.org/10.1099/0022-1317-83-6-1523"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/12029168">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Nucleotide+sequence+of+a+ssRNA+phage+from+Acinetobacter%3A+kinship+to+coliphages&amp;author=J+Klovins&amp;author=GP+Overbeek&amp;author=SH+van+den+Worm&amp;author=HW+Ackermann&amp;author=J+van+Duin&amp;journal=J+Gen+Virol&amp;volume=83&amp;doi=10.1099%2F0022-1317-83-6-1523&amp;pages=1523-1533&amp;publication_year=2002&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R22">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R22"></a><div class="citation"> <span class="reference-surname">Guzman</span> <span class="reference-given-names">LM</span>, <span class="reference-surname">Belin</span> <span class="reference-given-names">D</span>, <span class="reference-surname">Carson</span> <span class="reference-given-names">MJ</span>, <span class="reference-surname">Beckwith</span> <span class="reference-given-names">J</span>. <span class="reference-article-title"> Tight regulation, modulation, and high-level expression by vectors containing the arabinose P<sub>BAD</sub> promoter. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">1995</span>; 177:<span class="reference-fpage">4121–</span><span class="reference-lpage">4130</span><a target="xrefwindow" href="https://doi.org/10.1128/jb.177.14.4121-4130.1995"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/7608087">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Tight+regulation%2C+modulation%2C+and+high-level+expression+by+vectors+containing+the+arabinose+PBAD+promoter&amp;author=LM+Guzman&amp;author=D+Belin&amp;author=MJ+Carson&amp;author=J+Beckwith&amp;journal=J+Bacteriol&amp;volume=177&amp;doi=10.1128%2Fjb.177.14.4121-4130.1995&amp;pages=4121-4130&amp;publication_year=1995&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R23">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R23"></a><div class="citation"> <span class="reference-surname">Murphy</span> <span class="reference-given-names">FA</span>, <span class="reference-surname">Fauquet</span> <span class="reference-given-names">CM</span>, <span class="reference-surname">Bishop</span> <span class="reference-given-names">DHL</span>, <span class="reference-surname">Ghabrial</span> <span class="reference-given-names">SA</span>, <span class="reference-surname">Jarvis</span> <span class="reference-given-names">AW</span><span class="reference-etal"> et al.</span> <span class="reference-source">Virus Taxonomy: Classification and Nomenclature of Viruses: Sixth Report of the International Committee on Taxonomy of Viruses Vienna</span> Vienna: Springer-Verlag; <span class="reference-year">1995</span></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Virus+Taxonomy%3A+Classification+and+Nomenclature+of+Viruses%3A+Sixth+Report+of+the+International+Committee+on+Taxonomy+of+Viruses+Vienna&amp;author=FA+Murphy&amp;author=CM+Fauquet&amp;author=DHL+Bishop&amp;author=SA+Ghabrial&amp;author=AW+Jarvis&amp;publication_year=1995&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R24">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R24"></a><div class="citation"> <span class="reference-surname">Kazaks</span> <span class="reference-given-names">A</span>, <span class="reference-surname">Voronkova</span> <span class="reference-given-names">T</span>, <span class="reference-surname">Rumnieks</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Dishlers</span> <span class="reference-given-names">A</span>, <span class="reference-surname">Tars</span> <span class="reference-given-names">K</span>. <span class="reference-article-title"> Genome structure of caulobacter phage phiCb5. </span> <span class="reference-source">J Virol</span> <span class="reference-year">2011</span>; 85:<span class="reference-fpage">4628–</span><span class="reference-lpage">4631</span><a target="xrefwindow" href="https://doi.org/10.1128/JVI.02256-10"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/21325422">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Genome+structure+of+caulobacter+phage+phiCb5&amp;author=A+Kazaks&amp;author=T+Voronkova&amp;author=J+Rumnieks&amp;author=A+Dishlers&amp;author=K+Tars&amp;journal=J+Virol&amp;volume=85&amp;doi=10.1128%2FJVI.02256-10&amp;pages=4628-4631&amp;publication_year=2011&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R25">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R25"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">Roof</span> <span class="reference-given-names">WD</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> The <span class="reference-italic">Escherichia coli</span> FKBP-type PPIase SlyD is required for the stabilization of the E lysis protein of bacteriophage φX174. </span> <span class="reference-source">Mol Microbiol</span> <span class="reference-year">2002</span>; 45:<span class="reference-fpage">99–</span><span class="reference-lpage">108</span><a target="xrefwindow" href="https://doi.org/10.1046/j.1365-2958.2002.02984.x"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/12100551">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=The+Escherichia+coli+FKBP-type+PPIase+SlyD+is+required+for+the+stabilization+of+the+E+lysis+protein+of+bacteriophage+%CF%86X174&amp;author=TG+Bernhardt&amp;author=WD+Roof&amp;author=R+Young&amp;journal=Mol+Microbiol&amp;volume=45&amp;doi=10.1046%2Fj.1365-2958.2002.02984.x&amp;pages=99-108&amp;publication_year=2002&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R26">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R26"></a><div class="citation"> <span class="reference-surname">Tanaka</span> <span class="reference-given-names">S</span>, <span class="reference-surname">Clemons</span> <span class="reference-given-names">WM</span> <span class="reference-suffix">Jr</span>. <span class="reference-article-title"> Minimal requirements for inhibition of MraY by lysis protein E from bacteriophage φx174. </span> <span class="reference-source">Mol Microbiol</span> <span class="reference-year">2012</span>; 85:<span class="reference-fpage">975–</span><span class="reference-lpage">985</span><a target="xrefwindow" href="https://doi.org/10.1111/j.1365-2958.2012.08153.x"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/22742425">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Minimal+requirements+for+inhibition+of+MraY+by+lysis+protein+E+from+bacteriophage+%CF%86x174&amp;author=S+Tanaka&amp;author=WM+Clemons&amp;journal=Mol+Microbiol&amp;volume=85&amp;doi=10.1111%2Fj.1365-2958.2012.08153.x&amp;pages=975-985&amp;publication_year=2012&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R27">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R27"></a><div class="citation"> <span class="reference-surname">Drozdetskiy</span> <span class="reference-given-names">A</span>, <span class="reference-surname">Cole</span> <span class="reference-given-names">C</span>, <span class="reference-surname">Procter</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Barton</span> <span class="reference-given-names">GJ</span>. <span class="reference-article-title"> JPred4: a protein secondary structure prediction server. </span> <span class="reference-source">Nucleic Acids Res</span> <span class="reference-year">2015</span>; 43:<span class="reference-fpage">W389–</span><span class="reference-lpage">W394</span><a target="xrefwindow" href="https://doi.org/10.1093/nar/gkv332"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/25883141">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=JPred4%3A+a+protein+secondary+structure+prediction+server&amp;author=A+Drozdetskiy&amp;author=C+Cole&amp;author=J+Procter&amp;author=GJ+Barton&amp;journal=Nucleic+Acids+Res&amp;volume=43&amp;doi=10.1093%2Fnar%2Fgkv332&amp;pages=W389-W394&amp;publication_year=2015&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R28">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R28"></a><div class="citation"> <span class="reference-surname">Yohannan</span> <span class="reference-given-names">S</span>, <span class="reference-surname">Yang</span> <span class="reference-given-names">D</span>, <span class="reference-surname">Faham</span> <span class="reference-given-names">S</span>, <span class="reference-surname">Boulting</span> <span class="reference-given-names">G</span>, <span class="reference-surname">Whitelegge</span> <span class="reference-given-names">J</span><span class="reference-etal"> et al.</span> <span class="reference-article-title"> Proline substitutions are not easily accommodated in a membrane protein. </span> <span class="reference-source">J Mol Biol</span> <span class="reference-year">2004</span>; 341:<span class="reference-fpage">1–</span><span class="reference-lpage">6</span><a target="xrefwindow" href="https://doi.org/10.1016/j.jmb.2004.06.025"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/15312757">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Proline+substitutions+are+not+easily+accommodated+in+a+membrane+protein&amp;author=S+Yohannan&amp;author=D+Yang&amp;author=S+Faham&amp;author=G+Boulting&amp;author=J+Whitelegge&amp;journal=J+Mol+Biol&amp;volume=341&amp;doi=10.1016%2Fj.jmb.2004.06.025&amp;pages=1-6&amp;publication_year=2004&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R29">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R29"></a><div class="citation"> <span class="reference-surname">Kuk</span> <span class="reference-given-names">AC</span>, <span class="reference-surname">Mashalidis</span> <span class="reference-given-names">EH</span>, <span class="reference-surname">Lee</span> <span class="reference-given-names">SY</span>. <span class="reference-article-title"> Crystal structure of the MOP flippase MurJ in an inward-facing conformation. </span> <span class="reference-source">Nat Struct Mol Biol</span> <span class="reference-year">2017</span>; 24:<span class="reference-fpage">171–</span><span class="reference-lpage">176</span><a target="xrefwindow" href="https://doi.org/10.1038/nsmb.3346"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/28024149">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Crystal+structure+of+the+MOP+flippase+MurJ+in+an+inward-facing+conformation&amp;author=AC+Kuk&amp;author=EH+Mashalidis&amp;author=SY+Lee&amp;journal=Nat+Struct+Mol+Biol&amp;volume=24&amp;doi=10.1038%2Fnsmb.3346&amp;pages=171-176&amp;publication_year=2017&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R30">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R30"></a><div class="citation"> <span class="reference-surname">Ursell</span> <span class="reference-given-names">TS</span>, <span class="reference-surname">Nguyen</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Monds</span> <span class="reference-given-names">RD</span>, <span class="reference-surname">Colavin</span> <span class="reference-given-names">A</span>, <span class="reference-surname">Billings</span> <span class="reference-given-names">G</span><span class="reference-etal"> et al.</span> <span class="reference-article-title"> Rod-like bacterial shape is maintained by feedback between cell curvature and cytoskeletal localization. </span> <span class="reference-source">Proc Natl Acad Sci USA</span> <span class="reference-year">2014</span>; 111:<span class="reference-elocation-id">E1025</span><span class="reference-lpage">E1034</span><a target="xrefwindow" href="https://doi.org/10.1073/pnas.1317174111"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/24550515">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Rod-like+bacterial+shape+is+maintained+by+feedback+between+cell+curvature+and+cytoskeletal+localization&amp;author=TS+Ursell&amp;author=J+Nguyen&amp;author=RD+Monds&amp;author=A+Colavin&amp;author=G+Billings&amp;journal=Proc+Natl+Acad+Sci+USA&amp;volume=111&amp;doi=10.1073%2Fpnas.1317174111&amp;publication_year=2014&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R31">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R31"></a><div class="citation"> <span class="reference-surname">Morgenstein</span> <span class="reference-given-names">RM</span>, <span class="reference-surname">Bratton</span> <span class="reference-given-names">BP</span>, <span class="reference-surname">Nguyen</span> <span class="reference-given-names">JP</span>, <span class="reference-surname">Ouzounov</span> <span class="reference-given-names">N</span>, <span class="reference-surname">Shaevitz</span> <span class="reference-given-names">JW</span><span class="reference-etal"> et al.</span> <span class="reference-article-title"> RodZ links MreB to cell wall synthesis to mediate MreB rotation and robust morphogenesis. </span> <span class="reference-source">Proc Natl Acad Sci USA</span> <span class="reference-year">2015</span>; 112:<span class="reference-fpage">12510–</span><span class="reference-lpage">12515</span><a target="xrefwindow" href="https://doi.org/10.1073/pnas.1509610112"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/26396257">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=RodZ+links+MreB+to+cell+wall+synthesis+to+mediate+MreB+rotation+and+robust+morphogenesis&amp;author=RM+Morgenstein&amp;author=BP+Bratton&amp;author=JP+Nguyen&amp;author=N+Ouzounov&amp;author=JW+Shaevitz&amp;journal=Proc+Natl+Acad+Sci+USA&amp;volume=112&amp;doi=10.1073%2Fpnas.1509610112&amp;pages=12510-12515&amp;publication_year=2015&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R32">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R32"></a><div class="citation"> <span class="reference-surname">Holtje</span> <span class="reference-given-names">JV</span>, <span class="reference-surname">van Duin</span> <span class="reference-given-names">J</span>. <span class="reference-article-title"> MS2 phage induced lysis of <span class="reference-italic">E. coli</span> depends upon the activity of the bacterial autolysins. </span> In <span class="reference-surname">Nombela</span> <span class="reference-given-names">C</span>. (editor) <span class="reference-source">Microbial Cell Wall Synthesis and Autolysis</span> New York, NY: Elsevier Science Publishers; <span class="reference-year">1984</span> pp. <span class="reference-fpage">195–</span><span class="reference-lpage">199</span></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=MS2+phage+induced+lysis+of+E.+coli+depends+upon+the+activity+of+the+bacterial+autolysins&amp;author=JV+Holtje&amp;author=J+van+Duin&amp;journal=Microbial+Cell+Wall+Synthesis+and+Autolysis&amp;pages=195-199&amp;publication_year=1984&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R33">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R33"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>. <span class="reference-article-title"> Breaking free: small phages inhibit murein synthesis to lyse their host. </span> <span class="reference-source">Ph.D thesis</span> College Station, TX: Texas A&amp;M University; <span class="reference-year">2001</span></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Ph.D+thesis&amp;author=TG+Bernhardt&amp;publication_year=2001&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R34">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R34"></a><div class="citation"> <span class="reference-surname">Guyer</span> <span class="reference-given-names">MS</span>, <span class="reference-surname">Reed</span> <span class="reference-given-names">RR</span>, <span class="reference-surname">Steitz</span> <span class="reference-given-names">JA</span>, <span class="reference-surname">Low</span> <span class="reference-given-names">KB</span>. <span class="reference-article-title"> Identification of a sex-factor-affinity site in <span class="reference-italic">E. coli</span> as γδ. </span> <span class="reference-source">Cold Spring Harb Symp Quant Biol</span> <span class="reference-year">1981</span>; 45:<span class="reference-fpage">135–</span><span class="reference-lpage">140</span><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/6271456">[PubMed]</a> <a target="xrefwindow" href="https://dx.doi.org/10.1101/SQB.1981.045.01.022">[CrossRef]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Identification+of+a+sex-factor-affinity+site+in+E.+coli+as+%CE%B3%CE%B4&amp;author=MS+Guyer&amp;author=RR+Reed&amp;author=JA+Steitz&amp;author=KB+Low&amp;journal=Cold+Spring+Harb+Symp+Quant+Biol&amp;volume=45&amp;pages=135-140&amp;publication_year=1981&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R35">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R35"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">de Boer</span> <span class="reference-given-names">PA</span>. <span class="reference-article-title"> The <span class="reference-italic">Escherichia coli</span> amidase AmiC is a periplasmic septal ring component exported via the twin-arginine transport pathway. </span> <span class="reference-source">Mol Microbiol</span> <span class="reference-year">2003</span>; 48:<span class="reference-fpage">1171–</span><span class="reference-lpage">1182</span><a target="xrefwindow" href="https://doi.org/10.1046/j.1365-2958.2003.03511.x"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/12787347">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=The+Escherichia+coli+amidase+AmiC+is+a+periplasmic+septal+ring+component+exported+via+the+twin-arginine+transport+pathway&amp;author=TG+Bernhardt&amp;author=PA+de+Boer&amp;journal=Mol+Microbiol&amp;volume=48&amp;doi=10.1046%2Fj.1365-2958.2003.03511.x&amp;pages=1171-1182&amp;publication_year=2003&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="R36">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R36"></a><div class="citation"> <span class="reference-surname">McIntosh</span> <span class="reference-given-names">BK</span>. <span class="reference-article-title"> Bacteriophage MS2 L protein: genetic and biochemical charecterization. </span> <span class="reference-source">Ph.D thesis</span> College Station, TX: Texas A&amp;M University; <span class="reference-year">2008</span></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Ph.D+thesis&amp;author=BK+McIntosh&amp;publication_year=2008&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li></ol></span><div class="clearer">&nbsp;</div></div><div xmlns:f="http://www.example.org/functions" class="copyright-after-text">© 2017 The Authors</div>
</div>

<div id="article-level-0-figs-and-tables" class="bm">

</div>

<div id="article-level-0-end-metadata" class="fm">

</div></div></div>

</div>
<div id="referenceContainer" class="references hidden-js-div tabbedsection tab-pane">
<div id="referencedetails">
<p class="description"><span class="tl-main-part">References</span></p>
<ol id="articlereference">
<li class="refbody" id="ref-R1">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R1"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">Wang</span> <span class="reference-given-names">IN</span>, <span class="reference-surname">Struck</span> <span class="reference-given-names">DK</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Breaking free: "protein antibiotics" and phage lysis. </span> <span class="reference-source">Res Microbiol</span> <span class="reference-year">2002</span>; 153:<span class="reference-fpage">493–</span><span class="reference-lpage">501</span><a target="xrefwindow" href="https://doi.org/10.1016/S0923-2508(02)01330-X"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/12437210">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Breaking+free%3A+%22protein+antibiotics%22+and+phage+lysis&amp;author=TG+Bernhardt&amp;author=IN+Wang&amp;author=DK+Struck&amp;author=R+Young&amp;journal=Res+Microbiol&amp;volume=153&amp;doi=10.1016%2FS0923-2508%2802%2901330-X&amp;pages=493-501&amp;publication_year=2002&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R2">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R2"></a><div class="citation"> <span class="reference-surname">Sanger</span> <span class="reference-given-names">F</span>, <span class="reference-surname">Coulson</span> <span class="reference-given-names">AR</span>, <span class="reference-surname">Friedmann</span> <span class="reference-given-names">T</span>, <span class="reference-surname">Air</span> <span class="reference-given-names">GM</span>, <span class="reference-surname">Barrell</span> <span class="reference-given-names">BG</span><span class="reference-etal"> et al.</span> <span class="reference-article-title"> The nucleotide sequence of bacteriophage φX174. </span> <span class="reference-source">J Mol Biol</span> <span class="reference-year">1978</span>; 125:<span class="reference-fpage">225–</span><span class="reference-lpage">246</span><a target="xrefwindow" href="https://doi.org/10.1016/0022-2836(78)90346-7"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/731693">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=The+nucleotide+sequence+of+bacteriophage+%CF%86X174&amp;author=F+Sanger&amp;author=AR+Coulson&amp;author=T+Friedmann&amp;author=GM+Air&amp;author=BG+Barrell&amp;journal=J+Mol+Biol&amp;volume=125&amp;doi=10.1016%2F0022-2836%2878%2990346-7&amp;pages=225-246&amp;publication_year=1978&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R3">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R3"></a><div class="citation"> <span class="reference-surname">Winter</span> <span class="reference-given-names">RB</span>, <span class="reference-surname">Gold</span> <span class="reference-given-names">L</span>. <span class="reference-article-title"> Overproduction of bacteriophage Qβ maturation (A<sub>2</sub>) protein leads to cell lysis. </span> <span class="reference-source">Cell</span> <span class="reference-year">1983</span>; 33:<span class="reference-fpage">877–</span><span class="reference-lpage">885</span><a target="xrefwindow" href="https://doi.org/10.1016/0092-8674(83)90030-2"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/6871998">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Overproduction+of+bacteriophage+Q%CE%B2+maturation+%28A2%29+protein+leads+to+cell+lysis&amp;author=RB+Winter&amp;author=L+Gold&amp;journal=Cell&amp;volume=33&amp;doi=10.1016%2F0092-8674%2883%2990030-2&amp;pages=877-885&amp;publication_year=1983&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R4">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R4"></a><div class="citation"> <span class="reference-surname">Lubitz</span> <span class="reference-given-names">W</span>, <span class="reference-surname">Halfmann</span> <span class="reference-given-names">G</span>, <span class="reference-surname">Plapp</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Lysis of <span class="reference-italic">Escherichia coli</span> after infection with φX174 depends on the regulation of the cellular autolytic system. </span> <span class="reference-source">J Gen Microbiol</span> <span class="reference-year">1984</span>; 130:<span class="reference-fpage">1079–</span><span class="reference-lpage">1087</span><a target="xrefwindow" href="https://doi.org/10.1099/00221287-130-5-1079"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/6236279">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Lysis+of+Escherichia+coli+after+infection+with+%CF%86X174+depends+on+the+regulation+of+the+cellular+autolytic+system&amp;author=W+Lubitz&amp;author=G+Halfmann&amp;author=R+Plapp&amp;journal=J+Gen+Microbiol&amp;volume=130&amp;doi=10.1099%2F00221287-130-5-1079&amp;pages=1079-1087&amp;publication_year=1984&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R5">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R5"></a><div class="citation"> <span class="reference-surname">Witte</span> <span class="reference-given-names">A</span>, <span class="reference-surname">Wanner</span> <span class="reference-given-names">G</span>, <span class="reference-surname">Bläsi</span> <span class="reference-given-names">U</span>, <span class="reference-surname">Halfmann</span> <span class="reference-given-names">G</span>, <span class="reference-surname">Szostak</span> <span class="reference-given-names">M</span><span class="reference-etal"> et al.</span> <span class="reference-article-title"> Endogenous transmembrane tunnel formation mediated by φX174 lysis protein E. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">1990</span>; 172:<span class="reference-fpage">4109–</span><span class="reference-lpage">4114</span><a target="xrefwindow" href="https://doi.org/10.1128/jb.172.7.4109-4114.1990"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/2141836">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Endogenous+transmembrane+tunnel+formation+mediated+by+%CF%86X174+lysis+protein+E&amp;author=A+Witte&amp;author=G+Wanner&amp;author=U+Bl%C3%A4si&amp;author=G+Halfmann&amp;author=M+Szostak&amp;journal=J+Bacteriol&amp;volume=172&amp;doi=10.1128%2Fjb.172.7.4109-4114.1990&amp;pages=4109-4114&amp;publication_year=1990&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R6">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R6"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">Roof</span> <span class="reference-given-names">WD</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Genetic evidence that the bacteriophage φX174 lysis protein inhibits cell wall synthesis. </span> <span class="reference-source">Proc Natl Acad Sci USA</span> <span class="reference-year">2000</span>; 97:<span class="reference-fpage">4297–</span><span class="reference-lpage">4302</span><a target="xrefwindow" href="https://doi.org/10.1073/pnas.97.8.4297"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/10760296">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Genetic+evidence+that+the+bacteriophage+%CF%86X174+lysis+protein+inhibits+cell+wall+synthesis&amp;author=TG+Bernhardt&amp;author=WD+Roof&amp;author=R+Young&amp;journal=Proc+Natl+Acad+Sci+USA&amp;volume=97&amp;doi=10.1073%2Fpnas.97.8.4297&amp;pages=4297-4302&amp;publication_year=2000&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R7">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R7"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">Wang</span> <span class="reference-given-names">IN</span>, <span class="reference-surname">Struck</span> <span class="reference-given-names">DK</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> A protein antibiotic in the phage Qβ virion: diversity in lysis targets. </span> <span class="reference-source">Science</span> <span class="reference-year">2001</span>; 292:<span class="reference-fpage">2326–</span><span class="reference-lpage">2329</span><a target="xrefwindow" href="https://doi.org/10.1126/science.1058289"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/11423662">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=A+protein+antibiotic+in+the+phage+Q%CE%B2+virion%3A+diversity+in+lysis+targets&amp;author=TG+Bernhardt&amp;author=IN+Wang&amp;author=DK+Struck&amp;author=R+Young&amp;journal=Science&amp;volume=292&amp;doi=10.1126%2Fscience.1058289&amp;pages=2326-2329&amp;publication_year=2001&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R8">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R8"></a><div class="citation"> <span class="reference-surname">Zheng</span> <span class="reference-given-names">Y</span>, <span class="reference-surname">Struck</span> <span class="reference-given-names">DK</span>, <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Genetic analysis of MraY inhibition by the φX174 protein E. </span> <span class="reference-source">Genetics</span> <span class="reference-year">2008</span>; 180:<span class="reference-fpage">1459–</span><span class="reference-lpage">1466</span><a target="xrefwindow" href="https://doi.org/10.1534/genetics.108.093443"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/18791230">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Genetic+analysis+of+MraY+inhibition+by+the+%CF%86X174+protein+E&amp;author=Y+Zheng&amp;author=DK+Struck&amp;author=TG+Bernhardt&amp;author=R+Young&amp;journal=Genetics&amp;volume=180&amp;doi=10.1534%2Fgenetics.108.093443&amp;pages=1459-1466&amp;publication_year=2008&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R9">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R9"></a><div class="citation"> <span class="reference-surname">Zheng</span> <span class="reference-given-names">Y</span>, <span class="reference-surname">Struck</span> <span class="reference-given-names">DK</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Purification and functional characterization of φX174 lysis protein E. </span> <span class="reference-source">Biochemistry</span> <span class="reference-year">2009</span>; 48:<span class="reference-fpage">4999–</span><span class="reference-lpage">5006</span><a target="xrefwindow" href="https://doi.org/10.1021/bi900469g"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/19379010">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Purification+and+functional+characterization+of+%CF%86X174+lysis+protein+E&amp;author=Y+Zheng&amp;author=DK+Struck&amp;author=R+Young&amp;journal=Biochemistry&amp;volume=48&amp;doi=10.1021%2Fbi900469g&amp;pages=4999-5006&amp;publication_year=2009&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R10">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R10"></a><div class="citation"> <span class="reference-surname">Rumnieks</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Tars</span> <span class="reference-given-names">K</span>. <span class="reference-article-title"> Diversity of pili-specific bacteriophages: genome sequence of IncM plasmid-dependent RNA phage M. </span> <span class="reference-source">BMC Microbiol</span> <span class="reference-year">2012</span>; 12:<span class="reference-fpage">277</span><a target="xrefwindow" href="https://doi.org/10.1186/1471-2180-12-277"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/23176223">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Diversity+of+pili-specific+bacteriophages%3A+genome+sequence+of+IncM+plasmid-dependent+RNA+phage+M&amp;author=J+Rumnieks&amp;author=K+Tars&amp;journal=BMC+Microbiol&amp;volume=12&amp;doi=10.1186%2F1471-2180-12-277&amp;pages=277&amp;publication_year=2012&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R11">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R11"></a><div class="citation"> <span class="reference-surname">Beremand</span> <span class="reference-given-names">MN</span>, <span class="reference-surname">Blumenthal</span> <span class="reference-given-names">T</span>. <span class="reference-article-title"> Overlapping genes in RNA phage: a new protein implicated in lysis. </span> <span class="reference-source">Cell</span> <span class="reference-year">1979</span>; 18:<span class="reference-fpage">257–</span><span class="reference-lpage">266</span><a target="xrefwindow" href="https://doi.org/10.1016/0092-8674(79)90045-X"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/387256">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Overlapping+genes+in+RNA+phage%3A+a+new+protein+implicated+in+lysis&amp;author=MN+Beremand&amp;author=T+Blumenthal&amp;journal=Cell&amp;volume=18&amp;doi=10.1016%2F0092-8674%2879%2990045-X&amp;pages=257-266&amp;publication_year=1979&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R12">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R12"></a><div class="citation"> <span class="reference-surname">Berkhout</span> <span class="reference-given-names">B</span>, <span class="reference-surname">de Smit</span> <span class="reference-given-names">MH</span>, <span class="reference-surname">Spanjaard</span> <span class="reference-given-names">RA</span>, <span class="reference-surname">Blom</span> <span class="reference-given-names">T</span>, <span class="reference-surname">van Duin</span> <span class="reference-given-names">J</span>. <span class="reference-article-title"> The amino terminal half of the MS2-coded lysis protein is dispensable for function: implications for our understanding of coding region overlaps. </span> <span class="reference-source">EMBO J</span> <span class="reference-year">1985</span>; 4:<span class="reference-fpage">3315–</span><span class="reference-lpage">3320</span><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/3912168">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=The+amino+terminal+half+of+the+MS2-coded+lysis+protein+is+dispensable+for+function%3A+implications+for+our+understanding+of+coding+region+overlaps&amp;author=B+Berkhout&amp;author=MH+de+Smit&amp;author=RA+Spanjaard&amp;author=T+Blom&amp;author=J+van+Duin&amp;journal=EMBO+J&amp;volume=4&amp;pages=3315-3320&amp;publication_year=1985&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R13">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R13"></a><div class="citation"> <span class="reference-surname">Goessens</span> <span class="reference-given-names">WH</span>, <span class="reference-surname">Driessen</span> <span class="reference-given-names">AJ</span>, <span class="reference-surname">Wilschut</span> <span class="reference-given-names">J</span>, <span class="reference-surname">van Duin</span> <span class="reference-given-names">J</span>. <span class="reference-article-title"> A synthetic peptide corresponding to the C-terminal 25 residues of phage MS2 coded lysis protein dissipates the protonmotive force in <span class="reference-italic">Escherichia coli</span> membrane vesicles by generating hydrophilic pores. </span> <span class="reference-source">Embo J</span> <span class="reference-year">1988</span>; 7:<span class="reference-fpage">867–</span><span class="reference-lpage">873</span><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/2840287">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=A+synthetic+peptide+corresponding+to+the+C-terminal+25+residues+of+phage+MS2+coded+lysis+protein+dissipates+the+protonmotive+force+in+Escherichia+coli+membrane+vesicles+by+generating+hydrophilic+pores&amp;author=WH+Goessens&amp;author=AJ+Driessen&amp;author=J+Wilschut&amp;author=J+van+Duin&amp;journal=Embo+J&amp;volume=7&amp;pages=867-873&amp;publication_year=1988&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R14">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R14"></a><div class="citation"> <span class="reference-surname">Walderich</span> <span class="reference-given-names">B</span>, <span class="reference-surname">Höltje</span> <span class="reference-given-names">JV</span>. <span class="reference-article-title"> Specific localization of the lysis protein of bacteriophage MS2 in membrane adhesion sites of <span class="reference-italic">Escherichia coli</span>. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">1989</span>; 171:<span class="reference-fpage">3331–</span><span class="reference-lpage">3336</span><a target="xrefwindow" href="https://doi.org/10.1128/jb.171.6.3331-3336.1989"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/2656650">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Specific+localization+of+the+lysis+protein+of+bacteriophage+MS2+in+membrane+adhesion+sites+of+Escherichia+coli&amp;author=B+Walderich&amp;author=JV+H%C3%B6ltje&amp;journal=J+Bacteriol&amp;volume=171&amp;doi=10.1128%2Fjb.171.6.3331-3336.1989&amp;pages=3331-3336&amp;publication_year=1989&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R15">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R15"></a><div class="citation"> <span class="reference-surname">Walderich</span> <span class="reference-given-names">B</span>, <span class="reference-surname">Ursinus-Wössner</span> <span class="reference-given-names">A</span>, <span class="reference-surname">van Duin</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Höltje</span> <span class="reference-given-names">JV</span>. <span class="reference-article-title"> Induction of the autolytic system of <span class="reference-italic">Escherichia coli</span> by specific insertion of bacteriophage MS2 lysis protein into the bacterial cell envelope. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">1988</span>; 170:<span class="reference-fpage">5027–</span><span class="reference-lpage">5033</span><a target="xrefwindow" href="https://doi.org/10.1128/jb.170.11.5027-5033.1988"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/3053640">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Induction+of+the+autolytic+system+of+Escherichia+coli+by+specific+insertion+of+bacteriophage+MS2+lysis+protein+into+the+bacterial+cell+envelope&amp;author=B+Walderich&amp;author=A+Ursinus-W%C3%B6ssner&amp;author=J+van+Duin&amp;author=JV+H%C3%B6ltje&amp;journal=J+Bacteriol&amp;volume=170&amp;doi=10.1128%2Fjb.170.11.5027-5033.1988&amp;pages=5027-5033&amp;publication_year=1988&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R16">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R16"></a><div class="citation"> <span class="reference-surname">Chamakura</span> <span class="reference-given-names">KR</span>, <span class="reference-surname">Tran</span> <span class="reference-given-names">JS</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> MS2 lysis of <span class="reference-italic">Escherichia coli</span> depends on host chaperone DnaJ. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">2017</span>; 199:<span class="reference-elocation-id">e00058-17</span><a target="xrefwindow" href="https://doi.org/10.1128/JB.00058-17"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/28396351">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=MS2+lysis+of+Escherichia+coli+depends+on+host+chaperone+DnaJ&amp;author=KR+Chamakura&amp;author=JS+Tran&amp;author=R+Young&amp;journal=J+Bacteriol&amp;volume=199&amp;doi=10.1128%2FJB.00058-17&amp;publication_year=2017&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R17">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R17"></a><div class="citation"> <span class="reference-surname">Smith</span> <span class="reference-given-names">DL</span>, <span class="reference-surname">Chang</span> <span class="reference-given-names">CY</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> The λ holin accumulates beyond the lethal triggering concentration under hyperexpression conditions. </span> <span class="reference-source">Gene Expr</span> <span class="reference-year">1998</span>; 7:<span class="reference-fpage">39–</span><span class="reference-lpage">52</span><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/9572396">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=The+%CE%BB+holin+accumulates+beyond+the+lethal+triggering+concentration+under+hyperexpression+conditions&amp;author=DL+Smith&amp;author=CY+Chang&amp;author=R+Young&amp;journal=Gene+Expr&amp;volume=7&amp;pages=39-52&amp;publication_year=1998&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R18">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R18"></a><div class="citation"> <span class="reference-surname">Moussa</span> <span class="reference-given-names">SH</span>, <span class="reference-surname">Kuznetsov</span> <span class="reference-given-names">V</span>, <span class="reference-surname">Tran</span> <span class="reference-given-names">TA</span>, <span class="reference-surname">Sacchettini</span> <span class="reference-given-names">JC</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> Protein determinants of phage T4 lysis inhibition. </span> <span class="reference-source">Protein Sci</span> <span class="reference-year">2012</span>; 21:<span class="reference-fpage">571–</span><span class="reference-lpage">582</span><a target="xrefwindow" href="https://doi.org/10.1002/pro.2042"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/22389108">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Protein+determinants+of+phage+T4+lysis+inhibition&amp;author=SH+Moussa&amp;author=V+Kuznetsov&amp;author=TA+Tran&amp;author=JC+Sacchettini&amp;author=R+Young&amp;journal=Protein+Sci&amp;volume=21&amp;doi=10.1002%2Fpro.2042&amp;pages=571-582&amp;publication_year=2012&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R19">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R19"></a><div class="citation"> <span class="reference-surname">Ruokoranta</span> <span class="reference-given-names">TM</span>, <span class="reference-surname">Grahn</span> <span class="reference-given-names">AM</span>, <span class="reference-surname">Ravantti</span> <span class="reference-given-names">JJ</span>, <span class="reference-surname">Poranen</span> <span class="reference-given-names">MM</span>, <span class="reference-surname">Bamford</span> <span class="reference-given-names">DH</span>. <span class="reference-article-title"> Complete genome sequence of the broad host range single-stranded RNA phage PRR1 places it in the <span class="reference-italic">Levivirus</span> genus with characteristics shared with Alloleviviruses. </span> <span class="reference-source">J Virol</span> <span class="reference-year">2006</span>; 80:<span class="reference-fpage">9326–</span><span class="reference-lpage">9330</span><a target="xrefwindow" href="https://doi.org/10.1128/JVI.01005-06"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/16940544">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Complete+genome+sequence+of+the+broad+host+range+single-stranded+RNA+phage+PRR1+places+it+in+the+Levivirus+genus+with+characteristics+shared+with+Alloleviviruses&amp;author=TM+Ruokoranta&amp;author=AM+Grahn&amp;author=JJ+Ravantti&amp;author=MM+Poranen&amp;author=DH+Bamford&amp;journal=J+Virol&amp;volume=80&amp;doi=10.1128%2FJVI.01005-06&amp;pages=9326-9330&amp;publication_year=2006&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R20">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R20"></a><div class="citation"> <span class="reference-surname">Kannoly</span> <span class="reference-given-names">S</span>, <span class="reference-surname">Shao</span> <span class="reference-given-names">Y</span>, <span class="reference-surname">Wang</span> <span class="reference-given-names">IN</span>. <span class="reference-article-title"> Rethinking the evolution of single-stranded RNA (ssRNA) bacteriophages based on genomic sequences and characterizations of two R-plasmid-dependent ssRNA phages, C-1 and Hgal1. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">2012</span>; 194:<span class="reference-fpage">5073–</span><span class="reference-lpage">5079</span><a target="xrefwindow" href="https://doi.org/10.1128/JB.00929-12"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/22821966">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Rethinking+the+evolution+of+single-stranded+RNA+%28ssRNA%29+bacteriophages+based+on+genomic+sequences+and+characterizations+of+two+R-plasmid-dependent+ssRNA+phages%2C+C-1+and+Hgal1&amp;author=S+Kannoly&amp;author=Y+Shao&amp;author=IN+Wang&amp;journal=J+Bacteriol&amp;volume=194&amp;doi=10.1128%2FJB.00929-12&amp;pages=5073-5079&amp;publication_year=2012&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R21">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R21"></a><div class="citation"> <span class="reference-surname">Klovins</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Overbeek</span> <span class="reference-given-names">GP</span>, <span class="reference-surname">van den Worm</span> <span class="reference-given-names">SH</span>, <span class="reference-surname">Ackermann</span> <span class="reference-given-names">HW</span>, <span class="reference-surname">van Duin</span> <span class="reference-given-names">J</span>. <span class="reference-article-title"> Nucleotide sequence of a ssRNA phage from <span class="reference-italic">Acinetobacter</span>: kinship to coliphages. </span> <span class="reference-source">J Gen Virol</span> <span class="reference-year">2002</span>; 83:<span class="reference-fpage">1523–</span><span class="reference-lpage">1533</span><a target="xrefwindow" href="https://doi.org/10.1099/0022-1317-83-6-1523"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/12029168">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Nucleotide+sequence+of+a+ssRNA+phage+from+Acinetobacter%3A+kinship+to+coliphages&amp;author=J+Klovins&amp;author=GP+Overbeek&amp;author=SH+van+den+Worm&amp;author=HW+Ackermann&amp;author=J+van+Duin&amp;journal=J+Gen+Virol&amp;volume=83&amp;doi=10.1099%2F0022-1317-83-6-1523&amp;pages=1523-1533&amp;publication_year=2002&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R22">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R22"></a><div class="citation"> <span class="reference-surname">Guzman</span> <span class="reference-given-names">LM</span>, <span class="reference-surname">Belin</span> <span class="reference-given-names">D</span>, <span class="reference-surname">Carson</span> <span class="reference-given-names">MJ</span>, <span class="reference-surname">Beckwith</span> <span class="reference-given-names">J</span>. <span class="reference-article-title"> Tight regulation, modulation, and high-level expression by vectors containing the arabinose P<sub>BAD</sub> promoter. </span> <span class="reference-source">J Bacteriol</span> <span class="reference-year">1995</span>; 177:<span class="reference-fpage">4121–</span><span class="reference-lpage">4130</span><a target="xrefwindow" href="https://doi.org/10.1128/jb.177.14.4121-4130.1995"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/7608087">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Tight+regulation%2C+modulation%2C+and+high-level+expression+by+vectors+containing+the+arabinose+PBAD+promoter&amp;author=LM+Guzman&amp;author=D+Belin&amp;author=MJ+Carson&amp;author=J+Beckwith&amp;journal=J+Bacteriol&amp;volume=177&amp;doi=10.1128%2Fjb.177.14.4121-4130.1995&amp;pages=4121-4130&amp;publication_year=1995&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R23">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R23"></a><div class="citation"> <span class="reference-surname">Murphy</span> <span class="reference-given-names">FA</span>, <span class="reference-surname">Fauquet</span> <span class="reference-given-names">CM</span>, <span class="reference-surname">Bishop</span> <span class="reference-given-names">DHL</span>, <span class="reference-surname">Ghabrial</span> <span class="reference-given-names">SA</span>, <span class="reference-surname">Jarvis</span> <span class="reference-given-names">AW</span><span class="reference-etal"> et al.</span> <span class="reference-source">Virus Taxonomy: Classification and Nomenclature of Viruses: Sixth Report of the International Committee on Taxonomy of Viruses Vienna</span> Vienna: Springer-Verlag; <span class="reference-year">1995</span></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Virus+Taxonomy%3A+Classification+and+Nomenclature+of+Viruses%3A+Sixth+Report+of+the+International+Committee+on+Taxonomy+of+Viruses+Vienna&amp;author=FA+Murphy&amp;author=CM+Fauquet&amp;author=DHL+Bishop&amp;author=SA+Ghabrial&amp;author=AW+Jarvis&amp;publication_year=1995&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R24">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R24"></a><div class="citation"> <span class="reference-surname">Kazaks</span> <span class="reference-given-names">A</span>, <span class="reference-surname">Voronkova</span> <span class="reference-given-names">T</span>, <span class="reference-surname">Rumnieks</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Dishlers</span> <span class="reference-given-names">A</span>, <span class="reference-surname">Tars</span> <span class="reference-given-names">K</span>. <span class="reference-article-title"> Genome structure of caulobacter phage phiCb5. </span> <span class="reference-source">J Virol</span> <span class="reference-year">2011</span>; 85:<span class="reference-fpage">4628–</span><span class="reference-lpage">4631</span><a target="xrefwindow" href="https://doi.org/10.1128/JVI.02256-10"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/21325422">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Genome+structure+of+caulobacter+phage+phiCb5&amp;author=A+Kazaks&amp;author=T+Voronkova&amp;author=J+Rumnieks&amp;author=A+Dishlers&amp;author=K+Tars&amp;journal=J+Virol&amp;volume=85&amp;doi=10.1128%2FJVI.02256-10&amp;pages=4628-4631&amp;publication_year=2011&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R25">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R25"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">Roof</span> <span class="reference-given-names">WD</span>, <span class="reference-surname">Young</span> <span class="reference-given-names">R</span>. <span class="reference-article-title"> The <span class="reference-italic">Escherichia coli</span> FKBP-type PPIase SlyD is required for the stabilization of the E lysis protein of bacteriophage φX174. </span> <span class="reference-source">Mol Microbiol</span> <span class="reference-year">2002</span>; 45:<span class="reference-fpage">99–</span><span class="reference-lpage">108</span><a target="xrefwindow" href="https://doi.org/10.1046/j.1365-2958.2002.02984.x"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/12100551">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=The+Escherichia+coli+FKBP-type+PPIase+SlyD+is+required+for+the+stabilization+of+the+E+lysis+protein+of+bacteriophage+%CF%86X174&amp;author=TG+Bernhardt&amp;author=WD+Roof&amp;author=R+Young&amp;journal=Mol+Microbiol&amp;volume=45&amp;doi=10.1046%2Fj.1365-2958.2002.02984.x&amp;pages=99-108&amp;publication_year=2002&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R26">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R26"></a><div class="citation"> <span class="reference-surname">Tanaka</span> <span class="reference-given-names">S</span>, <span class="reference-surname">Clemons</span> <span class="reference-given-names">WM</span> <span class="reference-suffix">Jr</span>. <span class="reference-article-title"> Minimal requirements for inhibition of MraY by lysis protein E from bacteriophage φx174. </span> <span class="reference-source">Mol Microbiol</span> <span class="reference-year">2012</span>; 85:<span class="reference-fpage">975–</span><span class="reference-lpage">985</span><a target="xrefwindow" href="https://doi.org/10.1111/j.1365-2958.2012.08153.x"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/22742425">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Minimal+requirements+for+inhibition+of+MraY+by+lysis+protein+E+from+bacteriophage+%CF%86x174&amp;author=S+Tanaka&amp;author=WM+Clemons&amp;journal=Mol+Microbiol&amp;volume=85&amp;doi=10.1111%2Fj.1365-2958.2012.08153.x&amp;pages=975-985&amp;publication_year=2012&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R27">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R27"></a><div class="citation"> <span class="reference-surname">Drozdetskiy</span> <span class="reference-given-names">A</span>, <span class="reference-surname">Cole</span> <span class="reference-given-names">C</span>, <span class="reference-surname">Procter</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Barton</span> <span class="reference-given-names">GJ</span>. <span class="reference-article-title"> JPred4: a protein secondary structure prediction server. </span> <span class="reference-source">Nucleic Acids Res</span> <span class="reference-year">2015</span>; 43:<span class="reference-fpage">W389–</span><span class="reference-lpage">W394</span><a target="xrefwindow" href="https://doi.org/10.1093/nar/gkv332"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/25883141">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=JPred4%3A+a+protein+secondary+structure+prediction+server&amp;author=A+Drozdetskiy&amp;author=C+Cole&amp;author=J+Procter&amp;author=GJ+Barton&amp;journal=Nucleic+Acids+Res&amp;volume=43&amp;doi=10.1093%2Fnar%2Fgkv332&amp;pages=W389-W394&amp;publication_year=2015&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R28">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R28"></a><div class="citation"> <span class="reference-surname">Yohannan</span> <span class="reference-given-names">S</span>, <span class="reference-surname">Yang</span> <span class="reference-given-names">D</span>, <span class="reference-surname">Faham</span> <span class="reference-given-names">S</span>, <span class="reference-surname">Boulting</span> <span class="reference-given-names">G</span>, <span class="reference-surname">Whitelegge</span> <span class="reference-given-names">J</span><span class="reference-etal"> et al.</span> <span class="reference-article-title"> Proline substitutions are not easily accommodated in a membrane protein. </span> <span class="reference-source">J Mol Biol</span> <span class="reference-year">2004</span>; 341:<span class="reference-fpage">1–</span><span class="reference-lpage">6</span><a target="xrefwindow" href="https://doi.org/10.1016/j.jmb.2004.06.025"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/15312757">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Proline+substitutions+are+not+easily+accommodated+in+a+membrane+protein&amp;author=S+Yohannan&amp;author=D+Yang&amp;author=S+Faham&amp;author=G+Boulting&amp;author=J+Whitelegge&amp;journal=J+Mol+Biol&amp;volume=341&amp;doi=10.1016%2Fj.jmb.2004.06.025&amp;pages=1-6&amp;publication_year=2004&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R29">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R29"></a><div class="citation"> <span class="reference-surname">Kuk</span> <span class="reference-given-names">AC</span>, <span class="reference-surname">Mashalidis</span> <span class="reference-given-names">EH</span>, <span class="reference-surname">Lee</span> <span class="reference-given-names">SY</span>. <span class="reference-article-title"> Crystal structure of the MOP flippase MurJ in an inward-facing conformation. </span> <span class="reference-source">Nat Struct Mol Biol</span> <span class="reference-year">2017</span>; 24:<span class="reference-fpage">171–</span><span class="reference-lpage">176</span><a target="xrefwindow" href="https://doi.org/10.1038/nsmb.3346"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/28024149">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Crystal+structure+of+the+MOP+flippase+MurJ+in+an+inward-facing+conformation&amp;author=AC+Kuk&amp;author=EH+Mashalidis&amp;author=SY+Lee&amp;journal=Nat+Struct+Mol+Biol&amp;volume=24&amp;doi=10.1038%2Fnsmb.3346&amp;pages=171-176&amp;publication_year=2017&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R30">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R30"></a><div class="citation"> <span class="reference-surname">Ursell</span> <span class="reference-given-names">TS</span>, <span class="reference-surname">Nguyen</span> <span class="reference-given-names">J</span>, <span class="reference-surname">Monds</span> <span class="reference-given-names">RD</span>, <span class="reference-surname">Colavin</span> <span class="reference-given-names">A</span>, <span class="reference-surname">Billings</span> <span class="reference-given-names">G</span><span class="reference-etal"> et al.</span> <span class="reference-article-title"> Rod-like bacterial shape is maintained by feedback between cell curvature and cytoskeletal localization. </span> <span class="reference-source">Proc Natl Acad Sci USA</span> <span class="reference-year">2014</span>; 111:<span class="reference-elocation-id">E1025</span><span class="reference-lpage">E1034</span><a target="xrefwindow" href="https://doi.org/10.1073/pnas.1317174111"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/24550515">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Rod-like+bacterial+shape+is+maintained+by+feedback+between+cell+curvature+and+cytoskeletal+localization&amp;author=TS+Ursell&amp;author=J+Nguyen&amp;author=RD+Monds&amp;author=A+Colavin&amp;author=G+Billings&amp;journal=Proc+Natl+Acad+Sci+USA&amp;volume=111&amp;doi=10.1073%2Fpnas.1317174111&amp;publication_year=2014&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R31">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R31"></a><div class="citation"> <span class="reference-surname">Morgenstein</span> <span class="reference-given-names">RM</span>, <span class="reference-surname">Bratton</span> <span class="reference-given-names">BP</span>, <span class="reference-surname">Nguyen</span> <span class="reference-given-names">JP</span>, <span class="reference-surname">Ouzounov</span> <span class="reference-given-names">N</span>, <span class="reference-surname">Shaevitz</span> <span class="reference-given-names">JW</span><span class="reference-etal"> et al.</span> <span class="reference-article-title"> RodZ links MreB to cell wall synthesis to mediate MreB rotation and robust morphogenesis. </span> <span class="reference-source">Proc Natl Acad Sci USA</span> <span class="reference-year">2015</span>; 112:<span class="reference-fpage">12510–</span><span class="reference-lpage">12515</span><a target="xrefwindow" href="https://doi.org/10.1073/pnas.1509610112"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/26396257">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=RodZ+links+MreB+to+cell+wall+synthesis+to+mediate+MreB+rotation+and+robust+morphogenesis&amp;author=RM+Morgenstein&amp;author=BP+Bratton&amp;author=JP+Nguyen&amp;author=N+Ouzounov&amp;author=JW+Shaevitz&amp;journal=Proc+Natl+Acad+Sci+USA&amp;volume=112&amp;doi=10.1073%2Fpnas.1509610112&amp;pages=12510-12515&amp;publication_year=2015&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R32">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R32"></a><div class="citation"> <span class="reference-surname">Holtje</span> <span class="reference-given-names">JV</span>, <span class="reference-surname">van Duin</span> <span class="reference-given-names">J</span>. <span class="reference-article-title"> MS2 phage induced lysis of <span class="reference-italic">E. coli</span> depends upon the activity of the bacterial autolysins. </span> In <span class="reference-surname">Nombela</span> <span class="reference-given-names">C</span>. (editor) <span class="reference-source">Microbial Cell Wall Synthesis and Autolysis</span> New York, NY: Elsevier Science Publishers; <span class="reference-year">1984</span> pp. <span class="reference-fpage">195–</span><span class="reference-lpage">199</span></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=MS2+phage+induced+lysis+of+E.+coli+depends+upon+the+activity+of+the+bacterial+autolysins&amp;author=JV+Holtje&amp;author=J+van+Duin&amp;journal=Microbial+Cell+Wall+Synthesis+and+Autolysis&amp;pages=195-199&amp;publication_year=1984&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R33">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R33"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>. <span class="reference-article-title"> Breaking free: small phages inhibit murein synthesis to lyse their host. </span> <span class="reference-source">Ph.D thesis</span> College Station, TX: Texas A&amp;M University; <span class="reference-year">2001</span></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Ph.D+thesis&amp;author=TG+Bernhardt&amp;publication_year=2001&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R34">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R34"></a><div class="citation"> <span class="reference-surname">Guyer</span> <span class="reference-given-names">MS</span>, <span class="reference-surname">Reed</span> <span class="reference-given-names">RR</span>, <span class="reference-surname">Steitz</span> <span class="reference-given-names">JA</span>, <span class="reference-surname">Low</span> <span class="reference-given-names">KB</span>. <span class="reference-article-title"> Identification of a sex-factor-affinity site in <span class="reference-italic">E. coli</span> as γδ. </span> <span class="reference-source">Cold Spring Harb Symp Quant Biol</span> <span class="reference-year">1981</span>; 45:<span class="reference-fpage">135–</span><span class="reference-lpage">140</span><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/6271456">[PubMed]</a> <a target="xrefwindow" href="https://dx.doi.org/10.1101/SQB.1981.045.01.022">[CrossRef]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Identification+of+a+sex-factor-affinity+site+in+E.+coli+as+%CE%B3%CE%B4&amp;author=MS+Guyer&amp;author=RR+Reed&amp;author=JA+Steitz&amp;author=KB+Low&amp;journal=Cold+Spring+Harb+Symp+Quant+Biol&amp;volume=45&amp;pages=135-140&amp;publication_year=1981&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R35">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R35"></a><div class="citation"> <span class="reference-surname">Bernhardt</span> <span class="reference-given-names">TG</span>, <span class="reference-surname">de Boer</span> <span class="reference-given-names">PA</span>. <span class="reference-article-title"> The <span class="reference-italic">Escherichia coli</span> amidase AmiC is a periplasmic septal ring component exported via the twin-arginine transport pathway. </span> <span class="reference-source">Mol Microbiol</span> <span class="reference-year">2003</span>; 48:<span class="reference-fpage">1171–</span><span class="reference-lpage">1182</span><a target="xrefwindow" href="https://doi.org/10.1046/j.1365-2958.2003.03511.x"> [View Article]</a><a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/pubmed/12787347">[PubMed]</a></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=The+Escherichia+coli+amidase+AmiC+is+a+periplasmic+septal+ring+component+exported+via+the+twin-arginine+transport+pathway&amp;author=TG+Bernhardt&amp;author=PA+de+Boer&amp;journal=Mol+Microbiol&amp;volume=48&amp;doi=10.1046%2Fj.1365-2958.2003.03511.x&amp;pages=1171-1182&amp;publication_year=2003&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li>
<li class="refbody" id="ref-R36">
<xhtml:span xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:lang="en"><a name="R36"></a><div class="citation"> <span class="reference-surname">McIntosh</span> <span class="reference-given-names">BK</span>. <span class="reference-article-title"> Bacteriophage MS2 L protein: genetic and biochemical charecterization. </span> <span class="reference-source">Ph.D thesis</span> College Station, TX: Texas A&amp;M University; <span class="reference-year">2008</span></div>
</xhtml:span>
<a href="https://scholar.google.com/scholar_lookup?title=Ph.D+thesis&amp;author=BK+McIntosh&amp;publication_year=2008&amp;" data-location="same-new-window" class="js-externallink">[Google Scholar]</a>
</li></ol>
</div>
</div>
<div id="cited" class="citations hidden-js-div tabbedsection tab-pane">
<div class="hiddenjsdiv" id="itemIdForCitations">http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.000485</div>
<img class="citingArticlesLoading" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/spinner.gif" alt="Loading">
<div id="citationContent"></div>
</div>
<div class="hiddenjsdiv itemmetadata">
<div class="metadata_title">Mutational analysis of the MS2 lysis protein L</div>
<div class="metadata_citation">
<span>Microbiology <span class="citationvolume">163</span></span><span>, 961</span><span> (2017)</span><span>; <a href="https://doi.org/10.1099/mic.0.000485" rel="external" class="externallink" title="undefined - opens in new window/tab" target="_blank">https://doi.org/10.1099/mic.0.000485</a></span>
</div>
<div class="metadata_itemWebId">/content/journal/micro/10.1099/mic.0.000485</div>
</div>
<div id="dataandmedia" class="dataandmedia tabbedsection hidden-js-div tab-pane" data-fulltexturl="" data-fulltextexpandlinktext="Toggle display:" data-hideinlinetablesbydefault="false">



<div id="hiddenDataMediaWebid" class="hiddenjsdiv">/content/journal/micro/10.1099/mic.0.000485</div>
<div class="itemDataMediaLoading">
<img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/spinner.gif" alt="Loading">
<p>Data &amp; Media loading...</p>
</div>
<div id="datamediacontent"></div>
</div>
<div id="supplementary_data" class="supplementary-container tabbedsection hidden-js-div tab-pane">
<div id="supplementary_data_contents" data-statslogurl="/statslogredirect">
<p class="description">
<span class="tl-main-part">
Supplements
</span>
</p>
<h4 id="supp_title_1" class="supplementary-item-heading"><span class="capture-id">
<span>Supplementary File 1</span>
</span>
<br><br></h4>
<div class="col-sm-4 pull-left text-left mg-t-20">
<div class="access-options pdfItem-options">
<div class="ft-download-content ft-download-content--pdf">
<form method="POST" action="https://www.microbiologyresearch.org/deliver/fulltext/micro/163/7/000485.pdf?itemId=%2Fcontent%2Fsuppdata%2Fmicro%2F10.1099%2Fmic.0.000485-1&amp;mimeType=pdf" target="/content/suppdata/micro/10.1099/mic.0.000485-1-pdf" data-title="Download" data-itemid="http://instance.metastore.ingenta.com/content/suppdata/micro/10.1099/mic.0.000485-1" class="ft-download-content__form ft-download-content__form--pdf js-ft-download-form ">
<i class="fa fa-file-pdf-o 
access-options-icon" aria-hidden="true"></i>
<span class="hidden-xxs">PDF</span> 
</form>
</div>
</div>
</div>
</div>
</div>
</section>
<div class="clearfix"></div>
<div class="mostreadcontainer panel panel-default" style="">
<div class="panel-heading">
<h3 class="panel-title">
Most read this month
</h3>
</div>
<div class="panel-body">
<div id="hiddenmostviewedtype" class="hiddenjsdiv hidden-js-div">Article</div>
<div id="mostviewed">
<ul class="separated-list list-unstyled list-unstyled">
<li class="">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_PLoS_transparent.svg" alt="Open Access">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.001442" data-itemid="/content/journal/micro/10.1099/mic.0.001442" dir="auto" class="js-access-determined"> 
Ecological dependencies and the illusion of cooperation in microbial communities
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Elze+Hesse&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Elze Hesse</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=Siobh%C3%A1n+O%E2%80%99Brien&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Siobhán O’Brien</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_PLoS_transparent.svg" alt="Open Access">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.001269" data-itemid="/content/journal/micro/10.1099/mic.0.001269" dir="auto" class="js-access-determined"> 
A comprehensive list of bacterial pathogens infecting humans
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Abigail+Bartlett&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Abigail Bartlett</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Daniel+Padfield&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Daniel Padfield</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Luke+Lear&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Luke Lear</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Richard+Bendall&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Richard Bendall</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=Michiel+Vos&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Michiel Vos</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_PLoS_transparent.svg" alt="Open Access">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.001428" data-itemid="/content/journal/micro/10.1099/mic.0.001428" dir="auto" class="js-access-determined"> 
Microbial Primer: Bacterial growth kinetics
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Lorena+T.+Fern%C3%A1ndez-Mart%C3%ADnez&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Lorena T. Fernández-Martínez</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Arnaud+Javelle&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Arnaud Javelle</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=Paul+A.+Hoskisson&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Paul A. Hoskisson</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_PLoS_transparent.svg" alt="Open Access">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.001418" data-itemid="/content/journal/micro/10.1099/mic.0.001418" dir="auto" class="js-access-determined"> 
<span class="jp-italic">Clostridioides difficile</span> spores tolerate disinfection with sodium hypochlorite disinfectant and remain viable within surgical scrubs and gown fabrics
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Humaira+Ahmed&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Humaira Ahmed</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=Lovleen+Tina+Joshi&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Lovleen Tina Joshi</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_PLoS_transparent.svg" alt="Open Access">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.001227" data-itemid="/content/journal/micro/10.1099/mic.0.001227" dir="auto" class="js-access-determined"> 
A guide for membrane potential measurements in Gram-negative bacteria using voltage-sensitive dyes
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Jessica+A.+Buttress&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Jessica A. Buttress</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Manuel+Halte&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Manuel Halte</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=J.+Derk+te+Winkel&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">J. Derk te Winkel</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Marc+Erhardt&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Marc Erhardt</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Philipp+F.+Popp&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Philipp F. Popp</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=Henrik+Strahl&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Henrik Strahl</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li>
</li>
</ul>
<form class="ahahTocArticles">
<input type="hidden" name="articleIds" value="http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.001442,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.001269,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.001428,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.001418,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.001227">
<input type="hidden" name="fmt" value="ahah">
<input type="hidden" name="ahahcontent" value="toc">
</form>
</div> 

<div id="hiddenmostviewedparent" class="hiddenjsdiv hidden-js-div">content/journal/micro</div>
<div id="hiddenmostviewedparenttype" class="hiddenjsdiv hidden-js-div">Journal</div>
<div id="hiddenmostviewednumber" class="hiddenjsdiv hidden-js-div">5</div>
<div id="hiddenmostviewednumbershown" class="hiddenjsdiv hidden-js-div">3</div>
<div id="hiddenmostviewedemptymessage" class="hiddenjsdiv hidden-js-div">false</div>
<div id="hiddenmostviewedlanguage" class="hiddenjsdiv hidden-js-div">en</div>

</div>
</div>
<div class="mostcitedcontainer panel panel-default ">
<div class="panel-heading">
<h2 class="panel-title">
Most cited
<span class="rssicon">
<a href="https://www.microbiologyresearch.org/rss/content/journal/micro/mostcitedarticles?fmt=rss" class="rssiconheader rssFeed externallink" title="Subscribe to most Cited RSS feed">
<i class="fa fa-rss"></i>
<span class="sr-only">Most Cited RSS feed</span>
</a>
</span>
</h2>
</div>
<div class="panel-body">
<div id="mostcited">
<ul class="separated-list list-unstyled">
<li class="" id="mostCited__content_journal_micro_10.1099_mic.0.037143-0_1">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_green_transparent.svg" alt="Free">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.037143-0" data-itemid="/content/journal/micro/10.1099/mic.0.037143-0" dir="auto" class="js-access-determined"> 
Metals, minerals and microbes: geomicrobiology and bioremediation
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item author-list__item--last"><a href="https://www.microbiologyresearch.org/search?value1=Geoffrey+Michael+Gadd&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Geoffrey Michael Gadd</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="" id="mostCited__content_journal_micro_10.1099_00221287-111-1-1_2">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_green_transparent.svg" alt="Free">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/00221287-111-1-1" data-itemid="/content/journal/micro/10.1099/00221287-111-1-1" dir="auto" class="js-access-determined"> 
Generic Assignments, Strain Histories and Properties of Pure Cultures of Cyanobacteria
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Rosmarie+Rippka&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Rosmarie Rippka</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Josette+Deruelles&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Josette Deruelles</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=John+B.+Waterbury&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">John B. Waterbury</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Michael+Herdman&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Michael Herdman</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=Roger+Y.+Stanier&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Roger Y. Stanier</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="" id="mostCited__content_journal_micro_10.1099_00221287-146-10-2395_3">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_green_transparent.svg" alt="Free">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/00221287-146-10-2395" data-itemid="/content/journal/micro/10.1099/00221287-146-10-2395" dir="auto" class="js-access-determined"> 
Quantification of biofilm structures by the novel computer program <span class="jp-small">comstat</span>
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Arne++Heydorn&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Arne Heydorn</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Alex+Toftgaard++Nielsen&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Alex Toftgaard Nielsen</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Morten++Hentzer&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Morten Hentzer</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Claus++Sternberg&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Claus Sternberg</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Michael++Givskov&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Michael Givskov</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Bjarne+Kj%C3%A6r++Ersb%C3%B8ll&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Bjarne Kjær Ersbøll</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=S%C3%B8ren++Molin&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Søren Molin</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="" id="mostCited__content_journal_micro_10.1099_13500872-142-8-2187_4">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_green_transparent.svg" alt="Free">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/13500872-142-8-2187" data-itemid="/content/journal/micro/10.1099/13500872-142-8-2187" dir="auto" class="js-access-determined"> 
Autotrophic growth of anaerobic ammonium-oxidizing micro-organisms in a fluidized bed reactor
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Astrid+A.+van+de+Graaf&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Astrid A. van de Graaf</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Peter+de+Bruijn&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Peter de Bruijn</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Lesley+A.+Robertson&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Lesley A. Robertson</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Mike+S.+M.+Jetten&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Mike S. M. Jetten</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=J.+Gijs+Kuenen&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">J. Gijs Kuenen</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="" id="mostCited__content_journal_micro_10.1099_mic.0.052274-0_5">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_PLoS_transparent.svg" alt="Open Access">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.052274-0" data-itemid="/content/journal/micro/10.1099/mic.0.052274-0" dir="auto" class="js-access-determined"> 
Plant-beneficial effects of <span class="jp-italic">Trichoderma</span> and of its genes
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Rosa+Hermosa&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Rosa Hermosa</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Ada+Viterbo&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Ada Viterbo</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Ilan+Chet&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Ilan Chet</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=Enrique+Monte&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Enrique Monte</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="hiddenjsdiv" id="mostCited__content_journal_micro_10.1099_mic.0.28048-0_6">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_green_transparent.svg" alt="Free">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.28048-0" data-itemid="/content/journal/micro/10.1099/mic.0.28048-0" dir="auto" class="js-access-determined"> 
Clustered regularly interspaced short palindrome repeats (CRISPRs) have spacers of extrachromosomal origin
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Alexander+Bolotin&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Alexander Bolotin</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Benoit+Quinquis&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Benoit Quinquis</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Alexei+Sorokin&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Alexei Sorokin</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=S.+Dusko+Ehrlich&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">S. Dusko Ehrlich</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="hiddenjsdiv" id="mostCited__content_journal_micro_10.1099_00221287-148-1-257_7">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_green_transparent.svg" alt="Free">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/00221287-148-1-257" data-itemid="/content/journal/micro/10.1099/00221287-148-1-257" dir="auto" class="js-access-determined"> 
Determination of bacterial load by real-time PCR using a broad-range (universal) probe and primers set
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Mangala+A+Nadkarni&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Mangala A Nadkarni</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=F.+Elizabeth+Martin&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">F. Elizabeth Martin</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Nicholas+A+Jacques&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Nicholas A Jacques</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=Neil+Hunter&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Neil Hunter</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="hiddenjsdiv" id="mostCited__content_journal_micro_10.1099_mic.0.000860_8">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_PLoS_transparent.svg" alt="Open Access">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000860" data-itemid="/content/journal/micro/10.1099/mic.0.000860" dir="auto" class="js-access-determined"> 
Microbe Profile: <span class="jp-italic">Pseudomonas aeruginosa</span>: opportunistic pathogen and lab rat
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Stephen+P.+Diggle&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Stephen P. Diggle</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=Marvin+Whiteley&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Marvin Whiteley</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li class="hiddenjsdiv" id="mostCited__content_journal_micro_10.1099_00221287-143-12-3703_10">
<div class="table-cell meta-data-container pd-remove">
<ul class="list-unstyled browse-item js-browse-item">
<li>
<div class="articleMetadata js-articleMetadata">
<div class="articleMetadataInner js-articleMetadataInner">
<h3 class="accessIcons js-accessIcons"><img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_green_transparent.svg" alt="Free">
<span class="articleTitle js-articleTitle title">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/00221287-143-12-3703" data-itemid="/content/journal/micro/10.1099/00221287-143-12-3703" dir="auto" class="js-access-determined"> 
Quorum sensing and <span class="jp-italic">Chromobacterium violaceum</span>: exploitation of violacein production and inhibition for the detection of <span class="jp-italic">N</span>-acylhomoserine lactones
</a>
</span>
</h3>
<div class="authorsWithPopup">
<div><span class="meta-key authors_label ">
</span>
<span class="meta-value authors ">
<span class="author-list__item "><a href="https://www.microbiologyresearch.org/search?value1=Kay+H.+McClean&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Kay H. McClean</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Michael+K.+Winson&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Michael K. Winson</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Leigh+Fish&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Leigh Fish</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Adrian+Taylor&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Adrian Taylor</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Siri+Ram+Chhabra&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Siri Ram Chhabra</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Miguel+Camara&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Miguel Camara</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Mavis+Daykin&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Mavis Daykin</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=John+H.+Lamb&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">John H. Lamb</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Simon+Swift&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Simon Swift</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Barrie+W.+Bycroft&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Barrie W. Bycroft</a></span><span class="author-list__item ">,
<a href="https://www.microbiologyresearch.org/search?value1=Gordon+S.+A.+B.+Stewart&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Gordon S. A. B. Stewart</a></span><span class="author-list__item author-list__item--last">&nbsp;and
<a href="https://www.microbiologyresearch.org/search?value1=Paul+Williams&amp;option1=author&amp;noRedirect=true" class="nonDisambigAuthorLink">Paul Williams</a></span>
</span></div>
</div>
</div>
</div>
</li>
</ul>
</div>
</li>
<li>
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="moreLink std-display" role="button" aria-controls="mostCited__content_journal_micro_10.1099_mic.0.28048-0_6 mostCited__content_journal_micro_10.1099_00221287-148-1-257_7 mostCited__content_journal_micro_10.1099_mic.0.000860_8 mostCited__content_journal_micro_10.1099_00221287-143-12-3703_10" aria-expanded="false">
<i class="fa fa-plus"></i> More
</a> 
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="lessLink std-display" role="button" aria-controls="mostCited__content_journal_micro_10.1099_mic.0.28048-0_6 mostCited__content_journal_micro_10.1099_00221287-148-1-257_7 mostCited__content_journal_micro_10.1099_mic.0.000860_8 mostCited__content_journal_micro_10.1099_00221287-143-12-3703_10" aria-expanded="true">
<i class="fa fa-minus"></i> Less
</a>
</li>
</ul>
<form class="ahahTocArticles">
<input type="hidden" name="articleIds" value="http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.037143-0,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/00221287-111-1-1,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/00221287-146-10-2395,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/13500872-142-8-2187,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.052274-0,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.28048-0,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/00221287-148-1-257,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.000860,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/0022-1317-81-4-853,http://instance.metastore.ingenta.com/content/journal/micro/10.1099/00221287-143-12-3703">
<input type="hidden" name="fmt" value="ahah">
<input type="hidden" name="ahahcontent" value="toc">
</form>
</div>
</div>
</div>
<input type="hidden" id="fancyBoxImgLoadErrMsg" value="Image can not be loaded">
</main>
<aside class="col-xs-12 col-sm-12 col-md-3 footer-sidebar left-padding non-printed-section">
<div class="horizontally-centered-content altmetrics sidebar-pub2web-element">
<div data-badge-popover="bottom" data-badge-type="medium-donut" data-doi="10.1099/mic.0.000485" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed" data-uuid="4b195688-944c-70de-61ce-0b310a98ab54"><a target="_blank" href="https://www.altmetric.com/details.php?domain=www.microbiologyresearch.org&amp;citation_id=31549467" rel="popover" data-content="&lt;div&gt;    &lt;div style=&#39;padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;&#39;&gt;
      &lt;a class=&#39;link-to-altmetric-details-tab&#39; target=&#39;_blank&#39; href=&#39;https://www.altmetric.com/details.php?domain=www.microbiologyresearch.org&amp;citation_id=31549467&amp;tab=twitter&#39;&gt;
          Posted by &lt;b&gt;2&lt;/b&gt; X users
      &lt;/a&gt;
    &lt;/div&gt;
    &lt;div style=&#39;padding-left: 10px; line-height:18px; border-left: 16px solid #f27700;&#39;&gt;
      &lt;a class=&#39;link-to-altmetric-details-tab&#39; target=&#39;_blank&#39; href=&#39;https://www.altmetric.com/details.php?domain=www.microbiologyresearch.org&amp;citation_id=31549467&amp;tab=patents&#39;&gt;
          Referenced in &lt;b&gt;2&lt;/b&gt; patents
      &lt;/a&gt;
    &lt;/div&gt;
    &lt;div style=&#39;padding-left: 10px; line-height:18px; border-left: 16px solid #958899;&#39;&gt;
      &lt;a class=&#39;link-to-altmetric-details-tab&#39; target=&#39;_blank&#39; href=&#39;https://www.altmetric.com/details.php?domain=www.microbiologyresearch.org&amp;citation_id=31549467&amp;tab=wikipedia&#39;&gt;
          Referenced in &lt;b&gt;2&lt;/b&gt; Wikipedia pages
      &lt;/a&gt;
    &lt;/div&gt;

    &lt;div class=&#39;altmetric-embed readers&#39; style=&#39;margin-top: 10px;&#39;&gt;
          &lt;div class=&#39;altmetric-embed tip_mendeley&#39;
               style=&#39;padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;&#39;&gt;
              &lt;b&gt;20&lt;/b&gt; readers on Mendeley
          &lt;/div&gt;
    &lt;/div&gt;
    &lt;div style=&#39;margin-top: 10px; text-align: center;&#39;&gt;
        &lt;a class=&#39;altmetric_details&#39; target=&#39;_blank&#39; href=&#39;https://www.altmetric.com/details.php?domain=www.microbiologyresearch.org&amp;citation_id=31549467&#39;&gt;
            See more details
        &lt;/a&gt;
    &lt;/div&gt;
&lt;/div&gt;" style="display:inline-block;" data-badge-popover="bottom">
        <img alt="Article has an altmetric score of 10" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/saved_resource" width="120" height="120" style="border: 0px; margin: 0px; max-width: none; --darkreader-inline-border-top: initial; --darkreader-inline-border-right: initial; --darkreader-inline-border-bottom: initial; --darkreader-inline-border-left: initial;" data-darkreader-inline-border-top="" data-darkreader-inline-border-right="" data-darkreader-inline-border-bottom="" data-darkreader-inline-border-left="">
</a></div>
<script type="text/javascript" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/embed.js"></script>
<span class="__dimensions_badge_embed__" data-doi="10.1099/mic.0.000485" data-dimensions-badge-installed="true"><a class="__dimensions_Link" href="https://badge.dimensions.ai/details/doi/10.1099/mic.0.000485?domain=https://www.microbiologyresearch.org" target="_blank" referrerpolicy="no-referrer-when-downgrade" aria-label="14 total citations on Dimensions." style="display: none;"><div class="__dimensions_Badge __dimensions_Badge_style_medium_circle"><div class="__dimensions_Badge_Image">
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 600 600">
    <defs>
      <filter id="n3gbbp-blur" color-interpolation-filters="sRGB">
        <fegaussianblur in="SourceGraphic" stdDeviation="40"></fegaussianblur>
      </filter>
      <mask id="n3gbbp-shape-mask">
        <path fill="white" d="M546.5,287.3l-111.8-194c-1.8-3-4.1-5.6-6.7-7.7c-2.2-2.7-5.4-4.3-8.9-4.5c-2.3-0.7-4.7-1-7-1
          L188.2,80c-9.3,0-18,5-22.6,13.1l-112,193.8c-4.7,8.1-4.7,18,0,26.1l111.9,193.9c4.7,8.1,13.3,13,22.6,13.1l223.9,0.1
          c9.3,0,17.9-4.8,22.6-12.8l112-193.7l0,0C551.1,305.1,551.2,295.4,546.5,287.3z M516.6,287.4l-41.5,0.1l-26.7-118.2L516.6,287.4z
           M337.2,106.2l-114.5,36.3l-20.3-36.4L337.2,106.2z M180.7,119l20.1,36.2L112,237.7L180.7,119z M181,481.8l-69.3-120l89.2,84
          L181,481.8z M203.3,493.8l19.6-35.6l114.6,35.7L203.3,493.8z M408.1,489.3l-152.7-47.5l-32.3-10.6l-25.3-23.2L82.6,299.6
          l114.6-106.9l25-23.1v-0.2l0.7-0.2l32.4-10.7l153.6-48.7l35.5,156.6l7.5,33.4l-7.6,33.5L408.1,489.3z M447.1,432.9l27.9-120.3
          l41.7-0.1L447.1,432.9z" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #2d2e2f;"></path>
      </mask>
      <g id="n3gbbp-gradient" filter="url(#n3gbbp-blur)">
        <g>
          <path fill="#006EB6" d="M0,0l300,300v-424.4C182.9-124.4,76.8-76.9,0,0z" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #19608e;"></path>
          <path fill="#28348A" d="M300-124.4V300L600,0C523.2-76.9,417.1-124.4,300-124.4z" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #333b72;"></path>
          <path fill="#E20613" d="M600,600c76.8-76.8,124.3-182.9,124.3-300H300L600,600z" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #aa1d25;"></path>
          <path fill="#DA0078" d="M300,300h424.4c0-117.1-47.5-223.2-124.3-300L300,300z" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #a51966;"></path>
          <path fill="#5BC4F1" d="M-124.4,300H300L0,0C-76.9,76.8-124.4,182.9-124.4,300z" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #236e8e;"></path>
          <path fill="#A1C517" d="M0,600l300-300h-424.4C-124.4,417.1-76.9,523.2,0,600z" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #819828;"></path>
          <path fill="#F08800" d="M300,724.4c117.1,0,223.2-47.5,300-124.3L300,300V724.4z" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #b37119;"></path>
          <path fill="#FFCB00" d="M300,724.4V300L0,600C76.8,676.9,182.9,724.4,300,724.4z" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #947b19;"></path>
        </g>
      </g>
    </defs>
  </svg>

  <div class="__db_background"></div>

  <div class="__db_shape">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 600 600" xmlns:xlink="http://www.w3.org/1999/xlink">
      <use mask="url(#n3gbbp-shape-mask)" xlink:href="#n3gbbp-gradient"></use>
    </svg>
  </div>

  <div class="__db_score __db_score_normal">14</div>
  <div class="__db_label">CITATIONS</div>

  <img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/badge" class="__dimensions_png" alt="14 total citations on Dimensions.">
</div>
</div><div class="__dimensions_Badge_Legend_padding __dimensions_Badge_Legend_hover-right  __dimensions_Badge_Legend_style_medium_circle"><div class="__dimensions_Badge_Legend">
      <div class="__dimensions_Badge_stat_group __dimensions_Badge_stat_group_citations">
        <div class="__dimensions_Badge_stat __dimensions_Badge_stat_total_citations">
          <span class="__dimensions_Badge_stat_icon"></span>
          <span class="__dimensions_Badge_stat_count">14</span>
          <span class="__dimensions_Badge_stat_text">Total citations</span>
        </div>
        <div class="__dimensions_Badge_stat __dimensions_Badge_stat_recent_citations">
          <span class="__dimensions_Badge_stat_icon"></span>
          <span class="__dimensions_Badge_stat_count">8</span>
          <span class="__dimensions_Badge_stat_text">Recent citations</span>
        </div>
      </div>
      <div class="__dimensions_Badge_stat_group __dimensions_Badge_stat_group_cr">
        <div class="__dimensions_Badge_stat __dimensions_Badge_stat_fcr">
          <span class="__dimensions_Badge_stat_icon"></span>
          <span class="__dimensions_Badge_stat_count">1.55</span>
          <span class="__dimensions_Badge_stat_text">Field Citation Ratio</span>
        </div>
        <div class="__dimensions_Badge_stat __dimensions_Badge_stat_rcr">
          <span class="__dimensions_Badge_stat_icon"></span>
          <span class="__dimensions_Badge_stat_count">0.57</span>
          <span class="__dimensions_Badge_stat_text">Relative Citation Ratio</span>
        </div>
      </div></div></div></a></span><script async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/badge.js" charset="utf-8"></script>
</div>
<div class="panel panel-default sidebar-pub2web-element">
<div class="panel-heading">
<h4 class="panel-title">We Recommend</h4>
</div>
<div id="trendmd-suggestions"><div class="trendmd-widget-container trendmd-widget_title-normal trendmd-widget_body-normal" data-trendmd-journal-id="17540" data-trendmd-article-id="48446ab3-cca0-45eb-b1cd-bfb7cf9ac758" data-trendmd-swoop="e30=" similar-nonsponsored-merged="true" style="display: block;" data-trendmd-widget-id="0" similar-sponsored-merged="true">
  <div class="trendmd-widget trendmd-widget_vertical">
    <div class="trendmd-widget-inner">
        <div class="trendmd-widget-section">
          <div class="trendmd-widget-header">
                          <h3 class="trendmd-widget-header__heading">We recommend</h3>
                      </div>
          <ol class="trendmd-widget-list"><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="4d67ac07-dc87-4203-bd63-01648694a291" data-trendmd-journal-id="36655" data-trendmd-content-category="Scholarly" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="http://jgv.microbiologyresearch.org/content/journal/jgv/10.1099/0022-1317-83-6-1523?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=J_Gen_Virol_TrendMD_0" rel="noindex, nofollow" target="_self" data-trendmd-flags="2" data-trendmd-model-type="0">Nucleotide sequence of a ssRNA phage from Acinetobacter: kinship to coliphages</a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">J. Klovins et al.,</span>
                                                        J Gen Virol,                                                         <span class="js_publication_date">2002</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="f0f8d564-78d4-4d47-a505-63ec6323954b" data-trendmd-journal-id="17540" data-trendmd-content-category="Scholarly" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="http://mic.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.064790-0?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=Microbiology_TrendMD_0" rel="noindex, nofollow" target="_self" data-trendmd-flags="2" data-trendmd-model-type="0">A2 expression and assembly regulates lysis in Qβ infections</a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Catrina A. Reed et al.,</span>
                                                        Microbiology,                                                         <span class="js_publication_date">2013</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="8128df46-9b3e-4e11-b601-b53d2d312c9f" data-trendmd-journal-id="36655" data-trendmd-content-category="Scholarly" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="http://jgv.microbiologyresearch.org/content/journal/jgv/10.1099/0022-1317-77-1-129?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=J_Gen_Virol_TrendMD_0" rel="noindex, nofollow" target="_self" data-trendmd-flags="2" data-trendmd-model-type="0">Identification of a short domain within the non-structural protein NS2 of epizootic haemorrhagic disease virus that is important for single strand RNA-binding activity</a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">J. Theron et al.,</span>
                                                        J Gen Virol,                                                         <span class="js_publication_date">1996</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="bcee3252-b212-48b7-b0f3-90827e65ad2d" data-trendmd-journal-id="17540" data-trendmd-content-category="Scholarly" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="http://mic.microbiologyresearch.org/content/journal/micro/10.1099/00221287-144-12-3463?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=Microbiology_TrendMD_0" rel="noindex, nofollow" target="_self" data-trendmd-flags="2" data-trendmd-model-type="0">Cell integrity and morphogenesis in a budding yeast septin mutant</a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Victor J. Cid et al.,</span>
                                                        Microbiology,                                                         <span class="js_publication_date">1998</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="d40cf3d4-ff87-46ce-b42d-e3a4c991e7a4" data-trendmd-journal-id="17540" data-trendmd-content-category="Scholarly" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="http://mic.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.2008/016956-0?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=Microbiology_TrendMD_0" rel="noindex, nofollow" target="_self" data-trendmd-flags="2" data-trendmd-model-type="0">Evolutionary dominance of holin lysis systems derives from superior genetic malleability</a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Yi Zheng et al.,</span>
                                                        Microbiology,                                                         <span class="js_publication_date">2008</span>
                                  </div>
              </li></ol>
        </div>
      
        <div class="trendmd-widget-section">
          <div class="trendmd-widget-header">
                      </div>
          <ol class="trendmd-widget-list"><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="3ef75b27-7f01-4119-96c5-ed48124d8918" data-trendmd-journal-id="54233" data-trendmd-content-category="Scholarly" data-trendmd-campaign-id="19116" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.science.org/doi/10.1126/science.275.5302.980?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=TrendMD_1" rel="noindex, nofollow" target="_blank" data-trendmd-flags="2" data-trendmd-model-type="0">Structural and Functional Conservation of the Caenorhabditis elegans Timing Gene clk-1<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Jonathan J. Ewbank et al.,</span>
                                                        Science,                                                         <span class="js_publication_date">1997</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="02afc78f-71cc-4ffc-882c-c6b60888fbbc" data-trendmd-journal-id="54233" data-trendmd-content-category="Scholarly" data-trendmd-campaign-id="19116" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.science.org/doi/10.1126/science.280.5368.1447?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=TrendMD_1" rel="noindex, nofollow" target="_blank" data-trendmd-flags="2" data-trendmd-model-type="0">Association of Unconventional Myosin MYO15 Mutations with Human Nonsyndromic Deafness DFNB3<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Aihui Wang et al.,</span>
                                                        Science,                                                         <span class="js_publication_date">1998</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="26815895-f50a-4ebf-88d4-c804938e405a" data-trendmd-journal-id="55759" data-trendmd-content-category="Scholarly" data-trendmd-campaign-id="19116" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.science.org/doi/10.1126/sciimmunol.aah6109?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=TrendMD_1" rel="noindex, nofollow" target="_blank" data-trendmd-flags="2" data-trendmd-model-type="0">Characterization of T and B cell repertoire diversity in patients with RAG deficiency<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Yu Nee Lee et al.,</span>
                                                        Sci Immunol,                                                         <span class="js_publication_date">2016</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="c2e5f66b-867a-4953-9f4a-352e106bff69" data-trendmd-journal-id="55759" data-trendmd-content-category="Scholarly" data-trendmd-campaign-id="19116" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.science.org/doi/10.1126/sciimmunol.aan5165?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=TrendMD_1" rel="noindex, nofollow" target="_blank" data-trendmd-flags="2" data-trendmd-model-type="0">Human thymoproteasome variations influence CD8 T cell selection<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Takeshi Nitta et al.,</span>
                                                        Sci Immunol,                                                         <span class="js_publication_date">2017</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="662458a0-ca5b-4150-8f8e-81f5682cc6ae" data-trendmd-journal-id="3439" data-trendmd-content-category="Scholarly" data-trendmd-campaign-id="19116" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.science.org/doi/abs/10.1126/scitranslmed.aar3959?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=TrendMD_1" rel="noindex, nofollow" target="_blank" data-trendmd-flags="2" data-trendmd-model-type="0">Huntingtin suppression restores cognitive function in a mouse model of Huntington’s disease<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Amber L. Southwell et al.,</span>
                                                        Sci Transl Med,                                                         <span class="js_publication_date">2018</span>
                                  </div>
              </li></ol>
        </div>
      
    </div>
    <div class="trendmd-widget-footer"><div class="trendmd-widget-footer-inner">
        <div class="trendmd-widget-footer-inner-left">
          <a href="https://www.trendmd.com/how-it-works-readers" target="_blank" class="trendmd-widget-brand">
            <span class="trendmd-widget-brand__recommended">Powered by</span>
            <span class="trendmd-widget-brand__logo trendmd-widget-brand__logo__faded" style="display: inline-block; width: 93px">
              <svg viewBox="0 0 234 57" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
                <g id="logo">
                  <path d="M227.7,56.9 L149.7,56.9 C146.7,56.9 143.7,53.8 143.7,50.9 L143.7,6.9 C143.7,4 146.7,0.9 149.7,0.9 L227.7,0.9 C230.7,0.9 233.7,4 233.7,6.9 L233.7,50.9 C233.7,53.9 230.6,56.9 227.7,56.9 L227.7,56.9 Z" id="shape0" fill="#CA2127" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #9b2e32;"></path>
                  <g id="group1" transform="translate(0.000000, 14.000000)" fill="#16325c" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #283955;">
                    <path d="M15.8,5.9 L15.8,29.6 L9.2,29.6 L9.2,5.9 L0.9,5.9 L0.9,0.2 L24.1,0.2 L24.1,5.9 L15.8,5.9 L15.8,5.9 Z" id="shape1"></path>
                    <path d="M50.2,10 C50.2,14.7 48.3,17.7 44.6,19.1 L52,29.6 L44,29.6 L37.5,20.2 L33,20.2 L33,29.6 L26.4,29.6 L26.4,0.2 L37.5,0.2 C42.1,0.2 45.3,1 47.3,2.5 C49.2,4.1 50.2,6.6 50.2,10 L50.2,10 Z M42.3,13.5 C43.1,12.8 43.5,11.6 43.5,10 C43.5,8.4 43.1,7.3 42.2,6.7 C41.4,6.1 39.9,5.8 37.8,5.8 L32.9,5.8 L32.9,14.5 L37.7,14.5 C40,14.6 41.5,14.2 42.3,13.5 L42.3,13.5 Z" id="shape2"></path>
                    <path d="M75.6,0.3 L75.6,6.1 L61,6.1 L61,12.1 L74.1,12.1 L74.1,17.7 L61,17.7 L61,23.8 L76,23.8 L76,29.6 L54.4,29.6 L54.4,0.2 L75.6,0.2 L75.6,0.3 Z" id="shape3"></path>
                    <path d="M99.9,0.3 L106.5,0.3 L106.5,29.7 L99.9,29.7 L85.9,11.3 L85.9,29.7 L79.3,29.7 L79.3,0.3 L85.4,0.3 L99.8,19.2 L99.8,0.3 L99.9,0.3 Z" id="shape4"></path>
                    <path d="M133.6,4.1 C136.4,6.7 137.8,10.2 137.8,14.8 C137.8,19.4 136.4,23 133.7,25.6 C131,28.3 126.8,29.6 121.2,29.6 L111.2,29.6 L111.2,0.2 L121.6,0.2 C126.7,0.3 130.8,1.5 133.6,4.1 L133.6,4.1 Z M128.8,21.6 C130.4,20.1 131.2,17.9 131.2,15 C131.2,12.1 130.4,9.9 128.8,8.3 C127.2,6.7 124.7,6 121.4,6 L117.7,6 L117.7,23.9 L121.9,23.9 C124.8,23.8 127.1,23.1 128.8,21.6 L128.8,21.6 Z" id="shape5"></path>
                  </g>
                  <g id="group2" transform="translate(156.000000, 14.000000)" fill="#FFFFFF" data-darkreader-inline-fill="" style="--darkreader-inline-fill: #d3d2cf;">
                    <path d="M26.9,11.1 L19,27.1 L15,27.1 L7.1,11.1 L7.1,29.6 L0.5,29.6 L0.5,0.2 L9.4,0.2 L17,16.3 L24.6,0.2 L33.4,0.2 L33.4,29.6 L26.8,29.6 L26.8,11.1 L26.9,11.1 Z" id="shape6"></path>
                    <path d="M60.6,4.1 C63.4,6.7 64.8,10.2 64.8,14.8 C64.8,19.4 63.4,23 60.7,25.6 C58,28.3 53.8,29.6 48.2,29.6 L38.2,29.6 L38.2,0.2 L48.6,0.2 C53.7,0.3 57.8,1.5 60.6,4.1 L60.6,4.1 Z M55.8,21.6 C57.4,20.1 58.2,17.9 58.2,15 C58.2,12.1 57.4,9.9 55.8,8.3 C54.2,6.7 51.7,6 48.4,6 L44.7,6 L44.7,23.9 L48.9,23.9 C51.8,23.8 54.1,23.1 55.8,21.6 L55.8,21.6 Z" id="shape7"></path>
                  </g>
                </g>
              </svg>
            </span>
          </a>
        </div>
        <div class="trendmd-widget-footer-inner-right">
                    <div class="trendmd-widget-settings"><span class="trendmd-widget-settings__cog"></span>
            <div class="trendmd-widget-settings__modal">
              <ul class="trendmd-widget-settings__list">
                                
                  <li class="trendmd-widget-settings__item">
                    <a href="https://www.trendmd.com/targeting-settings" target="_blank" class="trendmd-widget-settings__link">Targeting settings</a>
                  </li>
                                <li class="trendmd-widget-settings__item trendmd-widget-dns" style="display:none">
                  <a href="https://www.trendmd.com/personaldata" target="_blank" class="trendmd-widget-settings__link">Do not sell my personal information</a>
                </li>
                                  <li class="trendmd-widget-settings__item">
                    <a href="https://www.trendmd.com/google-analytics" target="_blank" class="trendmd-widget-settings__link">Google Analytics settings</a>
                  </li>
                
                
                              </ul>
            </div>
          </div>
        </div>
        <div class="trendmd-widget-footer-inner-clear"></div>
        <div class="trendmd-widget-footer-inner-bottom">
                      <div class="trendmd-widget-cookie-notification">
              <div class="trendmd-widget-cookie-notification__info"><span>I consent to the use of Google Analytics and related cookies across the TrendMD network (widget, website, blog). </span><a href="https://www.trendmd.com/google-analytics#" target="_blank">Learn more</a></div>
              <div class="trendmd-widget-cookie-notification__actions">
                <button onclick="(function() { window.TrendMD.gdpr().setGaConsent(true); })(); return false;">Yes</button>
                <button onclick="(function() { window.TrendMD.gdpr().setGaConsent(false); })(); return false;">No</button>
                <span></span>
              </div>
            </div>
                  </div>
      </div></div>
  </div>
  
    <div id="trendmd-check">
      <div id="ad-holder"></div>
    </div>
  
</div>
</div>
<script defer="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/trendmd.min.js" data-trendmdconfig="{&quot;element&quot;:&quot;#trendmd-suggestions&quot;}"></script>
</div>
<div class="icon-key panel panel-default sidebar-pub2web-element">
<div class="panel-heading">
<h3 class="panel-title"> <i class="fa fa-key"></i> Access key</h3>
</div>
<div class="panel-body">
<ul class="list-unstyled">
<li class="icon-key__item access_icon_sc_wrapper">
<img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_blue_transparent.svg" alt="Subscribed"> Subscribed content
</li> 
<li class="icon-key__item access_icon_oa_wrapper">
<img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_PLoS_transparent.svg" alt="Open Access"> Open Access content
</li>
<li class="icon-key__item access_icon_ft_wrapper">
<img class="content__keyicon js-keyicon fr-fic fr-dii" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_green_transparent.svg" alt="Free Trial">&nbsp;Free Trial content
</li>
<li class="icon-key__item access_icon_fc_wrapper">
<img class="content__keyicon js-keyicon" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Open_Access_logo_green_transparent.svg" alt="Frees"> Free content
</li>
</ul>
</div>
</div>
<div class="sidebar-pub2web-element editable-snippet-image default-pane panel panel-default ">
<div class="panel-heading">
<div class="fr-view"><h3><a href="https://microbiologysociety.org/news/full-news-listing/thank-you-to-reviewers-of-the-microbiology-society-journals-in-2023.html" rel="noopener noreferrer" target="_blank">Thank you to our Reviewers</a></h3></div>
</div>
<div class="panel-body">
<div class="fr-view"><div style="text-align: center;"><a href="https://microbiologysociety.org/news/full-news-listing/thank-you-to-reviewers-of-the-microbiology-society-journals-in-2023.html" rel="noopener noreferrer" target="_blank"><img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Journal_web_adverts_Thank_you9.png" title="Reviewer Thank You" class="fr-fic fr-dii"></a></div><p>We are indebted to our reviewers, who refine and enhance research published in our titles to the benefit of everyone. They are essential to our publications and so support our work as a not-for-profit organisation.</p><p><a href="https://microbiologysociety.org/news/full-news-listing/thank-you-to-reviewers-of-the-microbiology-society-journals-in-2023.html" rel="noopener noreferrer" target="_blank"><strong>Thank you to reviewers of the Microbiology Society journals in 2023</strong></a></p></div>
</div>
</div> 
<div class="editable-snippet-image default-pane sidebar-pub2web-element panel panel-default">
<div class="panel-heading">
<div class="fr-view"><h3><a href="https://www.microbiologyresearch.org/content/bacterial-competitive-mechanisms" rel="noopener noreferrer">Bacterial Competitive Mechanisms</a></h3></div>
</div>
<div class="panel-body">
<div class="fr-view"><div style="text-align: center;"><a href="https://www.microbiologyresearch.org/content/bacterial-competitive-mechanisms" rel="noopener noreferrer" target="_blank"><img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/Mic-CollectionBacCom.png" title="Bacterial Competitive Mechanisms" class="fr-fic fr-dii"></a></div><p>Browse <em>Microbiology</em>'s <a href="https://www.microbiologyresearch.org/content/bacterial-competitive-mechanisms">dedicated article collection</a> exploring&nbsp;competitive mechanisms bacteria use to thrive in their niches, including the molecular characterization of toxins or secretion systems and the investigation of interbacterial competition at the organismal level.</p></div>
</div>
</div>
<div class="sidebar-pub2web-element optional-snippet">
<div class="fr-view"></div>
</div>
</aside>
</div>
<footer class="container-fluid footer non-printed-section " id="footer">
<div class="container">
<div class="backToTopLink">
<a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#header" title="Back to top" data-toggle="tooltip" data-placement="left" class="btn-back-top js-back-top" style="display: none;"><i class="fa fa-arrow-up" aria-hidden="true"></i><span class="sr-only">Back to top</span></a>
</div>
<div class="row">
<nav class="footer-aboutUs col-xs-12 col-sm-3 col-md-3" aria-labelledby="aboutus-footer">
<h3 id="aboutus-footer">About Us</h3>
<div class="fr-view"><ul class="list-unstyled"><li><span style="color: rgb(255, 255, 255); --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color=""><a class="js-externallink" href="https://microbiologysociety.org/" tabindex="-1" title="Microbiology Society">Microbiology Society</a></span></li><li><span style="color: rgb(255, 255, 255); --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color=""><a class="js-externallink" href="https://microbiologysociety.org/about/our-staff.html" tabindex="-1" title="Our staff">Our staff</a></span></li><li><span style="color: rgb(255, 255, 255); --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color=""><a href="https://www.microbiologyresearch.org/license-agreements" tabindex="-1" title="License agreements">License agreements</a></span></li><li><span style="color: rgb(255, 255, 255); --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color=""><a href="https://www.microbiologyresearch.org/terms-conditions" tabindex="-1" title="Terms and conditions">Terms and conditions</a></span></li><li><span style="color: rgb(255, 255, 255); --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color=""><a href="https://www.microbiologyresearch.org/privacy-policy" tabindex="-1" title="Privacy policy">Privacy policy</a></span></li><li><span style="color: rgb(255, 255, 255); --darkreader-inline-color: #d3d2cf;" data-darkreader-inline-color=""><a href="https://www.microbiologyresearch.org/contact-us" tabindex="-1" title="Contact us">Contact us</a></span></li></ul></div>
</nav>
<nav class="footer-journals col-xs-12 col-sm-3 col-md-3" aria-labelledby="publication-footer">
<h3 id="publication-footer">Publications</h3>
<div class="fr-view"><ul class="list-unstyled"><li><a href="https://www.microbiologyresearch.org/content/journal/micro" tabindex="-1" title="Microbiology"><em>Microbiology</em></a></li><li><a href="https://www.microbiologyresearch.org/content/journal/jgv" tabindex="-1" title="Journal of General Virology"><em>Journal of General Virology</em></a></li><li><a href="https://www.microbiologyresearch.org/content/journal/jmm" tabindex="-1" title="Journal of Medical Microbiology"><em>Journal of Medical Microbiology</em></a></li><li><a href="https://www.microbiologyresearch.org/content/journal/mgen" tabindex="-1" title="Open Access "><em>Microbial Genomics</em></a></li><li><a href="https://www.microbiologyresearch.org/content/journal/ijsem" tabindex="-1" title="International Journal of Systematic and Evolutionary Microbiology"><em>International Journal of Systematic and Evolutionary Microbiology</em></a></li><li><a href="https://www.microbiologyresearch.org/content/journal/acmi" tabindex="-1" title="Access Microbiology"><em>Access Microbiology</em>, an open research platform</a></li><li><a href="https://www.microbiologyresearch.org/content/journal/acmi" tabindex="-1" title="Access Microbiology"></a><a href="https://www.microbiologyresearch.org/content/journal/jmmcr" tabindex="-1" title="JMM Case Reports"><em>JMM Case Reports</em></a></li></ul></div>
</nav>
<nav class="footer-resources col-xs-12 col-sm-6 col-md-6" aria-labelledby="resource-footer">
<h3 id="resource-footer">Resources</h3>
<div class="fr-view"><ul class="list-unstyled col-xs-12 col-sm-6 col-md-6"><li><a href="https://www.microbiologyresearch.org/information-for-authors" tabindex="-1" title="Information for authors">Information for authors</a></li><li><a href="https://www.microbiologyresearch.org/information-for-reviewers" tabindex="-1" title="Information for reviewers">Information for reviewers</a></li><li><a href="https://www.microbiologyresearch.org/information-for-librarians" tabindex="-1" title="Information for librarians">Information for librarians</a></li><li><a href="https://www.microbiologyresearch.org/publish-oa" tabindex="-1" title="Open Access">Open Access</a></li><li><a href="https://www.microbiologyresearch.org/ethics-policies" tabindex="-1" title="Ethics">Ethics policies</a></li><li><a href="https://www.microbiologyresearch.org/frequently-asked-questions" tabindex="-1" title="FAQs">FAQs</a></li><li><a href="https://www.microbiologyresearch.org/rights-and-permissions" tabindex="-1" title="Rights and permissions">Rights and permissions</a></li><li><a href="https://www.microbiologyresearch.org/reprints-and-advertising" tabindex="-1" title="Reprints and advertising">Reprints and advertising</a></li><li><br></li><li><br></li></ul></div>
<div class="fr-view"></div>
</nav>
<div class="container pull-left">
<p class="pull-left">
© 2024 Microbiology Society. Registered as a Charity in England and Wales 264017. A Charity registered in Scotland SC039250. Company Limited by Guarantee. Registered in England 1039582.&nbsp;
</p>
<a class="js-externallink" href="https://www.ingenta.com/products/content-management/ingenta-edify/"><source srcset="/upload/ingentaedify/powered-by-Ingenta-Edify-white-on-gradient.png 2x, /upload/ingentaedify/powered-by-Ingenta-Edify-white-on-gradient.png 1x"><img data-toggle="tooltip" data-placement="top" data-trigger="click" title="Ingenta Edify" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/powered-by-Ingenta-Edify-white-on-gradient.png" alt="Powered by Ingenta Edify" class="logo-footer fr-fic fr-dii" style="width:150px;max-height:80px;"></a>
</div>
</div>
</div> 
</footer>
</div>
<div class="hiddenblurbsvalues hidden-js-div">
<div id="required_field">
This is a required field
</div>
<div id="valid_email_address">
Please enter a valid email address
</div>
<div id="makelivesuccess">Approval was a Success</div>
<div id="makeliveinvaliddata">Invalid data</div>
<div id="makeliveerror">An Error Occurred</div>
<div id="makelivepartialsuccess">Approval was partially successful, following selected items could not be processed due to error</div>
</div> 
<form class="hidden" action="https://www.microbiologyresearch.org/cart/add" method="get" id="hiddenPurchaseForm"></form>
<form class="hidden" action="https://www.microbiologyresearch.org/carnet" method="post" id="hiddenCarnetForm"></form>
<div class="siqPageTitle hidden">Microbiology Society:</div>
<div class="siqItemId hidden">http://instance.metastore.ingenta.com/content/journal/micro/10.1099/mic.0.000485</div>
<div class="siqDoi hidden">10.1099/mic.0.000485</div>
<div class="siqPub2webEventType hidden">SEARCH_EXPAND_ITEM</div>
<script type="text/x-mathjax-config;executed=true">
MathJax.Hub.Config({
CommonHTML: { linebreaks: { automatic: true } },
"HTML-CSS": { linebreaks: { automatic: true } },
SVG: { linebreaks: { automatic: true } }
});
</script>
<script type="text/javascript">
var script = document.createElement('script');
script.type = 'text/javascript';
script.async = true;
script.src = '//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js?config=TeX-AMS-MML_HTMLorMML';
document.getElementsByTagName('head')[0].appendChild(script);
</script> 
<!--[if lt IE 7]>
<script src="/js/v/10.4.2/jp/IE7.js" ></script>
<![endif]-->
<!--[if lte IE 9]>
<script src="/js/v/10.4.2/hybrid/placeholder.js" ></script>
<![endif]-->
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/jquery.form.js" charset="utf-8"></script><iframe id="SHR" title="shr-utils" style="display: none; border: none; --darkreader-inline-border-top: initial; --darkreader-inline-border-right: initial; --darkreader-inline-border-bottom: initial; --darkreader-inline-border-left: initial;" data-darkreader-inline-border-top="" data-darkreader-inline-border-right="" data-darkreader-inline-border-bottom="" data-darkreader-inline-border-left="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/saved_resource.html"></iframe> 
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/jquery.validate.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/plugins.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/sitejp.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/site_common.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/platform.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/site.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/preview.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/jquery.horiz-bar-graph.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/ecommerceicons.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/toc_ecommerceicons.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/moreLikeThis.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/fulltext-html_media-objects.js" charset="utf-8"></script>
<script src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/responsive-tabs.js"></script>
<script type="text/javascript" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/NamesforLife.js"></script>
<script async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/js(4)"></script>
<script>
window.dataLayer = window.dataLayer || [];
function gtag(){dataLayer.push(arguments);}
gtag('js', new Date());
gtag('config', 'UA-48529861-1');
</script>
<div class="modal fade" id="myModal" tabindex="-1" role="dialog">
</div>
<script type="application/json" class="js-hypothesis-config">{ "openSidebar": false }</script>
<script>
if (matchMedia) {
var mq = window.matchMedia("(min-width: 1195px)");
mq.addListener(WidthChange);
WidthChange(mq);
}
function WidthChange(mq) {
// below classes are toggled between mobile and desktop
if (mq.matches) {
var s = document.createElement('script');
s.async = true;
s.src = 'https://hypothes.is/embed.js';
document.head.appendChild(s);
}
}
</script>
<script defer="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/v84a3a4012de94ce1a686ba8c167c359c1696973893317" integrity="sha512-euoFGowhlaLqXsPWQ48qSkBSCFs3DPRyiwVu3FjR96cMPx+Fr+gpWRhIafcHwqwCqWS42RZhIudOvEI+Ckf6MA==" data-cf-beacon="{&quot;rayId&quot;:&quot;864ee71b6e3b2d4e&quot;,&quot;version&quot;:&quot;2024.2.4&quot;,&quot;token&quot;:&quot;65a26c30563146b9a313a7cf94780b3e&quot;}" crossorigin="anonymous"></script>


<ul class="ui-autocomplete ui-menu ui-widget ui-widget-content ui-corner-all" id="ui-id-1" tabindex="0" style="z-index: 2; display: none;"></ul><div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-draggable ui-resizable" tabindex="-1" role="dialog" aria-labelledby="ui-id-2" style="outline: 0px; z-index: 1000; position: absolute; top: 169px; left: 230px; display: none; height: auto; width: 480px; --darkreader-inline-outline: initial;" data-darkreader-inline-outline=""><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-2" class="ui-dialog-title">&nbsp;</span><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="ui-dialog-titlebar-close ui-corner-all" role="button"><span class="ui-icon ui-icon-closethick">close</span></a></div><div id="figuredialog" class="ui-dialog-content ui-widget-content" style="width: auto; min-height: 0px; height: 577px;"></div><div class="ui-resizable-handle ui-resizable-n" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-e" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-s" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-w" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-se ui-icon ui-icon-gripsmall-diagonal-se ui-icon-grip-diagonal-se" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-sw" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-ne" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-nw" style="z-index: 1000;"></div></div><div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-draggable ui-resizable" tabindex="-1" role="dialog" aria-labelledby="ui-id-3" style="display: none; outline: 0px; z-index: 1000; position: absolute; --darkreader-inline-outline: initial;" data-darkreader-inline-outline=""><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-3" class="ui-dialog-title">&nbsp;</span><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="ui-dialog-titlebar-close ui-corner-all" role="button"><span class="ui-icon ui-icon-closethick">close</span></a></div><div id="tabledialog" class="ui-dialog-content ui-widget-content"></div><div class="ui-resizable-handle ui-resizable-n" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-e" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-s" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-w" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-se ui-icon ui-icon-gripsmall-diagonal-se ui-icon-grip-diagonal-se" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-sw" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-ne" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-nw" style="z-index: 1000;"></div></div><div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-draggable ui-resizable" tabindex="-1" role="dialog" aria-labelledby="ui-id-4" style="display: none; outline: 0px; z-index: 1000; position: absolute; --darkreader-inline-outline: initial;" data-darkreader-inline-outline=""><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-4" class="ui-dialog-title">&nbsp;</span><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="ui-dialog-titlebar-close ui-corner-all" role="button"><span class="ui-icon ui-icon-closethick">close</span></a></div><div id="videodialog" class="hide ui-dialog-content ui-widget-content" style="display: block;"></div><div class="ui-resizable-handle ui-resizable-n" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-e" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-s" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-w" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-se ui-icon ui-icon-gripsmall-diagonal-se ui-icon-grip-diagonal-se" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-sw" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-ne" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-nw" style="z-index: 1000;"></div></div><div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-draggable ui-resizable" tabindex="-1" role="dialog" aria-labelledby="ui-id-5" style="display: none; outline: 0px; z-index: 1000; position: absolute; --darkreader-inline-outline: initial;" data-darkreader-inline-outline=""><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-5" class="ui-dialog-title">&nbsp;</span><a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.000485#" class="ui-dialog-titlebar-close ui-corner-all" role="button"><span class="ui-icon ui-icon-closethick">close</span></a></div><div id="multimediadialog" class="ui-dialog-content ui-widget-content"></div><div class="ui-resizable-handle ui-resizable-n" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-e" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-s" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-w" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-se ui-icon ui-icon-gripsmall-diagonal-se ui-icon-grip-diagonal-se" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-sw" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-ne" style="z-index: 1000;"></div><div class="ui-resizable-handle ui-resizable-nw" style="z-index: 1000;"></div></div><script type="text/javascript" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/mic.0.000485"></script><script data-cfasync="false" async="" id="shr-partners" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/partners.js"></script><div class="altmetric-embed altmetric-popover altmetric-bottom" data-uuid="4b195688-944c-70de-61ce-0b310a98ab54" id="_altmetric_popover_el_4b195688-944c-70de-61ce-0b310a98ab54" style="margin: 0px; display: none;"><div class="altmetric_container"><div class="altmetric_arrow altmetric-bottom"></div><div class="altmetric-embed altmetric-popover-inner altmetric-floating altmetric-bottom" data-uuid="4b195688-944c-70de-61ce-0b310a98ab54"><div class="altmetric-embed altmetric-popover-content altmetric-floating"><div>    <div style="padding-left: 10px; line-height: 18px; border-left: 16px solid rgb(116, 207, 237); --darkreader-inline-border-left: #276d83;" data-darkreader-inline-border-left="">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.microbiologyresearch.org&amp;citation_id=31549467&amp;tab=twitter">
          Posted by <b>2</b> X users
      </a>
    </div>
    <div style="padding-left: 10px; line-height: 18px; border-left: 16px solid rgb(242, 119, 0); --darkreader-inline-border-left: #ab6219;" data-darkreader-inline-border-left="">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.microbiologyresearch.org&amp;citation_id=31549467&amp;tab=patents">
          Referenced in <b>2</b> patents
      </a>
    </div>
    <div style="padding-left: 10px; line-height: 18px; border-left: 16px solid rgb(149, 136, 153); --darkreader-inline-border-left: #595e61;" data-darkreader-inline-border-left="">
      <a class="link-to-altmetric-details-tab" target="_blank" href="https://www.altmetric.com/details.php?domain=www.microbiologyresearch.org&amp;citation_id=31549467&amp;tab=wikipedia">
          Referenced in <b>2</b> Wikipedia pages
      </a>
    </div>

    <div class="altmetric-embed readers" style="margin-top: 10px;">
          <div class="altmetric-embed tip_mendeley" style="padding-left: 10px; line-height: 18px; border-left: 16px solid rgb(166, 0, 0); --darkreader-inline-border-left: #be1919;" data-darkreader-inline-border-left="">
              <b>20</b> readers on Mendeley
          </div>
    </div>
    <div style="margin-top: 10px; text-align: center;">
        <a class="altmetric_details" target="_blank" href="https://www.altmetric.com/details.php?domain=www.microbiologyresearch.org&amp;citation_id=31549467">
            See more details
        </a>
    </div>
</div></div></div></div></div><script async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/sholic.js"></script><script async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/dpx.js"></script><script async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/taglw.aspx"></script><img src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/p" alt="score" style="height: 0px; width: 0px; border: 0px; display: none; --darkreader-inline-border-top: initial; --darkreader-inline-border-right: initial; --darkreader-inline-border-bottom: initial; --darkreader-inline-border-left: initial;" data-darkreader-inline-border-top="" data-darkreader-inline-border-right="" data-darkreader-inline-border-bottom="" data-darkreader-inline-border-left=""><script type="text/javascript" async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/saved_resource(1)"></script><script async="" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/afsh.js"></script><iframe title="no content javascript process" src="./Mutational analysis of the MS2 lysis protein L _ Microbiology Society_files/eps.html" style="display: none;"></iframe></body></html>